INSERT INTO soterms (id, name, definition) VALUES ('000000', 'Sequence_Ontology', '');
INSERT INTO soterms (id, name, definition) VALUES ('000001', 'region', 'A sequence_feature with an extent greater than zero. A nucleotide region is composed of bases and a polypeptide region is composed of amino acids." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000002', 'sequence_secondary_structure', 'A folded sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000003', 'G_quartet', 'G-quartets are unusual nucleic acid structures consisting of a planar arrangement where each guanine is hydrogen bonded by hoogsteen pairing to another guanine in the quartet." [http://www.ncbi.nlm.nih.gov/pubmed/7919797?dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000004', 'interior_coding_exon', 'G-quartets are unusual nucleic acid structures consisting of a planar arrangement where each guanine is hydrogen bonded by hoogsteen pairing to another guanine in the quartet." [http://www.ncbi.nlm.nih.gov/pubmed/7919797?dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000005', 'satellite_DNA', 'The many tandem repeats (identical or related) of a short basic repeating unit; many have a base composition or other property different from the genome average that allows them to be separated from the bulk (main band) genomic DNA." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000006', 'PCR_product', 'A region amplified by a PCR reaction." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000007', 'read_pair', 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000008', 'gene_sensu_your_favorite_organism', 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000009', 'gene_class', 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000010', 'protein_coding', 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000011', 'non_protein_coding', 'One of a pair of sequencing reads in which the two members of the pair are related by originating at either end of a clone insert." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000012', 'scRNA_primary_transcript', 'The primary transcript of any one of several small cytoplasmic RNA molecules present in the cytoplasm and sometimes nucleus of a Eukaryote." [http://www.ebi.ac.uk/embl/WebFeat/align/scRNA_s.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000013', 'scRNA', 'A small non coding RNA sequence, present in the cytoplasm." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000014', 'INR_motif', 'A sequence element characteristic of some RNA polymerase II promoters required for the correct positioning of the polymerase for the start of transcription. Overlaps the TSS. The mammalian consensus sequence is YYAN(T|A)YY; the Drosophila consensus sequence is TCA(G|T)t(T|C). In each the A is at position +1 with respect to the TSS. Functionally similar to the TATA box element." [PMID:12651739, PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('000015', 'DPE_motif', 'A sequence element characteristic of some RNA polymerase II promoters; Positioned from +28 to +32 with respect to the TSS (+1). Experimental results suggest that the DPE acts in conjunction with the INR_motif to provide a binding site for TFIID in the absence of a TATA box to mediate transcription of TATA-less promoters. Consensus sequence (A|G)G(A|T)(C|T)(G|A|C)." [PMID:12651739\:12537576, PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('000016', 'BREu_motif', 'A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements at -37 to -32 with respect to the TSS (+1). Consensus sequence is (G|C)(G|C)(G|A)CGCC. Binds TFIIB." [PMID:12651739, PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('000017', 'PSE_motif', 'A sequence element characteristic of the promoters of snRNA genes transcribed by RNA polymerase II or by RNA polymerase III. Located between -45 and -60 relative to the TSS. The human PSE_motif consensus sequence is TCACCNTNA(C|G)TNAAAAG(T|G)." [PMID:12651739]');
INSERT INTO soterms (id, name, definition) VALUES ('000018', 'linkage_group', 'A group of loci that can be grouped in a linear order representing the different degrees of linkage among the genes concerned." [ISBN:038752046]');
INSERT INTO soterms (id, name, definition) VALUES ('000020', 'RNA_internal_loop', 'A region of double stranded RNA where the bases do not conform to WC base pairing. The loop is closed on both sides by canonical base pairing. If the interruption to base pairing occurs on one strand only, it is known as a bulge." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000021', 'asymmetric_RNA_internal_loop', 'An internal RNA loop where one of the strands includes more bases than the corresponding region on the other strand." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000022', 'A_minor_RNA_motif', 'A region forming a motif, composed of adenines, where the minor groove edges are inserted into the minor groove of another helix." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000023', 'K_turn_RNA_motif', 'The kink turn (K-turn) is an RNA structural motif that creates a sharp (~120 degree) bend between two continuous helices." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000024', 'sarcin_like_RNA_motif', 'A loop in ribosomal RNA containing the sites of attack for ricin and sarcin." [http://www.ncbi.nlm.nih.gov/pubmed/7897662]');
INSERT INTO soterms (id, name, definition) VALUES ('000025', 'symmetric_RNA_internal_loop', 'An internal RNA loop where the extent of the loop on both stands is the same size." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000026', 'RNA_junction_loop', 'An internal RNA loop where the extent of the loop on both stands is the same size." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000027', 'RNA_hook_turn', 'An internal RNA loop where the extent of the loop on both stands is the same size." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000028', 'base_pair', 'An internal RNA loop where the extent of the loop on both stands is the same size." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000029', 'WC_base_pair', 'The canonical base pair, where two bases interact via WC edges, with glycosidic bonds oriented cis relative to the axis of orientation." [PMID:12177293]');
INSERT INTO soterms (id, name, definition) VALUES ('000030', 'sugar_edge_base_pair', 'A type of non-canonical base-pairing." [PMID:12177293]');
INSERT INTO soterms (id, name, definition) VALUES ('000031', 'aptamer', 'DNA or RNA molecules that have been selected from random pools based on their ability to bind other molecules." [http://aptamer.icmb.utexas.edu]');
INSERT INTO soterms (id, name, definition) VALUES ('000032', 'DNA_aptamer', 'DNA molecules that have been selected from random pools based on their ability to bind other molecules." [http:aptamer.icmb.utexas.edu]');
INSERT INTO soterms (id, name, definition) VALUES ('000033', 'RNA_aptamer', 'RNA molecules that have been selected from random pools based on their ability to bind other molecules." [http://aptamer.icmb.utexas.edu]');
INSERT INTO soterms (id, name, definition) VALUES ('000034', 'morpholino_oligo', 'Morpholino oligos are synthesized from four different Morpholino subunits, each of which contains one of the four genetic bases (A, C, G, T) linked to a 6-membered morpholine ring. Eighteen to 25 subunits of these four subunit types are joined in a specific order by non-ionic phosphorodiamidate intersubunit linkages to give a Morpholino." [http://www.gene-tools.com/]');
INSERT INTO soterms (id, name, definition) VALUES ('000035', 'riboswitch', 'A riboswitch is a part of an mRNA that can act as a direct sensor of small molecules to control their own expression. A riboswitch is a cis element in the 5 end of an mRNA, that acts as a direct sensor of metabolites." [PMID:2820954]');
INSERT INTO soterms (id, name, definition) VALUES ('000036', 'matrix_attachment_site', 'A DNA region that is required for the binding of chromatin to the nuclear matrix." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000037', 'locus_control_region', 'A DNA region that includes DNAse hypersensitive sites located 5 to a gene that confers the high-level, position-independent, and copy number-dependent expression to that gene." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000038', 'match_set', 'A collection of match parts." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000039', 'match_part', 'A part of a match, for example an hsp from blast is a match_part." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000040', 'genomic_clone', 'A clone of a DNA region of a genome." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000041', 'sequence_operation', 'An operation that can be applied to a sequence, that results in a change." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000042', 'pseudogene_attribute', 'An attribute of a pseudogene (SO:0000336)." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000043', 'processed_pseudogene', 'A pseudogene created via retrotranposition of the mRNA of a functional protein-coding parent gene followed by accumulation of deleterious mutations lacking introns and promoters, often including a polyA tail." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000044', 'pseudogene_by_unequal_crossing_over', 'A pseudogene caused by unequal crossing over at recombination." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000045', 'delete', 'To remove a subsection of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000046', 'insert', 'To insert a subsection of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000047', 'invert', 'To invert a subsection of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000048', 'substitute', 'To substitute a subsection of sequence for another." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000049', 'translocate', 'To translocate a subsection of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000050', 'gene_part', 'A part of a gene, that has no other route in the ontology back to region. This concept is necessary for logical inference as these parts must have the properties of region. It also allows us to associate all the parts of genes with a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000051', 'probe', 'A DNA sequence used experimentally to detect the presence or absence of a complementary nucleic acid." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000052', 'assortment_derived_deficiency', 'A DNA sequence used experimentally to detect the presence or absence of a complementary nucleic acid." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000053', 'sequence_variant_affecting_regulatory_region', 'A sequence_variant_effect which changes the regulatory region of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000054', 'aneuploid', 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000055', 'hyperploid', 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as extra chromosomes are present." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000056', 'hypoploid', 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number as some chromosomes are missing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000057', 'operator', 'A regulatory element of an operon to which activators or repressors bind thereby effecting translation of genes in that operon." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000058', 'assortment_derived_aneuploid', 'A regulatory element of an operon to which activators or repressors bind thereby effecting translation of genes in that operon." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000059', 'nuclease_binding_site', 'A binding site that, of a nucleotide molecule, that interacts selectively and non-covalently with polypeptide residues of a nuclease." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000060', 'compound_chromosome_arm', 'A binding site that, of a nucleotide molecule, that interacts selectively and non-covalently with polypeptide residues of a nuclease." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000061', 'restriction_enzyme_binding_site', 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues of a restriction enzyme." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000062', 'deficient_intrachromosomal_transposition', 'An intrachromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000063', 'deficient_interchromosomal_transposition', 'An interchromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000064', 'gene_by_transcript_attribute', 'An interchromosomal transposition whereby a translocation in which one of the four broken ends loses a segment before re-joining." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000065', 'free_chromosome_arm', 'A chromosome structure variation whereby an arm exists as an individual chromosome element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000066', 'gene_by_polyadenylation_attribute', 'A chromosome structure variation whereby an arm exists as an individual chromosome element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000067', 'gene_to_gene_feature', 'A chromosome structure variation whereby an arm exists as an individual chromosome element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000068', 'overlapping', 'An attribute describing a gene that has a sequence that overlaps the sequence of another gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000069', 'inside_intron', 'An attribute to describe a gene when it is located within the intron of another gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000070', 'inside_intron_antiparallel', 'An attribute to describe a gene when it is located within the intron of another gene and on the opposite strand." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000071', 'inside_intron_parallel', 'An attribute to describe a gene when it is located within the intron of another gene and on the same strand." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000072', 'end_overlapping_gene', 'An attribute to describe a gene when it is located within the intron of another gene and on the same strand." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000073', 'five_prime_three_prime_overlap', 'An attribute to describe a gene when the five prime region overlaps with another genes 3 region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000074', 'five_prime_five_prime_overlap', 'An attribute to describe a gene when the five prime region overlaps with another genes five prime region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000075', 'three_prime_three_prime_overlap', 'An attribute to describe a gene when the 3 region overlaps with another genes 3 region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000076', 'three_prime_five_prime_overlap', 'An attribute to describe a gene when the 3 region overlaps with another genes 5 region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000077', 'antisense', 'A region sequence that is complementary to a sequence of messenger RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000078', 'polycistronic_transcript', 'A transcript that is polycistronic." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000079', 'dicistronic_transcript', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000080', 'operon_member', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000081', 'gene_array_member', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000082', 'processed_transcript_attribute', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000083', 'macronuclear_sequence', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000084', 'micronuclear_sequence', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000085', 'gene_by_genome_location', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000086', 'gene_by_organelle_of_genome', 'A transcript that is dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000087', 'nuclear_gene', 'A gene from nuclear sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000088', 'mt_gene', 'A gene located in mitochondrial sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000089', 'kinetoplast_gene', 'A gene located in kinetoplast sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000090', 'plastid_gene', 'A gene from plastid sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000091', 'apicoplast_gene', 'A gene from apicoplast sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000092', 'ct_gene', 'A gene from chloroplast sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000093', 'chromoplast_gene', 'A gene from chromoplast_sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000094', 'cyanelle_gene', 'A gene from cyanelle sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000095', 'leucoplast_gene', 'A plastid gene from leucoplast sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000096', 'proplastid_gene', 'A gene from proplastid sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000097', 'nucleomorph_gene', 'A gene from nucleomorph sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000098', 'plasmid_gene', 'A gene from plasmid sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000099', 'proviral_gene', 'A gene from proviral sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000100', 'endogenous_retroviral_gene', 'A proviral gene with origin endogenous retrovirus." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000101', 'transposable_element', 'A transposon or insertion sequence. An element that can insert in a variety of DNA sequences." [http://www.sci.sdsu.edu/~smaloy/Glossary/T.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000102', 'expressed_sequence_match', 'A match to an EST or cDNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000103', 'clone_insert_end', 'The end of the clone insert." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000104', 'polypeptide', 'A sequence of amino acids linked by peptide bonds which may lack appreciable tertiary structure and may not be liable to irreversible denaturation." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000105', 'chromosome_arm', 'A region of the chromosome between the centromere and the telomere. Human chromosomes have two arms, the p arm (short) and the q arm (long) which are separated from each other by the centromere." [http://www.medterms.com/script/main/art.asp?articlekey=5152]');
INSERT INTO soterms (id, name, definition) VALUES ('000106', 'non_capped_primary_transcript', 'A region of the chromosome between the centromere and the telomere. Human chromosomes have two arms, the p arm (short) and the q arm (long) which are separated from each other by the centromere." [http://www.medterms.com/script/main/art.asp?articlekey=5152]');
INSERT INTO soterms (id, name, definition) VALUES ('000107', 'sequencing_primer', 'A region of the chromosome between the centromere and the telomere. Human chromosomes have two arms, the p arm (short) and the q arm (long) which are separated from each other by the centromere." [http://www.medterms.com/script/main/art.asp?articlekey=5152]');
INSERT INTO soterms (id, name, definition) VALUES ('000108', 'mRNA_with_frameshift', 'An mRNA with a frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000109', 'sequence_variant_obs', 'A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000110', 'sequence_feature', 'Any extent of continuous biological sequence." [LAMHDI:mb, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000111', 'transposable_element_gene', 'A gene encoded within a transposable element. For example gag, int, env and pol are the transposable element genes of the TY element in yeast." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000112', 'primer', 'An oligo to which new deoxyribonucleotides can be added by DNA polymerase." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000113', 'proviral_region', 'A viral sequence which has integrated into a host genome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000114', 'methylated_cytosine', 'A methylated deoxy-cytosine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000115', 'transcript_feature', 'A methylated deoxy-cytosine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000116', 'edited', 'An attribute describing a sequence that is modified by editing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000117', 'transcript_with_readthrough_stop_codon', 'An attribute describing a sequence that is modified by editing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000118', 'transcript_with_translational_frameshift', 'A transcript with a translational frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000119', 'regulated', 'An attribute to describe a sequence that is regulated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000120', 'protein_coding_primary_transcript', 'A primary transcript that, at least in part, encodes one or more proteins." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000121', 'forward_primer', 'A single stranded oligo used for polymerase chain reaction." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000122', 'RNA_sequence_secondary_structure', 'A folded RNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000123', 'transcriptionally_regulated', 'An attribute describing a gene that is regulated at transcription." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000124', 'transcriptionally_constitutive', 'Expressed in relatively constant amounts without regard to cellular environmental conditions such as the concentration of a particular substrate." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000125', 'transcriptionally_induced', 'An inducer molecule is required for transcription to occur." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000126', 'transcriptionally_repressed', 'A repressor molecule is required for transcription to stop." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000127', 'silenced_gene', 'A gene that is silenced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000128', 'gene_silenced_by_DNA_modification', 'A gene that is silenced by DNA modification." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000129', 'gene_silenced_by_DNA_methylation', 'A gene that is silenced by DNA methylation." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000130', 'post_translationally_regulated', 'An attribute describing a gene that is regulated after it has been translated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000131', 'translationally_regulated', 'An attribute describing a gene that is regulated as it is translated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000132', 'reverse_primer', 'A single stranded oligo used for polymerase chain reaction." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000133', 'epigenetically_modified', 'This attribute describes a gene where heritable changes other than those in the DNA sequence occur. These changes include: modification to the DNA (such as DNA methylation, the covalent modification of cytosine), and post-translational modification of histones." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000134', 'genomically_imprinted', 'Imprinted genes are epigenetically modified genes that are expressed monoallelically according to their parent of origin." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000135', 'maternally_imprinted', 'The maternal copy of the gene is modified, rendering it transcriptionally silent." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000136', 'paternally_imprinted', 'The paternal copy of the gene is modified, rendering it transcriptionally silent." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000137', 'allelically_excluded', 'Allelic exclusion is a process occurring in diploid organisms, where a gene is inactivated and not expressed in that cell." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000138', 'gene_rearranged_at_DNA_level', 'An epigenetically modified gene, rearranged at the DNA level." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000139', 'ribosome_entry_site', 'Region in mRNA where ribosome assembles." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000140', 'attenuator', 'A sequence segment located within the five prime end of an mRNA that causes premature termination of translation." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000141', 'terminator', 'The sequence of DNA located either at the end of the transcript that causes RNA polymerase to terminate transcription." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000142', 'DNA_sequence_secondary_structure', 'A folded DNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000143', 'assembly_component', 'A region of known length which may be used to manufacture a longer region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000144', 'primary_transcript_attribute', 'A region of known length which may be used to manufacture a longer region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000145', 'recoded_codon', 'A codon that has been redefined at translation. The redefinition may be as a result of translational bypass, translational frameshifting or stop codon readthrough." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000146', 'capped', 'An attribute describing when a sequence, usually an mRNA is capped by the addition of a modified guanine nucleotide at the 5 end." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000147', 'exon', 'A region of the transcript sequence within a gene which is not removed from the primary RNA transcript by RNA splicing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000148', 'supercontig', 'One or more contigs that have been ordered and oriented using end-read information. Contains gaps that are filled with Ns." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000149', 'contig', 'A contiguous sequence derived from sequence assembly. Has no gaps, but may contain Ns from unavailable bases." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000150', 'read', 'A sequence obtained from a single sequencing experiment. Typically a read is produced when a base calling program interprets information from a chromatogram trace file produced from a sequencing machine." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000151', 'clone', 'A piece of DNA that has been inserted in a vector so that it can be propagated in a host bacterium or some other organism." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000152', 'YAC', 'Yeast Artificial Chromosome, a vector constructed from the telomeric, centromeric, and replication origin sequences needed for replication in yeast cells." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000153', 'BAC', 'Bacterial Artificial Chromosome, a cloning vector that can be propagated as mini-chromosomes in a bacterial host." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000154', 'PAC', 'The P1-derived artificial chromosome are DNA constructs that are derived from the DNA of P1 bacteriophage. They can carry large amounts (about 100-300 kilobases) of other sequences for a variety of bioengineering purposes. It is one type of vector used to clone DNA fragments (100- to 300-kb insert size; average, 150 kb) in Escherichia coli cells." [http://en.wikipedia.org/wiki/P1-derived_artificial_chromosome]');
INSERT INTO soterms (id, name, definition) VALUES ('000155', 'plasmid', 'A self replicating, using the hosts cellular machinery, often circular nucleic acid molecule that is distinct from a chromosome in the organism." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000156', 'cosmid', 'A cloning vector that is a hybrid of lambda phages and a plasmid that can be propagated as a plasmid or packaged as a phage,since they retain the lambda cos sites." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000157', 'phagemid', 'A plasmid which carries within its sequence a bacteriophage replication origin. When the host bacterium is infected with \"helper\" phage, a phagemid is replicated along with the phage DNA and packaged into phage capsids." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000158', 'fosmid', 'A cloning vector that utilizes the E. coli F factor." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000159', 'deletion', 'The point at which one or more contiguous nucleotides were excised." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000160', 'lambda_clone', 'A linear clone derived from lambda bacteriophage. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome." [ISBN:0-1767-2380-8]');
INSERT INTO soterms (id, name, definition) VALUES ('000161', 'methylated_adenine', 'A modified  base in which adenine has been methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000162', 'splice_site', 'Consensus region of primary transcript bordering junction of splicing. A region that overlaps exactly 2 base and adjacent_to splice_junction." [SO:cjm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000163', 'five_prime_cis_splice_site', 'Intronic 2 bp region bordering the exon, at the 5 edge of the intron. A splice_site that is downstream_adjacent_to exon and starts intron." [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html, SO:cjm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000164', 'three_prime_cis_splice_site', 'Intronic 2 bp region bordering the exon, at the 3 edge of the intron. A splice_site that is upstream_adjacent_to exon and finishes intron." [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html, SO:cjm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000165', 'enhancer', 'A cis-acting sequence that increases the utilization of (some) eukaryotic promoters, and can function in either orientation and in any location (upstream or downstream) relative to the promoter." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000166', 'enhancer_bound_by_factor', 'An enhancer bound by a factor." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000167', 'promoter', 'A regulatory_region composed of the TSS(s) and binding sites for TF_complexes of the basal transcription machinery." [SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('000168', 'restriction_enzyme_cut_site', 'A specific nucleotide sequence of DNA at or near which a particular restriction enzyme cuts the DNA." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000169', 'RNApol_I_promoter', 'A DNA sequence in eukaryotic DNA to which RNA polymerase I binds, to begin transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000170', 'RNApol_II_promoter', 'A DNA sequence in eukaryotic DNA to which RNA polymerase II binds, to begin transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000171', 'RNApol_III_promoter', 'A DNA sequence in eukaryotic DNA to which RNA polymerase III binds, to begin transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000172', 'CAAT_signal', 'Part of a conserved sequence located about 75-bp upstream of the start point of eukaryotic transcription units which may be involved in RNA polymerase binding; consensus=GG(C|T)CAATCT." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000173', 'GC_rich_promoter_region', 'A conserved GC-rich region located upstream of the start point of eukaryotic transcription units which may occur in multiple copies or in either orientation; consensus=GGGCGG." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000174', 'TATA_box', 'A conserved AT-rich septamer found about 25-bp before the start point of many eukaryotic RNA polymerase II transcript units; may be involved in positioning the enzyme for correct initiation; consensus=TATA(A|T)A(A|T)." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('000175', 'minus_10_signal', 'A conserved region about 10-bp upstream of the start point of bacterial transcription units which may be involved in binding RNA polymerase; consensus=TAtAaT. This region is associated with sigma factor 70." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000176', 'minus_35_signal', 'A conserved hexamer about 35-bp upstream of the start point of bacterial transcription units; consensus=TTGACa or TGTTGACA. This region is associated with sigma factor 70." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000177', 'cross_genome_match', 'A nucleotide match against a sequence from another organism." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000178', 'operon', 'A group of contiguous genes transcribed as a single (polycistronic) mRNA from a single regulatory region." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000179', 'clone_insert_start', 'The start of the clone insert." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000180', 'retrotransposon', 'A transposable element that is incorporated into a chromosome by a mechanism that requires reverse transcriptase." [http://www.dddmag.com/Glossary.aspx#r]');
INSERT INTO soterms (id, name, definition) VALUES ('000181', 'translated_nucleotide_match', 'A match against a translated sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000182', 'DNA_transposon', 'A transposon where the mechanism of transposition is via a DNA intermediate." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000183', 'non_transcribed_region', 'A region of the gene which is not transcribed." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000184', 'U2_intron', 'A major type of spliceosomal intron spliced by the U2 spliceosome, that includes U1, U2, U4/U6 and U5 snRNAs." [PMID:9428511]');
INSERT INTO soterms (id, name, definition) VALUES ('000185', 'primary_transcript', 'A transcript that in its initial state requires modification to be functional." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000186', 'LTR_retrotransposon', 'A retrotransposon flanked by long terminal repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000187', 'repeat_family', 'A group of characterized repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000188', 'intron', 'A region of a primary transcript that is transcribed, but removed from within the transcript by splicing together the sequences (exons) on either side of it." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000189', 'non_LTR_retrotransposon', 'A retrotransposon without long terminal repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000190', 'five_prime_intron', 'A retrotransposon without long terminal repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000191', 'interior_intron', 'A retrotransposon without long terminal repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000192', 'three_prime_intron', 'A retrotransposon without long terminal repeat sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000193', 'RFLP_fragment', 'A DNA fragment used as a reagent to detect the polymorphic genomic loci by hybridizing against the genomic DNA digested with a given restriction enzyme." [GOC:pj]');
INSERT INTO soterms (id, name, definition) VALUES ('000194', 'LINE_element', 'A dispersed repeat family with many copies, each from 1 to 6 kb long. New elements are generated by retroposition of a transcribed copy. Typically the LINE contains 2 ORFs one of which is reverse transcriptase, and 3and 5 direct repeats." [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000195', 'coding_exon', 'An exon whereby at least one base is part of a codon (here, codon is inclusive of the stop_codon)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000196', 'five_prime_coding_exon_coding_region', 'The sequence of the five_prime_coding_exon that codes for protein." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000197', 'three_prime_coding_exon_coding_region', 'The sequence of the three_prime_coding_exon that codes for protein." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000198', 'noncoding_exon', 'An exon that does not contain any codons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000199', 'translocation', 'A region of nucleotide sequence that has translocated to a new position. The observed adjacency of two previously separated regions." [NCBI:th, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000200', 'five_prime_coding_exon', 'The 5 most coding exon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000201', 'interior_exon', 'An exon that is bounded by 5 and 3 splice sites." [PMID:10373547]');
INSERT INTO soterms (id, name, definition) VALUES ('000202', 'three_prime_coding_exon', 'The coding exon that is most 3-prime on a given transcript." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000203', 'UTR', 'Messenger RNA sequences that are untranslated and lie five prime or three prime to sequences which are translated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000204', 'five_prime_UTR', 'A region at the 5 end of a mature transcript (preceding the initiation codon) that is not translated into a protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000205', 'three_prime_UTR', 'A region at the 3 end of a mature transcript (following the stop codon) that is not translated into a protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000206', 'SINE_element', 'A repetitive element, a few hundred base pairs long, that is dispersed throughout the genome. A common human SINE is the Alu element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000207', 'simple_sequence_length_variation', 'SSLP are a kind of sequence alteration where the number of repeated sequences in intergenic regions may differ." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000208', 'terminal_inverted_repeat_element', 'A DNA transposable element defined as having termini with perfect, or nearly perfect short inverted repeats, generally 10 - 40 nucleotides long." [http://www.genetics.org/cgi/reprint/156/4/1983.pdf]');
INSERT INTO soterms (id, name, definition) VALUES ('000209', 'rRNA_primary_transcript', 'A primary transcript encoding a ribosomal RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000210', 'tRNA_primary_transcript', 'A primary transcript encoding a transfer RNA (SO:0000253)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000211', 'alanine_tRNA_primary_transcript', 'A primary transcript encoding alanyl tRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000212', 'arginine_tRNA_primary_transcript', 'A primary transcript encoding arginyl tRNA (SO:0000255)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000213', 'asparagine_tRNA_primary_transcript', 'A primary transcript encoding asparaginyl tRNA (SO:0000256)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000214', 'aspartic_acid_tRNA_primary_transcript', 'A primary transcript encoding aspartyl tRNA (SO:0000257)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000215', 'cysteine_tRNA_primary_transcript', 'A primary transcript encoding cysteinyl tRNA (SO:0000258)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000216', 'glutamic_acid_tRNA_primary_transcript', 'A primary transcript encoding glutaminyl tRNA (SO:0000260)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000217', 'glutamine_tRNA_primary_transcript', 'A primary transcript encoding glutamyl tRNA (SO:0000260)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000218', 'glycine_tRNA_primary_transcript', 'A primary transcript encoding glycyl tRNA (SO:0000263)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000219', 'histidine_tRNA_primary_transcript', 'A primary transcript encoding histidyl tRNA (SO:0000262)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000220', 'isoleucine_tRNA_primary_transcript', 'A primary transcript encoding isoleucyl tRNA (SO:0000263)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000221', 'leucine_tRNA_primary_transcript', 'A primary transcript encoding leucyl tRNA (SO:0000264)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000222', 'lysine_tRNA_primary_transcript', 'A primary transcript encoding lysyl tRNA (SO:0000265)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000223', 'methionine_tRNA_primary_transcript', 'A primary transcript encoding methionyl tRNA (SO:0000266)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000224', 'phenylalanine_tRNA_primary_transcript', 'A primary transcript encoding phenylalanyl tRNA (SO:0000267)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000225', 'proline_tRNA_primary_transcript', 'A primary transcript encoding prolyl tRNA (SO:0000268)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000226', 'serine_tRNA_primary_transcript', 'A primary transcript encoding seryl tRNA (SO:000269)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000227', 'threonine_tRNA_primary_transcript', 'A primary transcript encoding threonyl tRNA (SO:000270)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000228', 'tryptophan_tRNA_primary_transcript', 'A primary transcript encoding tryptophanyl tRNA (SO:000271)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000229', 'tyrosine_tRNA_primary_transcript', 'A primary transcript encoding tyrosyl tRNA (SO:000272)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000230', 'valine_tRNA_primary_transcript', 'A primary transcript encoding valyl tRNA (SO:000273)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000231', 'snRNA_primary_transcript', 'A primary transcript encoding a small nuclear RNA (SO:0000274)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000232', 'snoRNA_primary_transcript', 'A primary transcript encoding a small nucleolar mRNA (SO:0000275)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000233', 'mature_transcript', 'A transcript which has undergone the necessary modifications, if any, for its function. In eukaryotes this includes, for example, processing of introns, cleavage, base modification, and modifications to the 5 and/or the 3 ends, other than addition of bases. In bacteria functional mRNAs are usually not modified." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000234', 'mRNA', 'Messenger RNA is the intermediate molecule between DNA and protein. It includes UTR and coding sequences. It does not contain introns." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000235', 'TF_binding_site', 'A region of a nucleotide molecule that binds a Transcription Factor or Transcription Factor complex [GO:0005667]." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000236', 'ORF', 'The in-frame interval between the stop codons of a reading frame which when read as sequential triplets, has the potential of encoding a sequential string of amino acids. TER(NNN)nTER." [SGD:rb, SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000237', 'transcript_attribute', 'The in-frame interval between the stop codons of a reading frame which when read as sequential triplets, has the potential of encoding a sequential string of amino acids. TER(NNN)nTER." [SGD:rb, SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000238', 'foldback_element', 'A transposable element with extensive secondary structure, characterized by large modular imperfect long inverted repeats." [http://www.genetics.org/cgi/reprint/156/4/1983.pdf]');
INSERT INTO soterms (id, name, definition) VALUES ('000239', 'flanking_region', 'The sequences extending on either side of a specific region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000240', 'chromosome_variation', 'The sequences extending on either side of a specific region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000241', 'internal_UTR', 'A UTR bordered by the terminal and initial codons of two CDSs in a polycistronic transcript. Every UTR is either 5, 3 or internal." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000242', 'untranslated_region_polycistronic_mRNA', 'The untranslated sequence separating the cistrons of multicistronic mRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000243', 'internal_ribosome_entry_site', 'Sequence element that recruits a ribosomal subunit to internal mRNA for translation initiation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000244', 'four_cutter_restriction_site', 'Sequence element that recruits a ribosomal subunit to internal mRNA for translation initiation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000245', 'mRNA_by_polyadenylation_status', 'Sequence element that recruits a ribosomal subunit to internal mRNA for translation initiation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000246', 'polyadenylated', 'A attribute describing the addition of a poly A tail to the 3 end of a mRNA molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000247', 'mRNA_not_polyadenylated', 'A attribute describing the addition of a poly A tail to the 3 end of a mRNA molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000248', 'sequence_length_alteration', 'A kind of kind of sequence alteration where the copies of a region present varies across a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000249', 'six_cutter_restriction_site', 'A kind of kind of sequence alteration where the copies of a region present varies across a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000250', 'modified_RNA_base_feature', 'A post_transcriptionally modified base." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000251', 'eight_cutter_restriction_site', 'A post_transcriptionally modified base." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000252', 'rRNA', 'RNA that comprises part of a ribosome, and that can provide both structural scaffolding and catalytic activity." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, ISBN:0198506732]');
INSERT INTO soterms (id, name, definition) VALUES ('000253', 'tRNA', 'Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3 end, to which the tRNAs corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and wobble base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00005, ISBN:0198506732]');
INSERT INTO soterms (id, name, definition) VALUES ('000254', 'alanyl_tRNA', 'A tRNA sequence that has an alanine anticodon, and a 3 alanine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000255', 'rRNA_small_subunit_primary_transcript', 'A primary transcript encoding a small ribosomal subunit RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000256', 'asparaginyl_tRNA', 'A tRNA sequence that has an asparagine anticodon, and a 3 asparagine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000257', 'aspartyl_tRNA', 'A tRNA sequence that has an aspartic acid anticodon, and a 3 aspartic acid binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000258', 'cysteinyl_tRNA', 'A tRNA sequence that has a cysteine anticodon, and a 3 cysteine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000259', 'glutaminyl_tRNA', 'A tRNA sequence that has a glutamine anticodon, and a 3 glutamine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000260', 'glutamyl_tRNA', 'A tRNA sequence that has a glutamic acid anticodon, and a 3 glutamic acid binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000261', 'glycyl_tRNA', 'A tRNA sequence that has a glycine anticodon, and a 3 glycine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000262', 'histidyl_tRNA', 'A tRNA sequence that has a histidine anticodon, and a 3 histidine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000263', 'isoleucyl_tRNA', 'A tRNA sequence that has an isoleucine anticodon, and a 3 isoleucine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000264', 'leucyl_tRNA', 'A tRNA sequence that has a leucine anticodon, and a 3 leucine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000265', 'lysyl_tRNA', 'A tRNA sequence that has a lysine anticodon, and a 3 lysine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000266', 'methionyl_tRNA', 'A tRNA sequence that has a methionine anticodon, and a 3 methionine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000267', 'phenylalanyl_tRNA', 'A tRNA sequence that has a phenylalanine anticodon, and a 3 phenylalanine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000268', 'prolyl_tRNA', 'A tRNA sequence that has a proline anticodon, and a 3 proline binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000269', 'seryl_tRNA', 'A tRNA sequence that has a serine anticodon, and a 3 serine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000270', 'threonyl_tRNA', 'A tRNA sequence that has a threonine anticodon, and a 3 threonine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000271', 'tryptophanyl_tRNA', 'A tRNA sequence that has a tryptophan anticodon, and a 3 tryptophan binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000272', 'tyrosyl_tRNA', 'A tRNA sequence that has a tyrosine anticodon, and a 3 tyrosine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000273', 'valyl_tRNA', 'A tRNA sequence that has a valine anticodon, and a 3 valine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000274', 'snRNA', 'A small nuclear RNA molecule involved in pre-mRNA splicing and processing." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, PMID:11733745, WB:ems]');
INSERT INTO soterms (id, name, definition) VALUES ('000275', 'snoRNA', 'A snoRNA (small nucleolar RNA) is any one of a class of small RNAs that are associated with the eukaryotic nucleus as components of small nucleolar ribonucleoproteins. They participate in the processing or modifications of many RNAs, mostly ribosomal RNAs (rRNAs) though snoRNAs are also known to target other classes of RNA, including spliceosomal RNAs, tRNAs, and mRNAs via a stretch of sequence that is complementary to a sequence in the targeted RNA." [GOC:kgc]');
INSERT INTO soterms (id, name, definition) VALUES ('000276', 'miRNA', 'Small, ~22-nt, RNA molecule that is the endogenous transcript of a miRNA gene (or the product of other non coding RNA genes. Micro RNAs are produced from precursor molecules (SO:0000647) that can form local hairpin structures, which ordinarily are processed (usually via the Dicer pathway) such that a single miRNA molecule accumulates from one arm of a hairpin precursor molecule. Micro RNAs may trigger the cleavage of their target molecules or act as translational repressors." [PMID:11081512, PMID:12592000]');
INSERT INTO soterms (id, name, definition) VALUES ('000277', 'bound_by_factor', 'An attribute describing a sequence that is bound by another molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000278', 'transcript_bound_by_nucleic_acid', 'A transcript that is bound by a nucleic acid." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000279', 'transcript_bound_by_protein', 'A transcript that is bound by a protein." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000280', 'engineered_gene', 'A gene that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000281', 'engineered_foreign_gene', 'A gene that is engineered and foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000282', 'mRNA_with_minus_1_frameshift', 'An mRNA with a minus 1 frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000283', 'engineered_foreign_transposable_element_gene', 'A transposable_element that is engineered and foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000284', 'type_I_enzyme_restriction_site', 'The recognition site is bipartite and interrupted." [http://www.promega.com]');
INSERT INTO soterms (id, name, definition) VALUES ('000285', 'foreign_gene', 'A gene that is foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000286', 'long_terminal_repeat', 'A sequence directly repeated at both ends of a defined sequence, of the sort typically found in retroviruses." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000287', 'fusion_gene', 'A gene that is a fusion." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000288', 'engineered_fusion_gene', 'A fusion gene that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000289', 'microsatellite', 'A repeat_region containing repeat_units of 2 to 10 bp repeated in tandem." [http://www.informatics.jax.org/silver/glossary.shtml, NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('000290', 'dinucleotide_repeat_microsatellite_feature', 'A repeat_region containing repeat_units of 2 to 10 bp repeated in tandem." [http://www.informatics.jax.org/silver/glossary.shtml, NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('000291', 'trinucleotide_repeat_microsatellite_feature', 'A repeat_region containing repeat_units of 2 to 10 bp repeated in tandem." [http://www.informatics.jax.org/silver/glossary.shtml, NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('000292', 'repetitive_element', 'A repeat_region containing repeat_units of 2 to 10 bp repeated in tandem." [http://www.informatics.jax.org/silver/glossary.shtml, NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('000293', 'engineered_foreign_repetitive_element', 'A repetitive element that is engineered and foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000294', 'inverted_repeat', 'The sequence is complementarily repeated on the opposite strand. It is a palindrome, and it may, or may not be hyphenated. Examples: GCTGATCAGC, or GCTGA-----TCAGC." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000295', 'U12_intron', 'A type of spliceosomal intron spliced by the U12 spliceosome, that includes U11, U12, U4atac/U6atac and U5 snRNAs." [PMID:9428511]');
INSERT INTO soterms (id, name, definition) VALUES ('000296', 'origin_of_replication', 'The origin of replication; starting site for duplication of a nucleic acid molecule to give two identical copies." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000297', 'D_loop', 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000298', 'recombination_feature', 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000299', 'specific_recombination_site', 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000300', 'recombination_feature_of_rearranged_gene', 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000301', 'vertebrate_immune_system_gene_recombination_feature', 'Displacement loop; a region within mitochondrial DNA in which a short stretch of RNA is paired with one strand of DNA, displacing the original partner DNA strand in this region; also used to describe the displacement of a region of one strand of duplex DNA by a single stranded invader in the reaction catalyzed by RecA protein." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000302', 'J_gene_recombination_feature', 'Recombination signal including J-heptamer, J-spacer and J-nonamer in 5 of J-region of a J-gene or J-sequence." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000303', 'clip', 'Part of the primary transcript that is clipped off during processing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000304', 'type_II_enzyme_restriction_site', 'The recognition site is either palindromic, partially palindromic or an interrupted palindrome. Cleavage occurs within the recognition site." [http://www.promega.com]');
INSERT INTO soterms (id, name, definition) VALUES ('000305', 'modified_DNA_base', 'A modified nucleotide, i.e. a nucleotide other than A, T, C. G." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000306', 'methylated_DNA_base_feature', 'A nucleotide modified by methylation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000307', 'CpG_island', 'Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5 ends of genes." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000308', 'sequence_feature_locating_method', 'Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5 ends of genes." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000309', 'computed_feature', 'Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5 ends of genes." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000310', 'predicted_ab_initio_computation', 'Regions of a few hundred to a few thousand bases in vertebrate genomes that are relatively GC and CpG rich; they are typically unmethylated and often found near the 5 ends of genes." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000311', 'computed_feature_by_similarity', '." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000312', 'experimentally_determined', 'Attribute to describe a feature that has been experimentally verified." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000313', 'stem_loop', 'A double-helical region of nucleic acid formed by base-pairing between adjacent (inverted) complementary sequences." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000314', 'direct_repeat', 'A repeat where the same sequence is repeated in the same direction. Example: GCTGA-followed by-GCTGA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000315', 'TSS', 'The first base where RNA polymerase begins to synthesize the RNA transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000316', 'CDS', 'A contiguous sequence which begins with, and includes, a start codon and ends with, and includes, a stop codon." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000317', 'cDNA_clone', 'Complementary DNA; A piece of DNA copied from an mRNA and spliced into a vector for propagation in a suitable host." [http://seqcore.brcf.med.umich.edu/doc/educ/dnapr/mbglossary/mbgloss.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000318', 'start_codon', 'First codon to be translated by a ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000319', 'stop_codon', 'In mRNA, a set of three nucleotides that indicates the end of information for protein synthesis." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000320', 'intronic_splice_enhancer', 'Sequences within the intron that modulate splice site selection for some introns." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000321', 'mRNA_with_plus_1_frameshift', 'An mRNA with a plus 1 frameshift." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000322', 'nuclease_hypersensitive_site', 'An mRNA with a plus 1 frameshift." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000323', 'coding_start', 'The first base to be translated into protein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000324', 'tag', 'A nucleotide sequence that may be used to identify a larger sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000325', 'rRNA_large_subunit_primary_transcript', 'A primary transcript encoding a large ribosomal subunit RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000326', 'SAGE_tag', 'A short diagnostic sequence tag, serial analysis of gene expression (SAGE), that allows the quantitative and simultaneous analysis of a large number of transcripts." [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=7570003&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000327', 'coding_end', 'The last base to be translated into protein. It does not include the stop codon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000328', 'microarray_oligo', 'The last base to be translated into protein. It does not include the stop codon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000329', 'mRNA_with_plus_2_frameshift', 'An mRNA with a plus 2 frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000330', 'conserved_region', 'Region of sequence similarity by descent from a common ancestor." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000331', 'STS', 'Short (typically a few hundred base pairs) DNA sequence that has a single occurrence in a genome and whose location and base sequence are known." [http://www.biospace.com]');
INSERT INTO soterms (id, name, definition) VALUES ('000332', 'coding_conserved_region', 'Coding region of sequence similarity by descent from a common ancestor." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000333', 'exon_junction', 'The boundary between two exons in a processed transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000334', 'nc_conserved_region', 'Non-coding region of sequence similarity by descent from a common ancestor." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000335', 'mRNA_with_minus_2_frameshift', 'A mRNA with a minus 2 frameshift." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000336', 'pseudogene', 'A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their \"normal\" paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its \"normal\" paralog)." [http://www.ucl.ac.uk/~ucbhjow/b241/glossary.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000337', 'RNAi_reagent', 'A double stranded RNA duplex, at least 20bp long, used experimentally to inhibit gene function by RNA interference." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000338', 'MITE', 'A highly repetitive and short (100-500 base pair) transposable element with terminal inverted repeats (TIR) and target site duplication (TSD). MITEs do not encode proteins." [http://www.pnas.org/cgi/content/full/97/18/10083]');
INSERT INTO soterms (id, name, definition) VALUES ('000339', 'recombination_hotspot', 'A region in a genome which promotes recombination." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000340', 'chromosome', 'Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000341', 'chromosome_band', 'A cytologically distinguishable feature of a chromosome, often made visible by staining, and usually alternating light and dark." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000342', 'site_specific_recombination_target_region', 'A cytologically distinguishable feature of a chromosome, often made visible by staining, and usually alternating light and dark." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000343', 'match', 'A region of sequence, aligned to another sequence with some statistical significance, using an algorithm such as BLAST or SIM4." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000344', 'splice_enhancer', 'Region of a transcript that regulates splicing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000345', 'EST', 'A tag produced from a single sequencing read from a cDNA clone or PCR product; typically a few hundred base pairs long." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000346', 'loxP_site', 'A tag produced from a single sequencing read from a cDNA clone or PCR product; typically a few hundred base pairs long." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000347', 'nucleotide_match', 'A match against a nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000348', 'nucleic_acid', 'An attribute describing a sequence consisting of nucleobases bound to repeating units. The forms found in nature are deoxyribonucleic acid (DNA), where the repeating units are 2-deoxy-D-ribose rings connected to a phosphate backbone, and ribonucleic acid (RNA), where the repeating units are D-ribose rings connected to a phosphate backbone." [CHEBI:33696, RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000349', 'protein_match', 'A match against a protein sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000350', 'FRT_site', 'An inversion site found on the Saccharomyces cerevisiae 2 micron plasmid." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000351', 'synthetic_sequence', 'An attribute to decide a sequence of nucleotides, nucleotide analogs, or amino acids that has been designed by an experimenter and which may, or may not, correspond with any natural sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000352', 'DNA', 'An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a 2-deoxy-D-ribose ring connected to a phosphate backbone." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000353', 'sequence_assembly', 'A sequence of nucleotides that has been algorithmically derived from an alignment of two or more different sequences." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000354', 'group_1_intron_homing_endonuclease_target_region', 'A region of intronic nucleotide sequence targeted by a nuclease enzyme." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000355', 'haplotype_block', 'A region of the genome which is co-inherited as the result of the lack of historic recombination within it." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000356', 'RNA', 'An attribute describing a sequence consisting of nucleobases bound to a repeating unit made of a D-ribose ring connected to a phosphate backbone." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000357', 'flanked', 'An attribute describing a region that is bounded either side by a particular kind of region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000359', 'floxed', 'An attribute describing sequence that is flanked by Lox-P sites." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000360', 'codon', 'A set of (usually) three nucleotide bases in a DNA or RNA sequence, which together code for a unique amino acid or the termination of translation and are contained within the CDS." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000361', 'FRT_flanked', 'An attribute to describe sequence that is flanked by the FLP recombinase recognition site, FRT." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000362', 'invalidated_by_chimeric_cDNA', 'A cDNA clone constructed from more than one mRNA. Usually an experimental artifact." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000363', 'floxed_gene', 'A transgene that is floxed." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000364', 'transposable_element_flanking_region', 'The region of sequence surrounding a transposable element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000365', 'integron', 'A region encoding an integrase which acts at a site adjacent to it (attI_site) to insert DNA which must include but is not limited to an attC_site." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000366', 'insertion_site', 'The junction where an insertion occurred." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000367', 'attI_site', 'A region within an integron, adjacent to an integrase, at which site specific recombination involving an attC_site takes place." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000368', 'transposable_element_insertion_site', 'The junction in a genome where a transposable_element has inserted." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000369', 'integrase_coding_region', 'The junction in a genome where a transposable_element has inserted." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000370', 'small_regulatory_ncRNA', 'A non-coding RNA, usually with a specific secondary structure, that acts to regulate gene expression." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000371', 'conjugative_transposon', 'A transposon that encodes function required for conjugation." [http://www.sci.sdsu.edu/~smaloy/Glossary/C.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000372', 'enzymatic_RNA', 'An RNA sequence that has catalytic activity with or without an associated ribonucleoprotein." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000373', 'recombinationally_inverted_gene', 'A recombinationally rearranged gene by inversion." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000374', 'ribozyme', 'An RNA with catalytic activity." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000375', 'rRNA_5_8S', '5_8S ribosomal RNA (5. 8S rRNA) is a component of the large subunit of the eukaryotic ribosome. It is transcribed by RNA polymerase I as part of the 45S precursor that also contains 18S and 28S rRNA. Functionally, it is thought that 5.8S rRNA may be involved in ribosome translocation. It is also known to form covalent linkage to the p53 tumour suppressor protein. 5_8S rRNA is also found in archaea." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00002]');
INSERT INTO soterms (id, name, definition) VALUES ('000376', 'RNA_6S', 'A small (184-nt in E. coli) RNA that forms a hairpin type structure. 6S RNA associates with RNA polymerase in a highly specific manner. 6S RNA represses expression from a sigma70-dependent promoter during stationary phase." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00013]');
INSERT INTO soterms (id, name, definition) VALUES ('000377', 'CsrB_RsmB_RNA', 'An enterobacterial RNA that binds the CsrA protein. The CsrB RNAs contain a conserved motif CAGGXXG that is found in up to 18 copies and has been suggested to bind CsrA. The Csr regulatory system has a strong negative regulatory effect on glycogen biosynthesis, glyconeogenesis and glycogen catabolism and a positive regulatory effect on glycolysis. In other bacteria such as Erwinia caratovara the RsmA protein has been shown to regulate the production of virulence determinants, such extracellular enzymes. RsmA binds to RsmB regulatory RNA which is also a member of this family." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00018]');
INSERT INTO soterms (id, name, definition) VALUES ('000378', 'DsrA_RNA', 'DsrA RNA regulates both transcription, by overcoming transcriptional silencing by the nucleoid-associated H-NS protein, and translation, by promoting efficient translation of the stress sigma factor, RpoS. These two activities of DsrA can be separated by mutation: the first of three stem-loops of the 85 nucleotide RNA is necessary for RpoS translation but not for anti-H-NS action, while the second stem-loop is essential for antisilencing and less critical for RpoS translation. The third stem-loop, which behaves as a transcription terminator, can be substituted by the trp transcription terminator without loss of either DsrA function. The sequence of the first stem-loop of DsrA is complementary with the upstream leader portion of RpoS messenger RNA, suggesting that pairing of DsrA with the RpoS message might be important for translational regulation." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00014]');
INSERT INTO soterms (id, name, definition) VALUES ('000379', 'GcvB_RNA', 'A small untranslated RNA involved in expression of the dipeptide and oligopeptide transport systems in Escherichia coli." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00022]');
INSERT INTO soterms (id, name, definition) VALUES ('000380', 'hammerhead_ribozyme', 'A small catalytic RNA motif that catalyzes self-cleavage reaction. Its name comes from its secondary structure which resembles a carpenters hammer. The hammerhead ribozyme is involved in the replication of some viroid and some satellite RNAs." [PMID:2436805]');
INSERT INTO soterms (id, name, definition) VALUES ('000381', 'group_IIA_intron', 'A small catalytic RNA motif that catalyzes self-cleavage reaction. Its name comes from its secondary structure which resembles a carpenters hammer. The hammerhead ribozyme is involved in the replication of some viroid and some satellite RNAs." [PMID:2436805]');
INSERT INTO soterms (id, name, definition) VALUES ('000382', 'group_IIB_intron', 'A small catalytic RNA motif that catalyzes self-cleavage reaction. Its name comes from its secondary structure which resembles a carpenters hammer. The hammerhead ribozyme is involved in the replication of some viroid and some satellite RNAs." [PMID:2436805]');
INSERT INTO soterms (id, name, definition) VALUES ('000383', 'MicF_RNA', 'A non-translated 93 nt antisense RNA that binds its target ompF mRNA and regulates ompF expression by inhibiting translation and inducing degradation of the message." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00033]');
INSERT INTO soterms (id, name, definition) VALUES ('000384', 'OxyS_RNA', 'A small untranslated RNA which is induced in response to oxidative stress in Escherichia coli. Acts as a global regulator to activate or repress the expression of as many as 40 genes, including the fhlA-encoded transcriptional activator and the rpoS-encoded sigma(s) subunit of RNA polymerase. OxyS is bound by the Hfq protein, that increases the OxyS RNA interaction with its target messages." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00035]');
INSERT INTO soterms (id, name, definition) VALUES ('000385', 'RNase_MRP_RNA', 'The RNA molecule essential for the catalytic activity of RNase MRP, an enzymatically active ribonucleoprotein with two distinct roles in eukaryotes. In mitochondria it plays a direct role in the initiation of mitochondrial DNA replication. In the nucleus it is involved in precursor rRNA processing, where it cleaves the internal transcribed spacer 1 between 18S and 5.8S rRNAs." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00030]');
INSERT INTO soterms (id, name, definition) VALUES ('000386', 'RNase_P_RNA', 'The RNA component of Ribonuclease P (RNase P), a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterized activity is the generation of mature 5 prime ends of tRNAs by cleaving the 5 prime leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzymes have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous. Helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00010]');
INSERT INTO soterms (id, name, definition) VALUES ('000387', 'RprA_RNA', 'Translational regulation of the stationary phase sigma factor RpoS is mediated by the formation of a double-stranded RNA stem-loop structure in the upstream region of the rpoS messenger RNA, occluding the translation initiation site. Clones carrying rprA (RpoS regulator RNA) increased the translation of RpoS. The rprA gene encodes a 106 nucleotide regulatory RNA. As with DsrA Rfam:RF00014, RprA is predicted to form three stem-loops. Thus, at least two small RNAs, DsrA and RprA, participate in the positive regulation of RpoS translation. Unlike DsrA, RprA does not have an extensive region of complementarity to the RpoS leader, leaving its mechanism of action unclear. RprA is non-essential." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00034]');
INSERT INTO soterms (id, name, definition) VALUES ('000388', 'RRE_RNA', 'The Rev response element (RRE) is encoded within the HIV-env gene. Rev is an essential regulatory protein of HIV that binds an internal loop of the RRE leading, encouraging further Rev-RRE binding. This RNP complex is critical for mRNA export and hence for expression of the HIV structural proteins." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00036]');
INSERT INTO soterms (id, name, definition) VALUES ('000389', 'spot_42_RNA', 'A 109-nucleotide RNA of E. coli that seems to have a regulatory role on the galactose operon. Changes in Spot 42 levels are implicated in affecting DNA polymerase I levels." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00021]');
INSERT INTO soterms (id, name, definition) VALUES ('000390', 'telomerase_RNA', 'The RNA component of telomerase, a reverse transcriptase that synthesizes telomeric DNA." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00025]');
INSERT INTO soterms (id, name, definition) VALUES ('000391', 'U1_snRNA', 'U1 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Its 5 end forms complementary base pairs with the 5 splice junction, thus defining the 5 donor site of an intron. There are significant differences in sequence and secondary structure between metazoan and yeast U1 snRNAs, the latter being much longer (568 nucleotides as compared to 164 nucleotides in human). Nevertheless, secondary structure predictions suggest that all U1 snRNAs share a common core consisting of helices I, II, the proximal region of III, and IV." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00003]');
INSERT INTO soterms (id, name, definition) VALUES ('000392', 'U2_snRNA', 'U2 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Complementary binding between U2 snRNA (in an area lying towards the 5 end but 3 to hairpin I) and the branchpoint sequence (BPS) of the intron results in the bulging out of an unpaired adenine, on the BPS, which initiates a nucleophilic attack at the intronic 5 splice site, thus starting the first of two transesterification reactions that mediate splicing." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00004]');
INSERT INTO soterms (id, name, definition) VALUES ('000393', 'U4_snRNA', 'U4 small nuclear RNA (U4 snRNA) is a component of the major U2-dependent spliceosome. It forms a duplex with U6, and with each splicing round, it is displaced from U6 (and the spliceosome) in an ATP-dependent manner, allowing U6 to refold and create the active site for splicing catalysis. A recycling process involving protein Prp24 re-anneals U4 and U6." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00015]');
INSERT INTO soterms (id, name, definition) VALUES ('000394', 'U4atac_snRNA', 'An snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U6atac_snRNA (SO:0000397)." [PMID:=12409455]');
INSERT INTO soterms (id, name, definition) VALUES ('000395', 'U5_snRNA', 'U5 RNA is a component of both types of known spliceosome. The precise function of this molecule is unknown, though it is known that the 5 loop is required for splice site selection and p220 binding, and that both the 3 stem-loop and the Sm site are important for Sm protein binding and cap methylation." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00020]');
INSERT INTO soterms (id, name, definition) VALUES ('000396', 'U6_snRNA', 'U6 snRNA is a component of the spliceosome which is involved in splicing pre-mRNA. The putative secondary structure consensus base pairing is confined to a short 5 stem loop, but U6 snRNA is thought to form extensive base-pair interactions with U4 snRNA." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00015]');
INSERT INTO soterms (id, name, definition) VALUES ('000397', 'U6atac_snRNA', 'U6atac_snRNA is an snRNA required for the splicing of the minor U12-dependent class of eukaryotic nuclear introns. It forms a base paired complex with U4atac_snRNA (SO:0000394)." [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&db=pubmed&list_uids=12409455&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000398', 'U11_snRNA', 'U11 snRNA plays a role in splicing of the minor U12-dependent class of eukaryotic nuclear introns, similar to U1 snRNA in the major class spliceosome it base pairs to the conserved 5 splice site sequence." [PMID:9622129]');
INSERT INTO soterms (id, name, definition) VALUES ('000399', 'U12_snRNA', 'The U12 small nuclear (snRNA), together with U4atac/U6atac, U5, and U11 snRNAs and associated proteins, forms a spliceosome that cleaves a divergent class of low-abundance pre-mRNA introns." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00007]');
INSERT INTO soterms (id, name, definition) VALUES ('000400', 'sequence_attribute', 'An attribute describes a quality of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000401', 'gene_attribute', 'An attribute describes a quality of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000402', 'enhancer_attribute', 'An attribute describes a quality of sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000403', 'U14_snoRNA', 'U14 small nucleolar RNA (U14 snoRNA) is required for early cleavages of eukaryotic precursor rRNAs. In yeasts, this molecule possess a stem-loop region (known as the Y-domain) which is essential for function. A similar structure, but with a different consensus sequence, is found in plants, but is absent in vertebrates." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00016, PMID:2551119]');
INSERT INTO soterms (id, name, definition) VALUES ('000404', 'vault_RNA', 'A family of RNAs are found as part of the enigmatic vault ribonucleoprotein complex. The complex consists of a major vault protein (MVP), two minor vault proteins (VPARP and TEP1), and several small untranslated RNA molecules. It has been suggested that the vault complex is involved in drug resistance." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00006]');
INSERT INTO soterms (id, name, definition) VALUES ('000405', 'Y_RNA', 'Y RNAs are components of the Ro ribonucleoprotein particle (Ro RNP), in association with Ro60 and La proteins. The Y RNAs and Ro60 and La proteins are well conserved, but the function of the Ro RNP is not known. In humans the RNA component can be one of four small RNAs: hY1, hY3, hY4 and hY5. These small RNAs are predicted to fold into a conserved secondary structure containing three stem structures. The largest of the four, hY1, contains an additional hairpin." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00019]');
INSERT INTO soterms (id, name, definition) VALUES ('000406', 'twintron', 'An intron within an intron. Twintrons are group II or III introns, into which another group II or III intron has been transposed." [PMID:1899376, PMID:7823908]');
INSERT INTO soterms (id, name, definition) VALUES ('000407', 'rRNA_18S', 'A large polynucleotide in eukaryotes, which functions as the small subunit of the ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000408', 'site', 'The interbase position where something (eg an aberration) occurred." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000409', 'binding_site', 'A biological_region of sequence that, in the molecule, interacts selectively and non-covalently with other molecules. A region on the surface of a molecule that may interact with another molecule. When applied to polypeptides: Amino acids involved in binding or interactions. It can also apply to an amino acid bond which is represented by the positions of the two flanking amino acids." [EBIBS:GAR, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000410', 'protein_binding_site', 'A binding site that, in the molecule, interacts selectively and non-covalently with polypeptide molecules." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000411', 'rescue_region', 'A region that rescues." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000412', 'restriction_fragment', 'A region of polynucleotide sequence produced by digestion with a restriction endonuclease." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000413', 'sequence_difference', 'A region where the sequence differs from that of a specified sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000414', 'invalidated_by_genomic_contamination', 'An attribute to describe a feature that is invalidated due to genomic contamination." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000415', 'invalidated_by_genomic_polyA_primed_cDNA', 'An attribute to describe a feature that is invalidated due to polyA priming." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000416', 'invalidated_by_partial_processing', 'An attribute to describe a feature that is invalidated due to partial processing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000417', 'polypeptide_domain', 'A structurally or functionally defined protein region. In proteins with multiple domains, the combination of the domains determines the function of the protein. A region which has been shown to recur throughout evolution." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('000418', 'signal_peptide', 'The signal_peptide is a short region of the peptide located at the N-terminus that directs the protein to be secreted or part of membrane components." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000419', 'mature_protein_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000420', 'five_prime_terminal_inverted_repeat', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000421', 'three_prime_terminal_inverted_repeat', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000422', 'U5_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000423', 'R_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000424', 'U3_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000425', 'five_prime_LTR', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000426', 'three_prime_LTR', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000427', 'R_five_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000428', 'U5_five_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000429', 'U3_five_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000430', 'R_three_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000431', 'U3_three_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000432', 'U5_three_prime_LTR_region', 'The polypeptide sequence that remains when the cleaved peptide regions have been cleaved from the immature peptide." [EBIBS:GAR, http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000433', 'non_LTR_retrotransposon_polymeric_tract', 'A polymeric tract, such as poly(dA), within a non_LTR_retrotransposon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000434', 'target_site_duplication', 'A sequence of the target DNA that is duplicated when a transposable element or phage inserts; usually found at each end the insertion." [http://www.koko.gov.my/CocoaBioTech/Glossaryt.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000435', 'RR_tract', 'A polypurine tract within an LTR_retrotransposon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000436', 'ARS', 'A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000437', 'assortment_derived_duplication', 'A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000438', 'gene_not_polyadenylated', 'A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000439', 'inverted_ring_chromosome', 'A sequence that can autonomously replicate, as a plasmid, when transformed into a bacterial host." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000440', 'vector_replicon', 'A replicon that has been modified to act as a vector for foreign sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000441', 'ss_oligo', 'A single stranded oligonucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000442', 'ds_oligo', 'A double stranded oligonucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000443', 'polymer_attribute', 'An attribute to describe the kind of biological sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000444', 'three_prime_noncoding_exon', 'Non-coding exon in the 3 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000445', 'five_prime_noncoding_exon', 'Non-coding exon in the 5 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000446', 'UTR_intron', 'Intron located in the untranslated region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000447', 'five_prime_UTR_intron', 'An intron located in the 5 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000448', 'three_prime_UTR_intron', 'An intron located in the 3 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000449', 'random_sequence', 'A sequence of nucleotides or amino acids which, by design, has a \"random\" order of components, given a predetermined input frequency of these components." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000450', 'interband', 'A light region between two darkly staining bands in a polytene chromosome." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000451', 'gene_with_polyadenylated_mRNA', 'A gene that encodes a polyadenylated mRNA." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000452', 'transgene_attribute', 'A gene that encodes a polyadenylated mRNA." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000453', 'chromosomal_transposition', 'A chromosome structure variant whereby a region of a chromosome has been transferred to another position. Among interchromosomal rearrangements, the term transposition is reserved for that class in which the telomeres of the chromosomes involved are coupled (that is to say, form the two ends of a single DNA molecule) as in wild-type." [FB:reference_manual, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000454', 'rasiRNA', 'A 17-28-nt, small interfering RNA derived from transcripts of repetitive elements." [http://www.developmentalcell.com/content/article/abstract?uid=PIIS1534580703002284]');
INSERT INTO soterms (id, name, definition) VALUES ('000455', 'gene_with_mRNA_with_frameshift', 'A gene that encodes an mRNA with a frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000456', 'recombinationally_rearranged_gene', 'A gene that is recombinationally rearranged." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000457', 'interchromosomal_duplication', 'A chromosome duplication involving an insertion from another chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000458', 'D_gene_segment', 'Germline genomic DNA including D-region with 5 UTR and 3 UTR, also designated as D-segment." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000459', 'gene_with_trans_spliced_transcript', 'A gene with a transcript that is trans-spliced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000460', 'vertebrate_immunoglobulin_T_cell_receptor_segment', 'A gene with a transcript that is trans-spliced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000461', 'inversion_derived_bipartite_deficiency', 'A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at each end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000462', 'pseudogenic_region', 'A non-functional descendant of a functional entity." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000463', 'encodes_alternately_spliced_transcripts', 'A gene that encodes more than one transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000464', 'decayed_exon', 'A non-functional descendant of an exon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000465', 'inversion_derived_deficiency_plus_duplication', 'A chromosome deletion whereby a chromosome is generated by recombination between two inversions; there is a deficiency at one end of the inversion and a duplication at the other end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000466', 'V_gene_segment', 'Germline genomic DNA including L-part1, V-intron and V-exon, with the 5 UTR and 3 UTR." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000467', 'post_translationally_regulated_by_protein_stability', 'An attribute describing a gene sequence where the resulting protein is regulated by the stability of the resulting protein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000468', 'golden_path_fragment', 'One of the pieces of sequence that make up a golden path." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000469', 'post_translationally_regulated_by_protein_modification', 'An attribute describing a gene sequence where the resulting protein is modified to regulate it." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000470', 'J_gene_segment', 'Germline genomic DNA of an immunoglobulin/T-cell receptor gene including J-region with 5 UTR (SO:0000204) and 3 UTR (SO:0000205), also designated as J-segment." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000471', 'autoregulated', 'The gene product is involved in its own transcriptional regulation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000472', 'tiling_path', 'A set of regions which overlap with minimal polymorphism to form a linear sequence." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000473', 'negatively_autoregulated', 'The gene product is involved in its own transcriptional regulation where it decreases transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000474', 'tiling_path_fragment', 'A piece of sequence that makes up a tiling_path (SO:0000472)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000475', 'positively_autoregulated', 'The gene product is involved in its own transcriptional regulation, where it increases transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000476', 'contig_read', 'A DNA sequencer read which is part of a contig." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000477', 'polycistronic_gene', 'A gene that is polycistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000478', 'C_gene_segment', 'Genomic DNA of immunoglobulin/T-cell receptor gene including C-region (and introns if present) with 5 UTR (SO:0000204) and 3 UTR (SO:0000205)." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000479', 'trans_spliced_transcript', 'A transcript that is trans-spliced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000480', 'tiling_path_clone', 'A clone which is part of a tiling path. A tiling path is a set of sequencing substrates, typically clones, which have been selected in order to efficiently cover a region of the genome in preparation for sequencing and assembly." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000481', 'terminal_inverted_repeat', 'An inverted repeat (SO:0000294) occurring at the termini of a DNA transposon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000482', 'vertebrate_immunoglobulin_T_cell_receptor_gene_cluster', 'An inverted repeat (SO:0000294) occurring at the termini of a DNA transposon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000483', 'nc_primary_transcript', 'A primary transcript that is never translated into a protein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000484', 'three_prime_coding_exon_noncoding_region', 'The sequence of the 3 exon that is not coding." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000485', 'DJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene, and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000486', 'five_prime_coding_exon_noncoding_region', 'The sequence of the 5 exon preceding the start codon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000487', 'VDJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000488', 'VDJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000489', 'VJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000490', 'VJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000491', 'VJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000492', 'D_gene_recombination_feature', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000493', 'three_prime_D_heptamer', '7 nucleotide recombination site like CACAGTG, part of a 3 D-recombination signal sequence of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000494', 'three_prime_D_nonamer', 'A 9 nucleotide recombination site (e.g. ACAAAAACC), part of a 3 D-recombination signal sequence of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000495', 'three_prime_D_spacer', 'A 12 or 23 nucleotide spacer between the 3D-HEPTAMER and 3D-NONAMER of a 3D-RS." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000496', 'five_prime_D_heptamer', '7 nucleotide recombination site (e.g. CACTGTG), part of a 5 D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000497', 'five_prime_D_nonamer', '9 nucleotide recombination site (e.g. GGTTTTTGT), part of a five_prime_D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000498', 'five_prime_D_spacer', '12 or 23 nucleotide spacer between the 5 D-heptamer (SO:0000496) and 5 D-nonamer (SO:0000497) of a 5 D-recombination signal sequence (SO:0000556) of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000499', 'virtual_sequence', 'A continuous piece of sequence similar to the virtual contig concept of the Ensembl database." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000500', 'Hoogsteen_base_pair', 'A type of non-canonical base-pairing. This is less energetically favourable than watson crick base pairing. Hoogsteen GC base pairs only have two hydrogen bonds." [PMID:12177293]');
INSERT INTO soterms (id, name, definition) VALUES ('000501', 'reverse_Hoogsteen_base_pair', 'A type of non-canonical base-pairing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000502', 'transcribed_region', 'A region of sequence that is transcribed. This region may cover the transcript of a gene, it may emcompas the sequence covered by all of the transcripts of a alternately spliced gene, or it may cover the region transcribed by a polycistronic transcript. A gene may have 1 or more transcribed regions and a transcribed_region may belong to one or more genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000503', 'alternately_spliced_gene_encodeing_one_transcript', 'A region of sequence that is transcribed. This region may cover the transcript of a gene, it may emcompas the sequence covered by all of the transcripts of a alternately spliced gene, or it may cover the region transcribed by a polycistronic transcript. A gene may have 1 or more transcribed regions and a transcribed_region may belong to one or more genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000504', 'D_DJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000505', 'D_DJ_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene and one DJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000506', 'D_DJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000507', 'pseudogenic_exon', 'A non functional descendant of an exon, part of a pseudogene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000508', 'D_DJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one D-gene, one DJ-gene, and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000509', 'D_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000510', 'VD_gene_segment', 'Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including L-part1, V-intron and V-D-exon, with the 5 UTR (SO:0000204) and 3 UTR (SO:0000205)." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000511', 'J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000512', 'inversion_derived_deficiency_plus_aneuploid', 'A chromosomal deletion whereby a chromosome generated by recombination between two inversions; has a deficiency at one end and presumed to have a deficiency or duplication at the other end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000513', 'J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000514', 'J_nonamer', '9 nucleotide recombination site (e.g. GGTTTTTGT), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000515', 'J_heptamer', '7 nucleotide recombination site (e.g. CACAGTG), part of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000516', 'pseudogenic_transcript', 'A non functional descendant of a transcript, part of a pseudogene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000517', 'J_spacer', '12 or 23 nucleotide spacer between the J-nonamer and the J-heptamer of a J-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000518', 'V_DJ_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one DJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000519', 'V_DJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000520', 'V_VDJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000521', 'V_VDJ_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VDJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000522', 'V_VDJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000523', 'V_VJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000524', 'V_VJ_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene and one VJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000525', 'V_VJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000526', 'V_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one V-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000527', 'V_D_DJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000528', 'V_D_DJ_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000529', 'V_D_DJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000530', 'V_D_DJ_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one D-gene, one DJ-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000531', 'V_D_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000532', 'V_D_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one D-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000533', 'V_heptamer', '7 nucleotide recombination site (e.g. CACAGTG), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000534', 'V_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000535', 'V_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one V-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000536', 'V_nonamer', '9 nucleotide recombination site (e.g. ACAAAAACC), part of V-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000537', 'V_spacer', '12 or 23 nucleotide spacer between the V-heptamer and the V-nonamer of a V-gene recombination feature of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000538', 'V_gene_recombination_feature', 'Recombination signal including V-heptamer, V-spacer and V-nonamer in 3 of V-region of a V-gene or V-sequence of an immunoglobulin/T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000539', 'DJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one DJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000540', 'DJ_J_C_cluster', 'Genomic DNA in rearranged configuration including at least one D-J-GENE, one J-GENE and one C-GENE." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000541', 'VDJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one VDJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000542', 'V_DJ_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000543', 'alternately_spliced_gene_encoding_greater_than_one_transcript', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000544', 'helitron', 'A rolling circle transposon. Autonomous helitrons encode a 5-to-3 DNA helicase and nuclease/ligase similar to those encoded by known rolling-circle replicons." [http://www.pnas.org/cgi/content/full/100/11/6569]');
INSERT INTO soterms (id, name, definition) VALUES ('000545', 'recoding_pseudoknot', 'The pseudoknots involved in recoding are unique in that, as they play their role as a structure, they are immediately unfolded and their now linear sequence serves as a template for decoding." [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=33937]');
INSERT INTO soterms (id, name, definition) VALUES ('000546', 'designed_sequence', 'The pseudoknots involved in recoding are unique in that, as they play their role as a structure, they are immediately unfolded and their now linear sequence serves as a template for decoding." [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=33937]');
INSERT INTO soterms (id, name, definition) VALUES ('000547', 'inversion_derived_bipartite_duplication', 'A chromosome generated by recombination between two inversions; there is a duplication at each end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000548', 'gene_with_edited_transcript', 'A gene that encodes a transcript that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000549', 'inversion_derived_duplication_plus_aneuploid', 'A chromosome generated by recombination between two inversions; has a duplication at one end and presumed to have a deficiency or duplication at the other end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000550', 'aneuploid_chromosome', 'A chromosome structural variation whereby either a chromosome exists in addition to the normal chromosome complement or is lacking." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000551', 'polyA_signal_sequence', 'The recognition sequence necessary for endonuclease cleavage of an RNA transcript that is followed by polyadenylation; consensus=AATAAA." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000552', 'Shine_Dalgarno_sequence', 'A region in the 5 UTR that pairs with the 16S rRNA during formation of the preinitiation complex." [SO:jh]');
INSERT INTO soterms (id, name, definition) VALUES ('000553', 'polyA_site', 'The site on an RNA transcript to which will be added adenine residues by post-transcriptional polyadenylation. The boundary between the UTR and the polyA sequence." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000554', 'assortment_derived_deficiency_plus_duplication', 'The site on an RNA transcript to which will be added adenine residues by post-transcriptional polyadenylation. The boundary between the UTR and the polyA sequence." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000555', 'five_prime_clip', '5 most region of a precursor transcript that is clipped off during processing." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000556', 'five_prime_D_recombination_signal_sequence', 'Recombination signal of an immunoglobulin/T-cell receptor gene, including the 5 D-nonamer (SO:0000497), 5 D-spacer (SO:0000498), and 5 D-heptamer (SO:0000396) in 5 of the D-region of a D-gene, or in 5 of the D-region of DJ-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000557', 'three_prime_clip', '3-most region of a precursor transcript that is clipped off during processing." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000558', 'C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene including more than one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000559', 'D_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including more than one D-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000560', 'D_J_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in germline configuration including at least one D-gene and one J-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000561', 'heptamer_of_recombination_feature_of_vertebrate_immune_system_gene', 'Seven nucleotide recombination site (e.g. CACAGTG), part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000562', 'nonamer_of_recombination_feature_of_vertebrate_immune_system_gene', 'Seven nucleotide recombination site (e.g. CACAGTG), part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000563', 'vertebrate_immune_system_gene_recombination_spacer', 'Seven nucleotide recombination site (e.g. CACAGTG), part of V-gene, D-gene or J-gene recombination feature of an immunoglobulin or T-cell receptor gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000564', 'V_DJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one DJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000565', 'V_VDJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VDJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000566', 'V_VJ_J_C_cluster', 'Genomic DNA of immunoglobulin/T-cell receptor gene in rearranged configuration including at least one V-gene, one VJ-gene, one J-gene and one C-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000567', 'inversion_derived_aneuploid_chromosome', 'A chromosome may be generated by recombination between two inversions; presumed to have a deficiency or duplication at each end of the inversion." [FB:km]');
INSERT INTO soterms (id, name, definition) VALUES ('000568', 'bidirectional_promoter', 'An unregulated promoter that allows continuous expression." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000569', 'retrotransposed', 'An attribute of a feature that occurred as the product of a reverse transcriptase mediated event." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000570', 'three_prime_D_recombination_signal_sequence', 'Recombination signal of an immunoglobulin/T-cell receptor gene, including the 3 D-heptamer (SO:0000493), 3 D-spacer, and 3 D-nonamer (SO:0000494) in 3 of the D-region of a D-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000571', 'miRNA_encoding', 'Recombination signal of an immunoglobulin/T-cell receptor gene, including the 3 D-heptamer (SO:0000493), 3 D-spacer, and 3 D-nonamer (SO:0000494) in 3 of the D-region of a D-gene." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000572', 'DJ_gene_segment', 'Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including D-J-region with 5 UTR and 3 UTR, also designated as D-J-segment." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000573', 'rRNA_encoding', 'Genomic DNA of immunoglobulin/T-cell receptor gene in partially rearranged genomic DNA including D-J-region with 5 UTR and 3 UTR, also designated as D-J-segment." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000574', 'VDJ_gene_segment', 'Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-D-J-exon, with the 5UTR (SO:0000204) and 3UTR (SO:0000205)." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000575', 'scRNA_encoding', 'Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-D-J-exon, with the 5UTR (SO:0000204) and 3UTR (SO:0000205)." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000576', 'VJ_gene_segment', 'Rearranged genomic DNA of immunoglobulin/T-cell receptor gene including L-part1, V-intron and V-J-exon, with the 5UTR (SO:0000204) and 3UTR (SO:0000205)." [http://www.imgt.org/cgi-bin/IMGTlect.jv?query=7#]');
INSERT INTO soterms (id, name, definition) VALUES ('000577', 'centromere', 'A region of chromosome where the spindle fibers attach during mitosis and meiosis." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000578', 'snoRNA_encoding', 'A region of chromosome where the spindle fibers attach during mitosis and meiosis." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000579', 'edited_transcript_feature', 'A locatable feature on a transcript that is edited." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000580', 'methylation_guide_snoRNA_primary_transcript', 'A primary transcript encoding a methylation guide small nucleolar RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000581', 'cap', 'A structure consisting of a 7-methylguanosine in 5-5 triphosphate linkage with the first nucleotide of an mRNA. It is added post-transcriptionally, and is not encoded in the DNA." [http://seqcore.brcf.med.umich.edu/doc/educ/dnapr/mbglossary/mbgloss.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000582', 'rRNA_cleavage_snoRNA_primary_transcript', 'A primary transcript encoding an rRNA cleavage snoRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000583', 'pre_edited_region', 'The region of a transcript that will be edited." [http://dna.kdna.ucla.edu/rna/index.aspx]');
INSERT INTO soterms (id, name, definition) VALUES ('000584', 'tmRNA', 'A tmRNA liberates a mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNAs genes have been found encoded in two parts. TmRNAs have been identified in eubacteria and some chloroplasts but are absent from archeal and Eukaryote nuclear genomes." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00023]');
INSERT INTO soterms (id, name, definition) VALUES ('000585', 'C_D_box_snoRNA_encoding', 'A tmRNA liberates a mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNAs genes have been found encoded in two parts. TmRNAs have been identified in eubacteria and some chloroplasts but are absent from archeal and Eukaryote nuclear genomes." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00023]');
INSERT INTO soterms (id, name, definition) VALUES ('000586', 'tmRNA_primary_transcript', 'A primary transcript encoding a tmRNA (SO:0000584)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000587', 'group_I_intron', 'Group I catalytic introns are large self-splicing ribozymes. They catalyze their own excision from mRNA, tRNA and rRNA precursors in a wide range of organisms. The core secondary structure consists of 9 paired regions (P1-P9). These fold to essentially two domains, the P4-P6 domain (formed from the stacking of P5, P4, P6 and P6a helices) and the P3-P9 domain (formed from the P8, P3, P7 and P9 helices). Group I catalytic introns often have long ORFs inserted in loop regions." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00028]');
INSERT INTO soterms (id, name, definition) VALUES ('000588', 'autocatalytically_spliced_intron', 'A self spliced intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000589', 'SRP_RNA_primary_transcript', 'A primary transcript encoding a signal recognition particle RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000590', 'SRP_RNA', 'The signal recognition particle (SRP) is a universally conserved ribonucleoprotein. It is involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most eubacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Eukaryotic and archaeal 7S RNAs have very similar secondary structures, with eight helical elements. These fold into the Alu and S domains, separated by a long linker region. Eubacterial SRP is generally a simpler structure, with the M domain of Ffh bound to a region of the 4.5S RNA that corresponds to helix 8 of the eukaryotic and archaeal SRP S domain. Some Gram-positive bacteria (e.g. Bacillus subtilis), however, have a larger SRP RNA that also has an Alu domain. The Alu domain is thought to mediate the peptide chain elongation retardation function of the SRP. The universally conserved helix which interacts with the SRP54/Ffh M domain mediates signal sequence recognition. In eukaryotes and archaea, the SRP19-helix 6 complex is thought to be involved in SRP assembly and stabilizes helix 8 for SRP54 binding." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00017]');
INSERT INTO soterms (id, name, definition) VALUES ('000591', 'pseudoknot', 'A tertiary structure in RNA where nucleotides in a loop form base pairs with a region of RNA downstream of the loop." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000592', 'H_pseudoknot', 'A pseudoknot which contains two stems and at least two loops." [http://www.ncbi.nlm.nih.gov\:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=10334330&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000593', 'C_D_box_snoRNA', 'Most box C/D snoRNAs also contain long (>10 nt) sequences complementary to rRNA. Boxes C and D, as well as boxes C and D, are usually located in close proximity, and form a structure known as the box C/D motif. This motif is important for snoRNA stability, processing, nucleolar targeting and function. A small number of box C/D snoRNAs are involved in rRNA processing; most, however, are known or predicted to serve as guide RNAs in ribose methylation of rRNA. Targeting involves direct base pairing of the snoRNA at the rRNA site to be modified and selection of a rRNA nucleotide a fixed distance from box D or D." [http://www.bio.umass.edu/biochem/rna-sequence/Yeast_snoRNA_Database/snoRNA_DataBase.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000594', 'H_ACA_box_snoRNA', 'Members of the box H/ACA family contain an ACA triplet, exactly 3 nt upstream from the 3 end and an H-box in a hinge region that links two structurally similar functional domains of the molecule. Both boxes are important for snoRNA biosynthesis and function. A few box H/ACA snoRNAs are involved in rRNA processing; most others are known or predicted to participate in selection of uridine nucleosides in rRNA to be converted to pseudouridines. Site selection is mediated by direct base pairing of the snoRNA with rRNA through one or both targeting domains." [http://www.bio.umass.edu/biochem/rna-sequence/Yeast_snoRNA_Database/snoRNA_DataBase.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000595', 'C_D_box_snoRNA_primary_transcript', 'A primary transcript encoding a small nucleolar RNA of the box C/D family." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000596', 'H_ACA_box_snoRNA_primary_transcript', 'A primary transcript encoding a small nucleolar RNA of the box H/ACA family." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000597', 'transcript_edited_by_U_insertion/deletion', 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000598', 'edited_by_C_insertion_and_dinucleotide_insertion', 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000599', 'edited_by_C_to_U_substitution', 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000600', 'edited_by_A_to_I_substitution', 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000601', 'edited_by_G_addition', 'The insertion and deletion of uridine (U) residues, usually within coding regions of mRNA transcripts of cryptogenes in the mitochondrial genome of kinetoplastid protozoa." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000602', 'guide_RNA', 'A short 3-uridylated RNA that can form a duplex (except for its post-transcriptionally added oligo_U tail (SO:0000609)) with a stretch of mature edited mRNA." [http://www.rna.ucla.edu/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000603', 'group_II_intron', 'Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI). A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2 hydroxyl of a bulged adenosine in domain VI attacks the 5 splice site, followed by nucleophilic attack on the 3 splice site by the 3 OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron to intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3 splice site, some tertiary interactions, and intronic ORF phylogeny." [http://www.sanger.ac.uk/Software/Rfam/browse/index.shtml]');
INSERT INTO soterms (id, name, definition) VALUES ('000604', 'editing_block', 'Edited mRNA sequence mediated by a single guide RNA (SO:0000602)." [http://dna.kdna.ucla.edu/rna/index.aspx]');
INSERT INTO soterms (id, name, definition) VALUES ('000605', 'intergenic_region', 'A region containing or overlapping no genes that is bounded on either side by a gene, or bounded by a gene and the end of the chromosome." [SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('000606', 'editing_domain', 'Edited mRNA sequence mediated by two or more overlapping guide RNAs (SO:0000602)." [http://dna.kdna.ucla.edu/rna/index.aspx]');
INSERT INTO soterms (id, name, definition) VALUES ('000607', 'unedited_region', 'The region of an edited transcript that will not be edited." [http://dna.kdna.ucla.edu/rna/index.aspx]');
INSERT INTO soterms (id, name, definition) VALUES ('000608', 'H_ACA_box_snoRNA_encoding', 'The region of an edited transcript that will not be edited." [http://dna.kdna.ucla.edu/rna/index.aspx]');
INSERT INTO soterms (id, name, definition) VALUES ('000609', 'oligo_U_tail', 'The string of non-encoded Us at the 3 end of a guide RNA (SO:0000602)." [http://www.rna.ucla.edu/]');
INSERT INTO soterms (id, name, definition) VALUES ('000610', 'polyA_sequence', 'Sequence of about 100 nucleotides of A added to the 3 end of most eukaryotic mRNAs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000611', 'branch_site', 'A pyrimidine rich sequence near the 3 end of an intron to which the 5end becomes covalently bound during nuclear splicing. The resulting structure resembles a lariat." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000612', 'polypyrimidine_tract', 'The polypyrimidine tract is one of the cis-acting sequence elements directing intron removal in pre-mRNA splicing." [http://nar.oupjournals.org/cgi/content/full/25/4/888]');
INSERT INTO soterms (id, name, definition) VALUES ('000613', 'bacterial_RNApol_promoter', 'A DNA sequence to which bacterial RNA polymerase binds, to begin transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000614', 'bacterial_terminator', 'A terminator signal for bacterial transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000615', 'terminator_of_type_2_RNApol_III_promoter', 'A terminator signal for RNA polymerase III transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000616', 'transcription_end_site', 'The base where transcription ends." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000617', 'RNApol_III_promoter_type_1', 'The base where transcription ends." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000618', 'RNApol_III_promoter_type_2', 'The base where transcription ends." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000619', 'A_box', 'A variably distant linear promoter region recognized by TFIIIC, with consensus sequence TGGCnnAGTGG." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000620', 'B_box', 'A variably distant linear promoter region recognized by TFIIIC, with consensus sequence AGGTTCCAnnCC." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000621', 'RNApol_III_promoter_type_3', 'A variably distant linear promoter region recognized by TFIIIC, with consensus sequence AGGTTCCAnnCC." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000622', 'C_box', 'An RNA polymerase III type 1 promoter with consensus sequence CAnnCCn." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000623', 'snRNA_encoding', 'An RNA polymerase III type 1 promoter with consensus sequence CAnnCCn." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000624', 'telomere', 'A specific structure at the end of a linear chromosome, required for the integrity and maintenance of the end." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000625', 'silencer', 'A regulatory region which upon binding of transcription factors, suppress the transcription of the gene or genes they control." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000626', 'chromosomal_regulatory_element', 'A regulatory region which upon binding of transcription factors, suppress the transcription of the gene or genes they control." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000627', 'insulator', 'A transcriptional cis regulatory region that when located between a CM and a genes promoter prevents the CRM from modulating that genes expression." [SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('000628', 'chromosomal_structural_element', 'A transcriptional cis regulatory region that when located between a CM and a genes promoter prevents the CRM from modulating that genes expression." [SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('000629', 'five_prime_open_reading_frame', 'A transcriptional cis regulatory region that when located between a CM and a genes promoter prevents the CRM from modulating that genes expression." [SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('000630', 'upstream_AUG_codon', 'A start codon upstream of the ORF." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000631', 'polycistronic_primary_transcript', 'A primary transcript encoding for more than one gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000632', 'monocistronic_primary_transcript', 'A primary transcript encoding for one gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000633', 'monocistronic_mRNA', 'An mRNA with either a single protein product, or for which the regions encoding all its protein products overlap." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000634', 'polycistronic_mRNA', 'An mRNA that encodes multiple proteins from at least two non-overlapping regions." [SO:rd]');
INSERT INTO soterms (id, name, definition) VALUES ('000635', 'mini_exon_donor_RNA', 'A primary transcript that donates the spliced leader to other mRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000636', 'spliced_leader_RNA', 'A primary transcript that donates the spliced leader to other mRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000637', 'engineered_plasmid', 'A plasmid that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000638', 'transcribed_spacer_region', 'Part of an rRNA transcription unit that is transcribed but discarded during maturation, not giving rise to any part of rRNA." [http://oregonstate.edu/instruction/bb492/general/glossary.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000639', 'internal_transcribed_spacer_region', 'Non-coding regions of DNA sequence that separate genes coding for the 28S, 5.8S, and 18S ribosomal RNAs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000640', 'external_transcribed_spacer_region', 'Non-coding regions of DNA that precede the sequence that codes for the ribosomal RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000641', 'tetranucleotide_repeat_microsatellite_feature', 'Non-coding regions of DNA that precede the sequence that codes for the ribosomal RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000642', 'SRP_RNA_encoding', 'Non-coding regions of DNA that precede the sequence that codes for the ribosomal RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000643', 'minisatellite', 'A repeat region containing tandemly repeated sequences having a unit length of 10 to 40 bp." [http://www.informatics.jax.org/silver/glossary.shtml]');
INSERT INTO soterms (id, name, definition) VALUES ('000644', 'antisense_RNA', 'Antisense RNA is RNA that is transcribed from the coding, rather than the template, strand of DNA. It is therefore complementary to mRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000645', 'antisense_primary_transcript', 'The reverse complement of the primary transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000646', 'siRNA', 'A small RNA molecule that is the product of a longer exogenous or endogenous dsRNA, which is either a bimolecular duplex or very long hairpin, processed (via the Dicer pathway) such that numerous siRNAs accumulate from both strands of the dsRNA. SRNAs trigger the cleavage of their target molecules." [PMID:12592000]');
INSERT INTO soterms (id, name, definition) VALUES ('000647', 'miRNA_primary_transcript', 'A primary transcript encoding a micro RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000650', 'small_subunit_rRNA', 'Ribosomal RNA transcript that structures the small subunit of the ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000651', 'large_subunit_rRNA', 'Ribosomal RNA transcript that structures the large subunit of the ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000652', 'rRNA_5S', '5S ribosomal RNA (5S rRNA) is a component of the large ribosomal subunit in both prokaryotes and eukaryotes. In eukaryotes, it is synthesised by RNA polymerase III (the other eukaryotic rRNAs are cleaved from a 45S precursor synthesised by RNA polymerase I). In Xenopus oocytes, it has been shown that fingers 4-7 of the nine-zinc finger transcription factor TFIIIA can bind to the central region of 5S RNA. Thus, in addition to positively regulating 5S rRNA transcription, TFIIIA also stabilizes 5S rRNA until it is required for transcription." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00001]');
INSERT INTO soterms (id, name, definition) VALUES ('000653', 'rRNA_28S', 'A component of the large ribosomal subunit." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000654', 'maxicircle_gene', 'A mitochondrial gene located in a maxicircle." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000655', 'ncRNA', 'An RNA transcript that does not encode for a protein rather the RNA molecule is the gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000656', 'stRNA_encoding', 'An RNA transcript that does not encode for a protein rather the RNA molecule is the gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000657', 'repeat_region', 'A region of sequence containing one or more repeat units." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000658', 'dispersed_repeat', 'A repeat that is located at dispersed sites in the genome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000659', 'tmRNA_encoding', 'A repeat that is located at dispersed sites in the genome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000660', 'DNA_invertase_target_sequence', 'A repeat that is located at dispersed sites in the genome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000661', 'intron_attribute', 'A repeat that is located at dispersed sites in the genome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000662', 'spliceosomal_intron', 'An intron which is spliced by the spliceosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000663', 'tRNA_encoding', 'An intron which is spliced by the spliceosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000664', 'introgressed_chromosome_region', 'An intron which is spliced by the spliceosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000665', 'monocistronic_transcript', 'A transcript that is monocistronic." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000666', 'mobile_intron', 'An intron (mitochondrial, chloroplast, nuclear or prokaryotic) that encodes a double strand sequence specific endonuclease allowing for mobility." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000667', 'insertion', 'The sequence of one or more nucleotides added between two adjacent nucleotides in the sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000668', 'EST_match', 'A match against an EST sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000669', 'sequence_rearrangement_feature', 'A match against an EST sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000670', 'chromosome_breakage_sequence', 'A sequence within the micronuclear DNA of ciliates at which chromosome breakage and telomere addition occurs during nuclear differentiation." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000671', 'internal_eliminated_sequence', 'A sequence eliminated from the genome of ciliates during nuclear differentiation." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000672', 'macronucleus_destined_segment', 'A sequence that is conserved, although rearranged relative to the micronucleus, in the macronucleus of a ciliate genome." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000673', 'transcript', 'An RNA synthesized on a DNA or RNA template by an RNA polymerase." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000674', 'non_canonical_splice_site', 'A splice site where the donor and acceptor sites differ from the canonical form." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000675', 'canonical_splice_site', 'The major class of splice site with dinucleotides GT and AG for donor and acceptor sites, respectively." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000676', 'canonical_three_prime_splice_site', 'The canonical 3 splice site has the sequence \"AG\"." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000677', 'canonical_five_prime_splice_site', 'The canonical 5 splice site has the sequence \"GT\"." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000678', 'non_canonical_three_prime_splice_site', 'A 3 splice site that does not have the sequence \"AG\"." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000679', 'non_canonical_five_prime_splice_site', 'A 5 splice site which does not have the sequence \"GT\"." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000680', 'non_canonical_start_codon', 'A start codon that is not the usual AUG sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000681', 'aberrant_processed_transcript', 'A transcript that has been processed \"incorrectly\", for example by the failure of splicing of one or more exons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000682', 'splicing_feature', 'A transcript that has been processed \"incorrectly\", for example by the failure of splicing of one or more exons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000683', 'exonic_splice_enhancer', 'Exonic splicing enhancers (ESEs) facilitate exon definition by assisting in the recruitment of splicing factors to the adjacent intron." [http://www.ncbi.nlm.nih.gov\:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12403462&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('000684', 'nuclease_sensitive_site', 'A region of nucleotide sequence targeted by a nuclease enzyme." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000685', 'DNAseI_hypersensitive_site', 'A region of nucleotide sequence targeted by a nuclease enzyme." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000686', 'translocation_element', 'A chromosomal translocation whereby the chromosomes carrying non-homologous centromeres may be recovered independently. These chromosomes are described as translocation elements. This occurs for some translocations, particularly but not exclusively, reciprocal translocations." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000687', 'deletion_junction', 'The space between two bases in a sequence which marks the position where a deletion has occurred." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000688', 'golden_path', 'A set of subregions selected from sequence contigs which when concatenated form a nonredundant linear sequence." [SO:ls]');
INSERT INTO soterms (id, name, definition) VALUES ('000689', 'cDNA_match', 'A match against cDNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000690', 'gene_with_polycistronic_transcript', 'A gene that encodes a polycistronic transcript." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000691', 'cleaved_initiator_methionine', 'The initiator methionine that has been cleaved from a mature polypeptide sequence." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('000692', 'gene_with_dicistronic_transcript', 'A gene that encodes a dicistronic transcript." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000693', 'gene_with_recoded_mRNA', 'A gene that encodes an mRNA that is recoded." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000694', 'SNP', 'SNPs are single base pair positions in genomic DNA at which different sequence alternatives exist in normal individuals in some population(s), wherein the least frequent variant has an abundance of 1% or greater." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000695', 'reagent', 'A sequence used in experiment." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000696', 'oligo', 'A short oligonucleotide sequence, of length on the order of 10s of bases; either single or double stranded." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000697', 'gene_with_stop_codon_read_through', 'A gene that encodes a transcript with stop codon readthrough." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000698', 'gene_with_stop_codon_redefined_as_pyrrolysine', 'A gene encoding an mRNA that has the stop codon redefined as pyrrolysine." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000699', 'junction', 'A sequence_feature with an extent of zero." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000700', 'remark', 'A comment about the sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000701', 'possible_base_call_error', 'A region of sequence where the validity of the base calling is questionable." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000702', 'possible_assembly_error', 'A region of sequence where there may have been an error in the assembly." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000703', 'experimental_result_region', 'A region of sequence implicated in an experimental result." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000704', 'gene', 'A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene may include regulatory regions, transcribed regions and/or other functional sequence regions." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('000705', 'tandem_repeat', 'Two or more adjacent copies of a region (of length greater than 1)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000706', 'trans_splice_acceptor_site', 'The 3 splice site of the acceptor primary transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000707', 'trans_splice_donor_site', 'The 5 five prime splice site region of the donor RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000708', 'SL1_acceptor_site', 'A trans_splicing_acceptor_site which appends the 22nt SL1 RNA leader sequence to the 5 end of most mRNAs." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('000709', 'SL2_acceptor_site', 'A trans_splicing_acceptor_site which appends the 22nt SL2 RNA leader sequence to the 5 end of mRNAs. SL2 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('000710', 'gene_with_stop_codon_redefined_as_selenocysteine', 'A gene encoding an mRNA that has the stop codon redefined as selenocysteine." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000711', 'gene_with_mRNA_recoded_by_translational_bypass', 'A gene with mRNA recoded by translational bypass." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000712', 'gene_with_transcript_with_translational_frameshift', 'A gene encoding a transcript that has a translational frameshift." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000713', 'DNA_motif', 'A motif that is active in the DNA form of the sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000714', 'nucleotide_motif', 'A region of nucleotide sequence corresponding to a known motif." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000715', 'RNA_motif', 'A motif that is active in RNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000716', 'dicistronic_mRNA', 'An mRNA that has the quality dicistronic." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000717', 'reading_frame', 'A nucleic acid sequence that when read as sequential triplets, has the potential of encoding a sequential string of amino acids. It need not contain the start or stop codon." [SGD:rb]');
INSERT INTO soterms (id, name, definition) VALUES ('000718', 'blocked_reading_frame', 'A reading_frame that is interrupted by one or more stop codons; usually identified through inter-genomic sequence comparisons." [SGD:rb]');
INSERT INTO soterms (id, name, definition) VALUES ('000719', 'ultracontig', 'An ordered and oriented set of scaffolds based on somewhat weaker sets of inferential evidence such as one set of mate pair reads together with supporting evidence from ESTs or location of markers from SNP or microsatellite maps, or cytogenetic localization of contained markers." [FB:WG]');
INSERT INTO soterms (id, name, definition) VALUES ('000720', 'foreign_transposable_element', 'A transposable element that is foreign." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000721', 'gene_with_dicistronic_primary_transcript', 'A gene that encodes a dicistronic primary transcript." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000722', 'gene_with_dicistronic_mRNA', 'A gene that encodes a polycistronic mRNA." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000723', 'iDNA', 'Genomic sequence removed from the genome, as a normal event, by a process of recombination." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000724', 'oriT', 'A region of a DNA molecule where transfer is initiated during the process of conjugation or mobilization." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000725', 'transit_peptide', 'The transit_peptide is a short region at the N-terminus of the peptide that directs the protein to an organelle (chloroplast, mitochondrion, microbody or cyanelle)." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000726', 'repeat_unit', 'The simplest repeated component of a repeat region. A single repeat." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000727', 'CRM', 'A regulatory region where transcription factor binding sites clustered to regulate various aspects of transcription activities. (CRMs can be located a few kb to hundred kb upstream of the basal promoter, in the coding sequence, within introns, or in the downstream 3UTR sequences, as well as on different chromosome). A single gene can be regulated by multiple CRMs to give precise control of its spatial and temporal expression. CRMs function as nodes in large, intertwined regulatory network." [PMID:19660565, SO:SG]');
INSERT INTO soterms (id, name, definition) VALUES ('000728', 'intein', 'A region of a peptide that is able to excise itself and rejoin the remaining portions with a peptide bond." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000729', 'intein_containing', 'An attribute of protein-coding genes where the initial protein product contains an intein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000730', 'gap', 'A gap in the sequence of known length. The unknown bases are filled in with Ns." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000731', 'fragmentary', 'An attribute to describe a feature that is incomplete." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000732', 'predicted', 'An attribute describing an unverified region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000733', 'feature_attribute', 'An attribute describing a located_sequence_feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000734', 'exemplar_mRNA', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000735', 'sequence_location', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000736', 'organelle_sequence', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000737', 'mitochondrial_sequence', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000738', 'nuclear_sequence', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000739', 'nucleomorphic_sequence', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000740', 'plastid_sequence', 'An exemplar is a representative cDNA sequence for each gene. The exemplar approach is a method that usually involves some initial clustering into gene groups and the subsequent selection of a representative from each gene group." [http://mged.sourceforge.net/ontologies/MGEDontology.php]');
INSERT INTO soterms (id, name, definition) VALUES ('000741', 'kinetoplast', 'A kinetoplast is an interlocked network of thousands of minicircles and tens of maxicircles, located near the base of the flagellum of some protozoan species." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000742', 'maxicircle', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000743', 'apicoplast_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000744', 'chromoplast_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000745', 'chloroplast_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000746', 'cyanelle_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000747', 'leucoplast_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000748', 'proplastid_sequence', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000749', 'plasmid_location', 'A maxicircle is a replicon, part of a kinetoplast, that contains open reading frames and replicates via a rolling circle method." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000750', 'amplification_origin', 'An origin_of_replication that is used for the amplification of a chromosomal nucleic acid sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000751', 'proviral_location', 'An origin_of_replication that is used for the amplification of a chromosomal nucleic acid sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000752', 'gene_group_regulatory_region', 'An origin_of_replication that is used for the amplification of a chromosomal nucleic acid sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000753', 'clone_insert', 'The region of sequence that has been inserted and is being propagated by the clone." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000754', 'lambda_vector', 'The lambda bacteriophage is the vector for the linear lambda clone. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome." [ISBN:0-1767-2380-8]');
INSERT INTO soterms (id, name, definition) VALUES ('000755', 'plasmid_vector', 'The lambda bacteriophage is the vector for the linear lambda clone. The genes involved in the lysogenic pathway are removed from the from the viral DNA. Up to 25 kb of foreign DNA can then be inserted into the lambda genome." [ISBN:0-1767-2380-8]');
INSERT INTO soterms (id, name, definition) VALUES ('000756', 'cDNA', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000757', 'single_stranded_cDNA', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000758', 'double_stranded_cDNA', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000759', 'plasmid_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000760', 'YAC_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000761', 'phagemid_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000762', 'PAC_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000763', 'fosmid_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000764', 'BAC_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000765', 'cosmid_clone', 'DNA synthesized by reverse transcriptase using RNA as a template." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000766', 'pyrrolysyl_tRNA', 'A tRNA sequence that has a pyrrolysine anticodon, and a 3 pyrrolysine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000767', 'clone_insert_start', 'A tRNA sequence that has a pyrrolysine anticodon, and a 3 pyrrolysine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000768', 'episome', 'A plasmid that may integrate with a chromosome." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000769', 'tmRNA_coding_piece', 'The region of a two-piece tmRNA that bears the reading frame encoding the proteolysis tag. The tmRNA gene undergoes circular permutation in some groups of bacteria. Processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together." [doi:10.1093/nar/gkh795, Indiana:kw, issn:1362-4962]');
INSERT INTO soterms (id, name, definition) VALUES ('000770', 'tmRNA_acceptor_piece', 'The acceptor region of a two-piece tmRNA that when mature is charged at its 3 end with alanine. The tmRNA gene undergoes circular permutation in some groups of bacteria; processing of the transcripts from such a gene leaves the mature tmRNA in two pieces, base-paired together." [doi:10.1093/nar/gkh795, Indiana:kw]');
INSERT INTO soterms (id, name, definition) VALUES ('000771', 'QTL', 'A quantitative trait locus (QTL) is a polymorphic locus which contains alleles that differentially affect the expression of a continuously distributed phenotypic trait. Usually it is a marker described by statistical association to quantitative variation in the particular phenotypic trait that is thought to be controlled by the cumulative action of alleles at multiple loci." [http://rgd.mcw.edu/tu/qtls/]');
INSERT INTO soterms (id, name, definition) VALUES ('000772', 'genomic_island', 'A genomic island is an integrated mobile genetic element, characterized by size (over 10 Kb). It that has features that suggest a foreign origin. These can include nucleotide distribution (oligonucleotides signature, CG content etc.) that differs from the bulk of the chromosome and/or genes suggesting DNA mobility." [Phigo:at, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000773', 'pathogenic_island', 'Mobile genetic elements that contribute to rapid changes in virulence potential. They are present on the genomes of pathogenic strains but absent from the genomes of non pathogenic members of the same or related species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000774', 'metabolic_island', 'A transmissible element containing genes involved in metabolism, analogous to the pathogenicity islands of gram negative bacteria." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000775', 'adaptive_island', 'An adaptive island is a genomic island that provides an adaptive advantage to the host." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000776', 'symbiosis_island', 'A transmissible element containing genes involved in symbiosis, analogous to the pathogenicity islands of gram negative bacteria." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000777', 'pseudogenic_rRNA', 'A non functional descendant of an rRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000778', 'pseudogenic_tRNA', 'A non functional descendent of a tRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000779', 'engineered_episome', 'An episome that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000780', 'transposable_element_attribute', 'An episome that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000781', 'transgenic', 'Attribute describing sequence that has been integrated with foreign sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000782', 'natural', 'An attribute describing a feature that occurs in nature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000783', 'engineered', 'An attribute to describe a region that was modified in vitro." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000784', 'foreign', 'An attribute to describe a region from another species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000785', 'cloned_region', 'An attribute to describe a region from another species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000786', 'reagent_attribute', 'An attribute to describe a region from another species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000787', 'clone_attribute', 'An attribute to describe a region from another species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000788', 'cloned', 'An attribute to describe a region from another species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000789', 'validated', 'An attribute to describe a feature that has been proven." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000790', 'invalidated', 'An attribute describing a feature that is invalidated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000791', 'cloned_genomic', 'An attribute describing a feature that is invalidated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000792', 'cloned_cDNA', 'An attribute describing a feature that is invalidated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000793', 'engineered_DNA', 'An attribute describing a feature that is invalidated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000794', 'engineered_rescue_region', 'A rescue region that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000795', 'rescue_mini_gene', 'A mini_gene that rescues." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000796', 'transgenic_transposable_element', 'TE that has been modified in vitro, including insertion of DNA derived from a source other than the originating TE." [FB:mc]');
INSERT INTO soterms (id, name, definition) VALUES ('000797', 'natural_transposable_element', 'TE that exists (or existed) in nature." [FB:mc]');
INSERT INTO soterms (id, name, definition) VALUES ('000798', 'engineered_transposable_element', 'TE that has been modified by manipulations in vitro." [FB:mc]');
INSERT INTO soterms (id, name, definition) VALUES ('000799', 'engineered_foreign_transposable_element', 'A transposable_element that is engineered and foreign." [FB:mc]');
INSERT INTO soterms (id, name, definition) VALUES ('000800', 'assortment_derived_duplication', 'A multi-chromosome duplication aberration generated by reassortment of other aberration components." [FB:gm]');
INSERT INTO soterms (id, name, definition) VALUES ('000801', 'assortment_derived_deficiency_plus_duplication', 'A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency and a duplication." [FB:gm]');
INSERT INTO soterms (id, name, definition) VALUES ('000802', 'assortment_derived_deficiency', 'A multi-chromosome deficiency aberration generated by reassortment of other aberration components." [FB:gm]');
INSERT INTO soterms (id, name, definition) VALUES ('000803', 'assortment_derived_aneuploid', 'A multi-chromosome aberration generated by reassortment of other aberration components; presumed to have a deficiency or a duplication." [FB:gm]');
INSERT INTO soterms (id, name, definition) VALUES ('000804', 'engineered_region', 'A region that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000805', 'engineered_foreign_region', 'A region that is engineered and foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000806', 'fusion', 'A region that is engineered and foreign." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000807', 'engineered_tag', 'A tag that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000808', 'validated_cDNA_clone', 'A cDNA clone that has been validated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000809', 'invalidated_cDNA_clone', 'A cDNA clone that is invalid." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000810', 'chimeric_cDNA_clone', 'A cDNA clone invalidated because it is chimeric." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000811', 'genomically_contaminated_cDNA_clone', 'A cDNA clone invalidated by genomic contamination." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000812', 'polyA_primed_cDNA_clone', 'A cDNA clone invalidated by polyA priming." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000813', 'partially_processed_cDNA_clone', 'A cDNA invalidated clone by partial processing." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000814', 'rescue', 'An attribute describing a regions ability, when introduced to a mutant organism, to re-establish (rescue) a phenotype." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000815', 'mini_gene', 'By definition, minigenes are short open-reading frames (ORF), usually encoding approximately 9 to 20 amino acids, which are expressed in vivo (as distinct from being synthesized as peptide or protein ex vivo and subsequently injected). The in vivo synthesis confers a distinct advantage: the expressed sequences can enter both antigen presentation pathways, MHC I (inducing CD8+ T- cells, which are usually cytotoxic T-lymphocytes (CTL)) and MHC II (inducing CD4+ T-cells, usually T-helpers (Th)); and can encounter B-cells, inducing antibody responses. Three main vector approaches have been used to deliver minigenes: viral vectors, bacterial vectors and plasmid DNA." [PMID:15992143]');
INSERT INTO soterms (id, name, definition) VALUES ('000816', 'rescue_gene', 'A gene that rescues." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000817', 'wild_type', 'An attribute describing sequence with the genotype found in nature and/or standard laboratory stock." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000818', 'wild_type_rescue_gene', 'A gene that rescues." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000819', 'mitochondrial_chromosome', 'A chromosome originating in a mitochondria." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000820', 'chloroplast_chromosome', 'A chromosome originating in a chloroplast." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000821', 'chromoplast_chromosome', 'A chromosome originating in a chromoplast." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000822', 'cyanelle_chromosome', 'A chromosome originating in a cyanelle." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000823', 'leucoplast_chromosome', 'A chromosome with origin in a leucoplast." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000824', 'macronuclear_chromosome', 'A chromosome originating in a macronucleus." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000825', 'micronuclear_chromosome', 'A chromosome originating in a micronucleus." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000828', 'nuclear_chromosome', 'A chromosome originating in a nucleus." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000829', 'nucleomorphic_chromosome', 'A chromosome originating in a nucleomorph." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000830', 'chromosome_part', 'A region of a chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000831', 'gene_member_region', 'A region of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000832', 'promoter_region', 'A region of sequence which is part of a promoter." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000833', 'transcript_region', 'A region of a transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000834', 'mature_transcript_region', 'A region of a mature transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000835', 'primary_transcript_region', 'A part of a primary transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000836', 'mRNA_region', 'A region of an mRNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000837', 'UTR_region', 'A region of UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000838', 'rRNA_primary_transcript_region', 'A region of an rRNA primary transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000839', 'polypeptide_region', 'Biological sequence region that can be assigned to a specific subsequence of a polypeptide." [SO:GAR, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000840', 'repeat_component', 'A region of a repeated sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000841', 'spliceosomal_intron_region', 'A region within an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000842', 'gene_component_region', 'A region within an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000843', 'bacterial_RNApol_promoter_region', 'A region which is part of a bacterial RNA polymerase promoter." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000844', 'RNApol_II_promoter_region', 'A region of sequence which is a promoter for RNA polymerase II." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000845', 'RNApol_III_promoter_type_1_region', 'A region of sequence which is a promoter for RNA polymerase III type 1." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000846', 'RNApol_III_promoter_type_2_region', 'A region of sequence which is a promoter for RNA polymerase III type 2." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000847', 'tmRNA_region', 'A region of a tmRNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000848', 'LTR_component', 'A region of a tmRNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000849', 'three_prime_LTR_component', 'A region of a tmRNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000850', 'five_prime_LTR_component', 'A region of a tmRNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000851', 'CDS_region', 'A region of a CDS." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000852', 'exon_region', 'A region of an exon." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000853', 'homologous_region', 'A region that is homologous to another region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000854', 'paralogous_region', 'A homologous_region that is paralogous to another region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000855', 'orthologous_region', 'A homologous_region that is orthologous to another region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000856', 'conserved', 'A homologous_region that is orthologous to another region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000857', 'homologous', 'Similarity due to common ancestry." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000858', 'orthologous', 'An attribute describing a kind of homology where divergence occurred after a speciation event." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000859', 'paralogous', 'An attribute describing a kind of homology where divergence occurred after a duplication event." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000860', 'syntenic', 'Attribute describing sequence regions occurring in same order on chromosome of different species." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000861', 'capped_primary_transcript', 'A primary transcript that is capped." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000862', 'capped_mRNA', 'An mRNA that is capped." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000863', 'mRNA_attribute', 'An attribute describing an mRNA feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000864', 'exemplar', 'An attribute describing a sequence is representative of a class of similar sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000865', 'frameshift', 'An attribute describing a sequence that contains a mutation involving the deletion or insertion of one or more bases, where this number is not divisible by 3." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000866', 'minus_1_frameshift', 'A frameshift caused by deleting one base." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000867', 'minus_2_frameshift', 'A frameshift caused by deleting two bases." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000868', 'plus_1_frameshift', 'A frameshift caused by inserting one base." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000869', 'plus_2_framshift', 'A frameshift caused by inserting two bases." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000870', 'trans_spliced', 'An attribute describing transcript sequence that is created by splicing exons from diferent genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000871', 'polyadenylated_mRNA', 'An mRNA that is polyadenylated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000872', 'trans_spliced_mRNA', 'An mRNA that is trans-spliced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000873', 'edited_transcript', 'A transcript that is edited." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000874', 'edited_transcript_by_A_to_I_substitution', 'A transcript that has been edited by A to I substitution." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000875', 'bound_by_protein', 'An attribute describing a sequence that is bound by a protein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000876', 'bound_by_nucleic_acid', 'An attribute describing a sequence that is bound by a nucleic acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000877', 'alternatively_spliced', 'An attribute describing a situation where a gene may encode for more than 1 transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000878', 'monocistronic', 'An attribute describing a sequence that contains the code for one gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000879', 'dicistronic', 'An attribute describing a sequence that contains the code for two gene products." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000880', 'polycistronic', 'An attribute describing a sequence that contains the code for more than one gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000881', 'recoded', 'An attribute describing an mRNA sequence that has been reprogrammed at translation, causing localized alterations." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000882', 'codon_redefined', 'An attribute describing the alteration of codon meaning." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000883', 'stop_codon_read_through', 'A stop codon redefined to be a new amino acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000884', 'stop_codon_redefined_as_pyrrolysine', 'A stop codon redefined to be the new amino acid, pyrrolysine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000885', 'stop_codon_redefined_as_selenocysteine', 'A stop codon redefined to be the new amino acid, selenocysteine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000886', 'recoded_by_translational_bypass', 'Recoded mRNA where a block of nucleotides is not translated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000887', 'translationally_frameshifted', 'Recoding by frameshifting a particular site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000888', 'maternally_imprinted_gene', 'A gene that is maternally_imprinted." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000889', 'paternally_imprinted_gene', 'A gene that is paternally imprinted." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000890', 'post_translationally_regulated_gene', 'A gene that is post translationally regulated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000891', 'negatively_autoregulated_gene', 'A gene that is negatively autoreguated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000892', 'positively_autoregulated_gene', 'A gene that is positively autoregulated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000893', 'silenced', 'An attribute describing an epigenetic process where a gene is inactivated at transcriptional or translational level." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000894', 'silenced_by_DNA_modification', 'An attribute describing an epigenetic process where a gene is inactivated by DNA modifications, resulting in repression of transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000895', 'silenced_by_DNA_methylation', 'An attribute describing an epigenetic process where a gene is inactivated by DNA methylation, resulting in repression of transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000896', 'translationally_regulated_gene', 'A gene that is translationally regulated." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000897', 'allelically_excluded_gene', 'A gene that is allelically_excluded." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000898', 'epigenetically_modified_gene', 'A gene that is epigenetically modified." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000899', 'nuclear_mitochondrial', 'An attribute describing a nuclear pseudogene of a mitochndrial gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000900', 'processed', 'An attribute describing a pseudogene where by an mRNA was retrotransposed. The mRNA sequence is transcribed back into the genome, lacking introns and promotors, but often including a polyA tail." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000901', 'unequally_crossed_over', 'An attribute describing a pseudogene that was created by tandem duplication and unequal crossing over during recombination." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000902', 'transgene', 'A transgene is a gene that has been transferred naturally or by any of a number of genetic engineering techniques from one organism to another." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000903', 'endogenous_retroviral_sequence', 'A transgene is a gene that has been transferred naturally or by any of a number of genetic engineering techniques from one organism to another." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000904', 'rearranged_at_DNA_level', 'An attribute to describe the sequence of a feature, where the DNA is rearranged." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000905', 'status', 'An attribute describing the status of a feature, based on the available evidence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000906', 'independently_known', 'Attribute to describe a feature that is independently known - not predicted." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000907', 'supported_by_sequence_similarity', 'An attribute to describe a feature that has been predicted using sequence similarity techniques." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000908', 'supported_by_domain_match', 'An attribute to describe a feature that has been predicted using sequence similarity of a known domain." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000909', 'supported_by_EST_or_cDNA', 'An attribute to describe a feature that has been predicted using sequence similarity to EST or cDNA data." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000910', 'orphan', 'An attribute to describe a feature that has been predicted using sequence similarity to EST or cDNA data." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000911', 'predicted_by_ab_initio_computation', 'An attribute describing a feature that is predicted by a computer program that did not rely on sequence similarity." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000912', 'asx_turn', 'A motif of three consecutive residues and one H-bond in which: residue(i) is Aspartate or Asparagine (Asx), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2)." [http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('000913', 'cloned_cDNA_insert', 'A clone insert made from cDNA." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000914', 'cloned_genomic_insert', 'A clone insert made from genomic DNA." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000915', 'engineered_insert', 'A clone insert that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000916', 'edit_operation', 'A clone insert that is engineered." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000917', 'insert_U', 'An edit to insert a U." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000918', 'delete_U', 'An edit to delete a uridine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000919', 'substitute_A_to_I', 'An edit to substitute an I for an A." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000920', 'insert_C', 'An edit to insert a cytidine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000921', 'insert_dinucleotide', 'An edit to insert a dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000922', 'substitute_C_to_U', 'An edit to substitute an U for a C." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000923', 'insert_G', 'An edit to insert a G." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000924', 'insert_GC', 'An edit to insert a GC dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000925', 'insert_GU', 'An edit to insert a GU dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000926', 'insert_CU', 'An edit to insert a CU dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000927', 'insert_AU', 'An edit to insert a AU dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000928', 'insert_AA', 'An edit to insert a AA dinucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000929', 'edited_mRNA', 'An mRNA that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000930', 'guide_RNA_region', 'A region of guide RNA." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000931', 'anchor_region', 'A region of a guide_RNA that base-pairs to a target mRNA." [SO:jk]');
INSERT INTO soterms (id, name, definition) VALUES ('000932', 'pre_edited_mRNA', 'A region of a guide_RNA that base-pairs to a target mRNA." [SO:jk]');
INSERT INTO soterms (id, name, definition) VALUES ('000933', 'intermediate', 'An attribute to describe a feature between stages of processing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000934', 'miRNA_target_site', 'A miRNA target site is a binding site where the molecule is a micro RNA." [FB:cds]');
INSERT INTO soterms (id, name, definition) VALUES ('000935', 'edited_CDS', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000936', 'vertebrate_immunoglobulin_T_cell_receptor_rearranged_segment', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000937', 'vertebrate_immune_system_feature', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000938', 'vertebrate_immunoglobulin_T_cell_receptor_rearranged_gene_cluster', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000939', 'vertebrate_immune_system_gene_recombination_signal_feature', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000940', 'recombinationally_rearranged', 'A CDS that is edited." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000941', 'recombinationally_rearranged_vertebrate_immune_system_gene', 'A recombinationally rearranged gene of the vertebrate immune system." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('000942', 'attP_site', 'An integration/excision site of a phage chromosome at which a recombinase acts to insert the phage DNA at a cognate integration/excision site on a bacterial chromosome." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000943', 'attB_site', 'An integration/excision site of a bacterial chromosome at which a recombinase acts to insert foreign DNA containing a cognate integration/excision site." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000944', 'attL_site', 'A region that results from recombination between attP_site and attB_site, composed of the 5 portion of attB_site and the 3 portion of attP_site." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000945', 'attR_site', 'A region that results from recombination between attP_site and attB_site, composed of the 5 portion of attP_site and the 3 portion of attB_site." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000946', 'integration_excision_site', 'A region specifically recognised by a recombinase, which inserts or removes another region marked by a distinct cognate integration/excision site." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000947', 'resolution_site', 'A region specifically recognized by a recombinase, which separates a physically contiguous circle of DNA into two physically separate circles." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000948', 'inversion_site', 'A region specifically recognised by a recombinase, which inverts the region flanked by a pair of sites." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000949', 'dif_site', 'A site at which replicated bacterial circular chromosomes are decatenated by site specific resolvase." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000950', 'attC_site', 'An attC site is a sequence required for the integration of a DNA of an integron." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000951', 'eukaryotic_terminator', 'An attC site is a sequence required for the integration of a DNA of an integron." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000952', 'oriV', 'An origin of vegetative replication in plasmids and phages." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000953', 'oriC', 'An origin of bacterial chromosome replication." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000954', 'DNA_chromosome', 'Structural unit composed of a self-replicating, DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000955', 'double_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000956', 'single_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000957', 'linear_double_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded, linear DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000958', 'circular_double_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded, circular DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000959', 'linear_single_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded, linear DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000960', 'circular_single_stranded_DNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded, circular DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000961', 'RNA_chromosome', 'Structural unit composed of a self-replicating, RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000962', 'single_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000963', 'linear_single_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded, linear RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000964', 'linear_double_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded, linear RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000965', 'double_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000966', 'circular_single_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, single-stranded, circular DNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000967', 'circular_double_stranded_RNA_chromosome', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000968', 'sequence_replication_mode', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000969', 'rolling_circle', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000970', 'theta_replication', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000971', 'DNA_replication_mode', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000972', 'RNA_replication_mode', 'Structural unit composed of a self-replicating, double-stranded, circular RNA molecule." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000973', 'insertion_sequence', 'A terminal_inverted_repeat_element that is bacterial and only encodes the functions required for its transposition between these inverted repeats." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000975', 'minicircle_gene', 'A terminal_inverted_repeat_element that is bacterial and only encodes the functions required for its transposition between these inverted repeats." [SO:as]');
INSERT INTO soterms (id, name, definition) VALUES ('000976', 'cryptic', 'A feature_attribute describing a feature that is not manifest under normal conditions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000977', 'anchor_binding_site', 'A feature_attribute describing a feature that is not manifest under normal conditions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000978', 'template_region', 'A region of a guide_RNA that specifies the insertions and deletions of bases in the editing of a target mRNA." [SO:jk]');
INSERT INTO soterms (id, name, definition) VALUES ('000979', 'gRNA_encoding', 'A non-protein_coding gene that encodes a guide_RNA." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('000980', 'minicircle', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000981', 'rho_dependent_bacterial_terminator', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000982', 'rho_independent_bacterial_terminator', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000983', 'strand_attribute', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000984', 'single', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000985', 'double', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000986', 'topology_attribute', 'A minicircle is a replicon, part of a kinetoplast, that encodes for guide RNAs." [PMID:8395055]');
INSERT INTO soterms (id, name, definition) VALUES ('000987', 'linear', 'A quality of a nucleotide polymer that has a 3-terminal residue and a 5-terminal residue." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000988', 'circular', 'A quality of a nucleotide polymer that has no terminal nucleotide residues." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('000989', 'class_II_RNA', 'Small non-coding RNA (59-60 nt long) containing 5 and 3 ends that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000990', 'class_I_RNA', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000991', 'genomic_DNA', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000992', 'BAC_cloned_genomic_insert', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000993', 'consensus', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000994', 'consensus_region', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000995', 'consensus_mRNA', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000996', 'predicted_gene', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000997', 'gene_fragment', 'Small non-coding RNA (55-65 nt long) containing highly conserved 5 and 3 ends (16 and 8 nt, respectively) that are predicted to come together to form a stem structure. Identified in the social amoeba Dictyostelium discoideum and localized in the cytoplasm." [PMID:15333696]');
INSERT INTO soterms (id, name, definition) VALUES ('000998', 'recursive_splice_site', 'A recursive splice site is a splice site which subdivides a large intron. Recursive splicing is a mechanism that splices large introns by sub dividing the intron at non exonic elements and alternate exons." [http://www.genetics.org/cgi/content/full/170/2/661]');
INSERT INTO soterms (id, name, definition) VALUES ('000999', 'BAC_end', 'A region of sequence from the end of a BAC clone that may provide a highly specific marker." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001000', 'rRNA_16S', 'A large polynucleotide in Bacteria and Archaea, which functions as the small subunit of the ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001001', 'rRNA_23S', 'A large polynucleotide in Bacteria and Archaea, which functions as the large subunit of the ribosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001002', 'rRNA_25S', 'A large polynucleotide which functions as part of the large subunit of the ribosome in some eukaryotes." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001003', 'solo_LTR', 'A recombination product between the 2 LTR of the same element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001004', 'low_complexity', 'A recombination product between the 2 LTR of the same element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001005', 'low_complexity_region', 'A recombination product between the 2 LTR of the same element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001006', 'prophage', 'A phage genome after it has established in the host genome in a latent/immune state either as a plasmid or as an integrated \"island\"." [GOC:jl]');
INSERT INTO soterms (id, name, definition) VALUES ('001007', 'cryptic_prophage', 'A remnant of an integrated prophage in the host genome or an \"island\" in the host genome that includes phage like-genes." [GOC:jl]');
INSERT INTO soterms (id, name, definition) VALUES ('001008', 'tetraloop', 'A base-paired stem with loop of 4 non-hydrogen bonded nucleotides." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001009', 'DNA_constraint_sequence', 'A double-stranded DNA used to control macromolecular structure and function." [http:/www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=pubmed&term=SILVERMAN+SK[au\]&dispmax=50]');
INSERT INTO soterms (id, name, definition) VALUES ('001010', 'i_motif', 'A cytosine rich domain whereby strands associate both inter- and intramolecularly at moderately acidic pH." [PMID:9753739]');
INSERT INTO soterms (id, name, definition) VALUES ('001011', 'PNA_oligo', 'Peptide nucleic acid, is a chemical not known to occur naturally but is artificially synthesized and used in some biological research and medical treatments. The PNA backbone is composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001012', 'DNAzyme', 'A DNA sequence with catalytic activity." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001013', 'MNP', 'A multiple nucleotide polymorphism with alleles of common length > 1, for example AAA/TTT." [http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=rs2067431]');
INSERT INTO soterms (id, name, definition) VALUES ('001014', 'intron_domain', 'A multiple nucleotide polymorphism with alleles of common length > 1, for example AAA/TTT." [http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=rs2067431]');
INSERT INTO soterms (id, name, definition) VALUES ('001015', 'wobble_base_pair', 'A type of non-canonical base pairing, most commonly between G and U, which is important for the secondary structure of RNAs. It has similar thermodynamic stability to the Watson-Crick pairing. Wobble base pairs only have two hydrogen bonds. Other wobble base pair possibilities are I-A, I-U and I-C." [PMID:11256617]');
INSERT INTO soterms (id, name, definition) VALUES ('001016', 'internal_guide_sequence', 'A purine-rich sequence in the group I introns which determines the locations of the splice sites in group I intron splicing and has catalytic activity." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001017', 'silent_mutation', 'A sequence variant that does not affect protein function. Silent mutations may occur in genic ( CDS, UTR, intron etc) and intergenic regions. Silent mutations may have affects on processes such as splicing and regulation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001018', 'epitope', 'A binding site that, in the molecule, interacts selectively and non-covalently with antibodies, B cells or T cells." [http://en.wikipedia.org/wiki/Epitope, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001019', 'copy_number_variation', 'A variation that increases or decreases the copy number of a given region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001020', 'sequence_variant_affecting_copy_number', 'A variation that increases or decreases the copy number of a given region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001021', 'chromosome_breakpoint', 'A variation that increases or decreases the copy number of a given region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001022', 'inversion_breakpoint', 'The point within a chromosome where an inversion begins or ends." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001023', 'allele', 'An allele is one of a set of coexisting sequence variants of a gene." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001024', 'haplotype', 'A haplotype is one of a set of coexisting sequence variants of a haplotype block." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001025', 'polymorphic_sequence_variant', 'A sequence variant that is segregating in one or more natural populations of a species." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001026', 'genome', 'A genome is the sum of genetic material within a cell or virion." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001027', 'genotype', 'A genotype is a variant genome, complete or incomplete." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001028', 'diplotype', 'A diplotype is a pair of haplotypes from a given individual. It is a genotype where the phase is known." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001029', 'direction_attribute', 'A diplotype is a pair of haplotypes from a given individual. It is a genotype where the phase is known." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('001030', 'forward', 'Forward is an attribute of the feature, where the feature is in the 5 to 3 direction." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001031', 'reverse', 'Reverse is an attribute of the feature, where the feature is in the 3 to 5 direction. Again could be applied to primer." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001032', 'mitochondrial_DNA', 'Reverse is an attribute of the feature, where the feature is in the 3 to 5 direction. Again could be applied to primer." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001033', 'chloroplast_DNA', 'Reverse is an attribute of the feature, where the feature is in the 3 to 5 direction. Again could be applied to primer." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001034', 'miRtron', 'A de-branched intron which mimics the structure of pre-miRNA and enters the miRNA processing pathway without Drosha mediated cleavage." [PMID:17589500, SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001035', 'piRNA', 'A small non coding RNA, part of a silencing system that prevents the spreading of selfish genetic elements." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001036', 'arginyl_tRNA', 'A tRNA sequence that has an arginine anticodon, and a 3 arginine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001037', 'mobile_genetic_element', 'A nucleotide region with either intra-genome or intracellular mobility, of varying length, which often carry the information necessary for transfer and recombination with the host genome." [PMID:14681355]');
INSERT INTO soterms (id, name, definition) VALUES ('001038', 'extrachromosomal_mobile_genetic_element', 'An MGE that is not integrated into the host chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001039', 'integrated_mobile_genetic_element', 'An MGE that is integrated into the host chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001040', 'integrated_plasmid', 'A plasmid sequence that is integrated within the host chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001041', 'viral_sequence', 'The region of nucleotide sequence of a virus, a submicroscopic particle that replicates by infecting a host cell." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001042', 'phage_sequence', 'The nucleotide sequence of a virus that infects bacteria." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001043', 'attCtn_site', 'An attachment site located on a conjugative transposon and used for site-specific integration of a conjugative transposon." [Phigo:at]');
INSERT INTO soterms (id, name, definition) VALUES ('001044', 'nuclear_mt_pseudogene', 'A nuclear pseudogene of either coding or non-coding mitochondria derived sequence." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001045', 'cointegrated_plasmid', 'A MGE region consisting of two fused plasmids resulting from a replicative transposition event." [phigo:at]');
INSERT INTO soterms (id, name, definition) VALUES ('001046', 'IRLinv_site', 'Component of the inversion site located at the left of a region susceptible to site-specific inversion." [Phigo:at]');
INSERT INTO soterms (id, name, definition) VALUES ('001047', 'IRRinv_site', 'Component of the inversion site located at the right of a region susceptible to site-specific inversion." [Phigo:at]');
INSERT INTO soterms (id, name, definition) VALUES ('001048', 'inversion_site_part', 'A region located within an inversion site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001049', 'defective_conjugative_transposon', 'An island that contains genes for integration/excision and the gene and site for the initiation of intercellular transfer by conjugation. It can be complemented for transfer by a conjugative transposon." [Phigo:ariane]');
INSERT INTO soterms (id, name, definition) VALUES ('001050', 'repeat_fragment', 'A portion of a repeat, interrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001051', 'nested_region', 'A portion of a repeat, interrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001052', 'nested_repeat', 'A portion of a repeat, interrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001053', 'nested_transposon', 'A portion of a repeat, interrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001054', 'transposon_fragment', 'A portion of a transposon, interrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001055', 'transcriptional_cis_regulatory_region', 'A regulatory_region that modulates the transcription of a gene or genes." [PMID:9679020, SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('001056', 'splicing_regulatory_region', 'A regulatory_region that modulates splicing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001057', 'enhanceosome', 'A regulatory_region that modulates splicing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001058', 'promoter_targeting_sequence', 'A transcriptional_cis_regulatory_region that restricts the activity of a CRM to a single promoter and which functions only when both itself and an insulator are located between the CRM and the promoter." [SO:regcreative]');
INSERT INTO soterms (id, name, definition) VALUES ('001059', 'sequence_alteration', 'A sequence_alteration is a sequence_feature whose extent is the deviation from another sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001060', 'sequence_variant', 'A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001061', 'propeptide_cleavage_site', 'The propeptide_cleavage_site is the arginine/lysine boundary on a propeptide where cleavage occurs." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001062', 'propeptide', 'Part of a peptide chain which is cleaved off during the formation of the mature protein." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001063', 'immature_peptide_region', 'An immature_peptide_region is the extent of the peptide after it has been translated and before any processing occurs." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001064', 'active_peptide', 'Active peptides are proteins which are biologically active, released from a precursor molecule." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001066', 'compositionally_biased_region_of_peptide', 'Polypeptide region that is rich in a particular amino acid or homopolymeric and greater than three residues in length." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001067', 'polypeptide_motif', 'A sequence motif is a short (up to 20 amino acids) region of biological interest. Such motifs, although they are too short to constitute functional domains, share sequence similarities and are conserved in different proteins. They display a common function (protein-binding, subcellular location etc.)." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001068', 'polypeptide_repeat', 'A polypeptide_repeat is a single copy of an internal sequence repetition." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001070', 'polypeptide_structural_region', 'Region of polypeptide with a given structural property." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001071', 'membrane_structure', 'Arrangement of the polypeptide with respect to the lipid bilayer." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001072', 'extramembrane_polypeptide_region', 'Polypeptide region that is localized outside of a lipid bilayer." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001073', 'cytoplasmic_polypeptide_region', 'Polypeptide region that is localized inside the cytoplasm." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001074', 'non_cytoplasmic_polypeptide_region', 'Polypeptide region that is localized outside of a lipid bilayer and outside of the cytoplasm." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001075', 'intramembrane_polypeptide_region', 'Polypeptide region present in the lipid bilayer." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001076', 'membrane_peptide_loop', 'Polypeptide region localized within the lipid bilayer where both ends traverse the same membrane." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001077', 'transmembrane_polypeptide_region', 'Polypeptide region traversing the lipid bilayer." [EBIBS:GAR, UniProt:curator_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001078', 'polypeptide_secondary_structure', 'A region of peptide with secondary structure has hydrogen bonding along the peptide chain that causes a defined conformation of the chain." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001079', 'polypeptide_structural_motif', 'Motif is a three-dimensional structural element within the chain, which appears also in a variety of other molecules. Unlike a domain, a motif does not need to form a stable globular unit." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001080', 'coiled_coil', 'A coiled coil is a structural motif in proteins, in which alpha-helices are coiled together like the strands of a rope." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001081', 'helix_turn_helix', 'A motif comprising two helices separated by a turn." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001082', 'polypeptide_sequencing_information', 'Incompatibility in the sequence due to some experimental problem." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001083', 'non_adjacent_residues', 'Indicates that two consecutive residues in a fragment sequence are not consecutive in the full-length protein and that there are a number of unsequenced residues between them." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001084', 'non_terminal_residue', 'The residue at an extremity of the sequence is not the terminal residue." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001085', 'sequence_conflict', 'Different sources report differing sequences." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001086', 'sequence_uncertainty', 'Describes the positions in a sequence where the authors are unsure about the sequence assignment." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001087', 'cross_link', 'Posttranslationally formed amino acid bonds." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001088', 'disulfide_bond', 'The covalent bond between sulfur atoms that binds two peptide chains or different parts of one peptide chain and is a structural determinant in many protein molecules." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001089', 'post_translationally_modified_region', 'A region where a transformation occurs in a protein after it has been synthesized. This which may regulate, stabilize, crosslink or introduce new chemical functionalities in the protein." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001090', 'covalent_binding_site', 'Binding involving a covalent bond." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001091', 'non_covalent_binding_site', 'Binding site for any chemical group (co-enzyme, prosthetic group, etc.)." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001092', 'polypeptide_metal_contact', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with metal ions." [EBIBS:GAR, SO:cb, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001093', 'protein_protein_contact', 'A binding site that, in the protein molecule, interacts selectively and non-covalently with polypeptide residues." [EBIBS:GAR, UniProt:Curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001094', 'polypeptide_calcium_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with calcium ions." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001095', 'polypeptide_cobalt_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with cobalt ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001096', 'polypeptide_copper_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with copper ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001097', 'polypeptide_iron_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with iron ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001098', 'polypeptide_magnesium_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with magnesium ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001099', 'polypeptide_manganese_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with manganese ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001100', 'polypeptide_molybdenum_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with molybdenum ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001101', 'polypeptide_nickel_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with nickel ions." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001102', 'polypeptide_tungsten_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with tungsten ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001103', 'polypeptide_zinc_ion_contact_site', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with zinc ions." [EBIBS:GAR, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001104', 'catalytic_residue', 'Amino acid involved in the activity of an enzyme." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001105', 'polypeptide_ligand_contact', 'Residues which interact with a ligand." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001106', 'asx_motif', 'A motif of five consecutive residues and two H-bonds in which: Residue(i) is Aspartate or Asparagine (Asx), side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3), main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4)." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001107', 'beta_bulge', 'A motif of three residues within a beta-sheet in which the main chains of two consecutive residues are H-bonded to that of the third, and in which the dihedral angles are as follows: Residue(i): -140 degrees < phi(l) -20 degrees , -90 degrees < psi(l) < 40 degrees. Residue (i+1): -180 degrees < phi < -25 degrees or +120 degrees < phi < +180 degrees, +40 degrees < psi < +180 degrees or -180 degrees < psi < -120 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001108', 'beta_bulge_loop', 'A motif of three residues within a beta-sheet consisting of two H-bonds. Beta bulge loops often occur at the loop ends of beta-hairpins." [EBIBS:GAR, Http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001109', 'beta_bulge_loop_five', 'A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+4), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+3), these loops have an RL nest at residues i+2 and i+3." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001110', 'beta_bulge_loop_six', 'A motif of three residues within a beta-sheet consisting of two H-bonds in which: the main-chain NH of residue(i) is H-bonded to the main-chain CO of residue(i+5), the main-chain CO of residue i is H-bonded to the main-chain NH of residue(i+4), these loops have an RL nest at residues i+3 and i+4." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001111', 'beta_strand', 'A beta strand describes a single length of polypeptide chain that forms part of a beta sheet. A single continuous stretch of amino acids adopting an extended conformation of hydrogen bonds between the N-O and the C=O of another part of the peptide. This forms a secondary protein structure in which two or more extended polypeptide regions are hydrogen-bonded to one another in a planar array." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001112', 'antiparallel_beta_strand', 'A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (one running N-terminal to C-terminal and one running C-terminal to N-terminal). Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i) and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they form two mutual backbone hydrogen bonds to each others flanking peptide groups; this is known as a close pair of hydrogen bonds. The peptide backbone dihedral angles (phi, psi) are about (-140 degrees, 135 degrees) in antiparallel sheets." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001113', 'parallel_beta_strand', 'A peptide region which hydrogen bonded to another region of peptide running in the oposite direction (both running N-terminal to C-terminal). This orientation is slightly less stable because it introduces nonplanarity in the inter-strand hydrogen bonding pattern. Hydrogen bonding occurs between every other C=O from one strand to every other N-H on the adjacent strand. In this case, if two atoms C-alpha (i)and C-alpha (j) are adjacent in two hydrogen-bonded beta strands, then they do not hydrogen bond to each other; rather, one residue forms hydrogen bonds to the residues that flank the other (but not vice versa). For example, residue i may form hydrogen bonds to residues j - 1 and j + 1; this is known as a wide pair of hydrogen bonds. By contrast, residue j may hydrogen-bond to different residues altogether, or to none at all. The dihedral angles (phi, psi) are about (-120 degrees, 115 degrees) in parallel sheets." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001114', 'peptide_helix', 'A helix is a secondary_structure conformation where the peptide backbone forms a coil." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001115', 'left_handed_peptide_helix', 'A left handed helix is a region of peptide where the coiled conformation turns in an anticlockwise, left handed screw." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001116', 'right_handed_peptide_helix', 'A right handed helix is a region of peptide where the coiled conformation turns in a clockwise, right handed screw." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001117', 'alpha_helix', 'The helix has 3.6 residues per turn which corresponds to a translation of 1.5 angstroms (= 0.15 nm) along the helical axis. Every backbone N-H group donates a hydrogen bond to the backbone C=O group of the amino acid four residues earlier." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001118', 'pi_helix', 'The pi helix has 4.1 residues per turn and a translation of 1.15  (=0.115 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid five residues earlier." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001119', 'three_ten_helix', 'The 3-10 helix has 3 residues per turn with a translation of 2.0 angstroms (=0.2 nm) along the helical axis. The N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid three residues earlier." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('001120', 'polypeptide_nest_motif', 'A motif of two consecutive residues with dihedral angles. Nest should not have Proline as any residue. Nests frequently occur as parts of other motifs such as Schellman loops." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001121', 'polypeptide_nest_left_right_motif', 'A motif of two consecutive residues with dihedral angles: Residue(i): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001122', 'polypeptide_nest_right_left_motif', 'A motif of two consecutive residues with dihedral angles: Residue(i): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001123', 'schellmann_loop', 'A motif of six or seven consecutive residues that contains two H-bonds." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001124', 'schellmann_loop_seven', 'Wild type: A motif of seven consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+6), the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+5)." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001125', 'schellmann_loop_six', 'Common Type: A motif of six consecutive residues that contains two H-bonds in which: the main-chain CO of residue(i) is H-bonded to the main-chain NH of residue(i+5) the main-chain CO of residue(i+1) is H-bonded to the main-chain NH of residue(i+4)." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001126', 'serine_threonine_motif', 'A motif of five consecutive residues and two hydrogen bonds in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2) or (i+3) , the main-chain CO group of residue(i) is H-bonded to the main-chain NH of residue(i+3) or (i+4)." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001127', 'serine_threonine_staple_motif', 'A motif of four or five consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain OH of residue(i) is H-bonded to the main-chain CO of residue(i3) or (i4), Phi angles of residues(i1), (i2) and (i3) are negative." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001128', 'polypeptide_turn_motif', 'A reversal in the direction of the backbone of a protein that is stabilized by hydrogen bond between backbone NH and CO groups, involving no more than 4 amino acid residues." [EBIBS:GAR, uniprot:feature_type]');
INSERT INTO soterms (id, name, definition) VALUES ('001129', 'asx_turn_left_handed_type_one', 'Left handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001130', 'asx_turn_left_handed_type_two', 'Left handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001131', 'asx_turn_right_handed_type_two', 'Right handed type II (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, +80 degrees < psi +120 degrees < +180 degrees. Residue(i+1): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001132', 'asx_turn_right_handed_type_one', 'Right handed type I (dihedral angles):- Residue(i): -140 degrees < chi (1) -120 degrees < -20 degrees, -90 degrees < psi +120 degrees < +40 degrees. Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001133', 'beta_turn', 'A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles of the second and third residues, which are the basis for sub-categorization." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001134', 'beta_turn_left_handed_type_one', 'Left handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles:- Residue(i+1): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees. Residue(i+2): -140 degrees > phi > -20 degrees, -90 degrees > psi > +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001135', 'beta_turn_left_handed_type_two', 'Left handed type II: A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees > phi > -20 degrees, +80 degrees > psi > +180 degrees. Residue(i+2): +20 degrees > phi > +140 degrees, -40 degrees > psi > +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001136', 'beta_turn_right_handed_type_one', 'Right handed type I:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees. Residue(i+2): -140 degrees < phi < -20 degrees, -90 degrees < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001137', 'beta_turn_right_handed_type_two', 'Right handed type II:A motif of four consecutive residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth. It is characterized by the dihedral angles: Residue(i+1): -140 degrees < phi < -20 degrees, +80 degrees < psi < +180 degrees. Residue(i+2): +20 degrees < phi < +140 degrees, -40 degrees < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001138', 'gamma_turn', 'Gamma turns, defined for 3 residues i,( i+1),( i+2) if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001139', 'gamma_turn_classic', 'Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=75.0 - psi(i+1)=-64.0." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001140', 'gamma_turn_inverse', 'Gamma turns, defined for 3 residues i, i+1, i+2 if a hydrogen bond exists between residues i and i+2 and the phi and psi angles of residue i+1 fall within 40 degrees: phi(i+1)=-79.0 - psi(i+1)=69.0." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001141', 'serine_threonine_turn', 'A motif of three consecutive residues and one H-bond in which: residue(i) is Serine (S) or Threonine (T), the side-chain O of residue(i) is H-bonded to the main-chain NH of residue(i+2)." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001142', 'st_turn_left_handed_type_one', 'The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001143', 'st_turn_left_handed_type_two', 'The peptide twists in an anticlockwise, left handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001144', 'st_turn_right_handed_type_one', 'The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, -90 degrees psi +120 degrees < +40 degrees, residue(i+1): -140 degrees < phi < -20 degrees, -90 < psi < +40 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001145', 'st_turn_right_handed_type_two', 'The peptide twists in an clockwise, right handed manner. The dihedral angles for this turn are: Residue(i): -140 degrees < chi(1) -120 degrees < -20 degrees, +80 degrees psi +120 degrees < +180 degrees, residue(i+1): +20 degrees < phi < +140 degrees, -40 < psi < +90 degrees." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('001146', 'polypeptide_variation_site', 'A site of sequence variation (alteration). Alternative sequence due to naturally occurring events such as polymorphisms and alternative splicing or experimental methods such as site directed mutagenesis." [EBIBS:GAR, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001147', 'natural_variant_site', 'Describes the natural sequence variants due to polymorphisms, disease-associated mutations, RNA editing and variations between strains, isolates or cultivars." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001148', 'mutated_variant_site', 'Site which has been experimentally altered." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001149', 'alternate_sequence_site', 'Description of sequence variants produced by alternative splicing, alternative promoter usage, alternative initiation and ribosomal frameshifting." [EBIBS:GAR, UniProt:curation_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('001150', 'beta_turn_type_six', 'A motif of four consecutive peptide resides of type VIa or type VIb and where the i+2 residue is cis-proline." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001151', 'beta_turn_type_six_a', 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -90 degrees, psi ~ 0 degrees." [PMID:2371257, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001152', 'beta_turn_type_six_a_one', 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -90 degrees, psi ~ 0 degrees." [PMID:2371257, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001153', 'beta_turn_type_six_a_two', 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -90 degrees, psi ~ 0 degrees." [PMID:2371257, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001154', 'beta_turn_type_six_b', 'A motif of four consecutive peptide residues, of which the i+2 residue is proline, and that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -120 degrees, psi ~ 120 degrees. Residue(i+2): phi ~ -60 degrees, psi ~ 0 degrees." [PMID:2371257, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001155', 'beta_turn_type_eight', 'A motif of four consecutive peptide residues that may contain one H-bond, which, if present, is between the main-chain CO of the first residue and the main-chain NH of the fourth and is characterized by the dihedral angles: Residue(i+1): phi ~ -60 degrees, psi ~ -30 degrees. Residue(i+2): phi ~ -120 degrees, psi ~ 120 degrees." [PMID:2371257, SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001156', 'DRE_motif', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -10 and -60 relative to the TSS. Consensus sequence is WATCGATW." [PMID:12537576]');
INSERT INTO soterms (id, name, definition) VALUES ('001157', 'DMv4_motif', 'A sequence element characteristic of some RNA polymerase II promoters, located immediately upstream of some TATA box elements with respect to the TSS (+1). Consensus sequence is YGGTCACACTR. Marked spatial preference within core promoter; tend to occur near the TSS, although not as tightly as INR (SO:0000014)." [PMID:16827941\:12537576]');
INSERT INTO soterms (id, name, definition) VALUES ('001158', 'E_box_motif', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and +1 relative to the TSS. Consensus sequence is AWCAGCTGWT. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015)." [PMID:12537576\:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001159', 'DMv5_motif', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -50 and -10 relative to the TSS. Consensus sequence is KTYRGTATWTTT. Tends to co-occur with DMv4 (SO:0001157) . Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162)." [PMID:12537576\:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001160', 'DMv3_motif', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -30 and +15 relative to the TSS. Consensus sequence is KNNCAKCNCTRNY. Tends to co-occur with DMv2 (SO:0001161). Tends to not occur with DPE motif (SO:0000015) or MTE (0001162)." [PMID:12537576\:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001161', 'DMv2_motif', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between -60 and -45 relative to the TSS. Consensus sequence is MKSYGGCARCGSYSS. Tends to co-occur with DMv3 (SO:0001160). Tends to not occur with DPE motif (SO:0000015) or MTE (SO:0001162)." [PMID:12537576\:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001162', 'MTE', 'A sequence element characteristic of some RNA polymerase II promoters, usually located between +20 and +30 relative to the TSS. Consensus sequence is CSARCSSAACGS. Tends to co-occur with INR motif (SO:0000014). Tends to not occur with DPE motif (SO:0000015) or DMv5 (SO:0001159)." [PMID:12537576\:15231738, PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001163', 'INR1_motif', 'A promoter motif with consensus sequence TCATTCG." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001164', 'DPE1_motif', 'A promoter motif with consensus sequence CGGACGT." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001165', 'DMv1_motif', 'A promoter motif with consensus sequence CARCCCT." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001166', 'GAGA_motif', 'A non directional promoter motif with consensus sequence GAGAGCG." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001167', 'NDM2_motif', 'A non directional promoter motif with consensus CGMYGYCR." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001168', 'NDM3_motif', 'A non directional promoter motif with consensus sequence GAAAGCT." [PMID:16827941]');
INSERT INTO soterms (id, name, definition) VALUES ('001169', 'ds_RNA_viral_sequence', 'A ds_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001170', 'polinton', 'A kind of DNA transposon that populates the genomes of protists, fungi, and animals, characterized by a unique set of proteins necessary for their transposition, including a protein-primed DNA polymerase B, retroviral integrase, cysteine protease, and ATPase. Polintons are characterized by 6-bp target site duplications, terminal-inverted repeats that are several hundred nucleotides long, and 5-AG and TC-3 termini. Polintons exist as autonomous and nonautonomous elements." [PMID:16537396]');
INSERT INTO soterms (id, name, definition) VALUES ('001171', 'rRNA_21S', 'A component of the large ribosomal subunit in mitochondrial rRNA." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001172', 'tRNA_region', 'A region of a tRNA." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001173', 'anticodon_loop', 'A sequence of seven nucleotide bases in tRNA which contains the anticodon. It has the sequence 5-pyrimidine-purine-anticodon-modified purine-any base-3." [ISBN:0716719207]');
INSERT INTO soterms (id, name, definition) VALUES ('001174', 'anticodon', 'A sequence of three nucleotide bases in tRNA which recognizes a codon in mRNA." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001175', 'CCA_tail', 'Base sequence at the 3 end of a tRNA. The 3-hydroxyl group on the terminal adenosine is the attachment point for the amino acid." [ISBN:0716719207]');
INSERT INTO soterms (id, name, definition) VALUES ('001176', 'DHU_loop', 'Non-base-paired sequence of nucleotide bases in tRNA. It contains several dihydrouracil residues." [ISBN:071671920]');
INSERT INTO soterms (id, name, definition) VALUES ('001177', 'T_loop', 'Non-base-paired sequence of three nucleotide bases in tRNA. It has sequence T-Psi-C." [ISBN:0716719207]');
INSERT INTO soterms (id, name, definition) VALUES ('001178', 'pyrrolysine_tRNA_primary_transcript', 'A primary transcript encoding pyrrolysyl tRNA (SO:0000766)." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001179', 'U3_snoRNA', 'U3 snoRNA is a member of the box C/D class of small nucleolar RNAs. The U3 snoRNA secondary structure is characterised by a small 5 domain (with boxes A and A), and a larger 3 domain (with boxes B, C, C, and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C and D form the C/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5 domain and the hinge region act as a pre-rRNA-binding domain. The 3 domain has conserved protein-binding sites. Both the box B/C and box C/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C/D motif is also necessary for nucleolar localization, stability and hypermethylation of U3 snoRNA. Both box B/C and C/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00012]');
INSERT INTO soterms (id, name, definition) VALUES ('001180', 'AU_rich_element', 'A cis-acting element found in the 3 UTR of some mRNA which is rich in AUUUA pentamers. Messenger RNAs bearing multiple AU-rich elements are often unstable." [PMID:7892223]');
INSERT INTO soterms (id, name, definition) VALUES ('001181', 'Bruno_response_element', 'A cis-acting element found in the 3 UTR of some mRNA which is bound by the Drosophila Bruno protein and its homologs." [PMID:10893231]');
INSERT INTO soterms (id, name, definition) VALUES ('001182', 'iron_responsive_element', 'A regulatory sequence found in the 5 and 3 UTRs of many mRNAs which encode iron-binding proteins. It has a hairpin structure and is recognized by trans-acting proteins known as iron-regulatory proteins." [PMID:3198610, PMID:8710843]');
INSERT INTO soterms (id, name, definition) VALUES ('001183', 'morpholino_backbone', 'An attribute describing a sequence composed of nucleobases bound to a morpholino backbone. A morpholino backbone consists of morpholine (CHEBI:34856) rings connected by phosphorodiamidate linkages." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001184', 'PNA', 'An attribute describing a sequence composed of peptide nucleic acid (CHEBI:48021), a chemical consisting of nucleobases bound to a backbone composed of repeating N-(2-aminoethyl)-glycine units linked by peptide bonds. The purine and pyrimidine bases are linked to the backbone by methylene carbonyl bonds." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001185', 'enzymatic', 'An attribute describing the sequence of a transcript that has catalytic activity with or without an associated ribonucleoprotein." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001186', 'ribozymic', 'An attribute describing the sequence of a transcript that has catalytic activity even without an associated ribonucleoprotein." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001187', 'pseudouridylation_guide_snoRNA', 'A snoRNA that specifies the site of pseudouridylation in an RNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]');
INSERT INTO soterms (id, name, definition) VALUES ('001188', 'LNA', 'An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of locked deoxyribose rings connected to a phosphate backbone. The deoxyribose units conformation is locked by a 2-C,4-C-oxymethylene link." [CHEBI:48010]');
INSERT INTO soterms (id, name, definition) VALUES ('001189', 'LNA_oligo', 'An oligo composed of LNA residues." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001190', 'TNA', 'An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of threose rings connected to a phosphate backbone." [CHEBI:48019]');
INSERT INTO soterms (id, name, definition) VALUES ('001191', 'TNA_oligo', 'An oligo composed of TNA residues." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001192', 'GNA', 'An attribute describing a sequence consisting of nucleobases attached to a repeating unit made of an acyclic three-carbon propylene glycol connected to a phosphate backbone. It has two enantiomeric forms, (R)-GNA and (S)-GNA." [CHEBI:48015]');
INSERT INTO soterms (id, name, definition) VALUES ('001193', 'GNA_oligo', 'An oligo composed of GNA residues." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001194', 'R_GNA', 'An attribute describing a GNA sequence in the (R)-GNA enantiomer." [CHEBI:48016]');
INSERT INTO soterms (id, name, definition) VALUES ('001195', 'R_GNA_oligo', 'An oligo composed of (R)-GNA residues." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001196', 'S_GNA', 'An attribute describing a GNA sequence in the (S)-GNA enantiomer." [CHEBI:48017]');
INSERT INTO soterms (id, name, definition) VALUES ('001197', 'S_GNA_oligo', 'An oligo composed of (S)-GNA residues." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001198', 'ds_DNA_viral_sequence', 'A ds_DNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as double stranded DNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001199', 'ss_RNA_viral_sequence', 'A ss_RNA_viral_sequence is a viral_sequence that is the sequence of a virus that exists as single stranded RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001200', 'negative_sense_ssRNA_viral_sequence', 'A negative_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that is complementary to mRNA and must be converted to positive sense RNA by RNA polymerase before translation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001201', 'positive_sense_ssRNA_viral_sequence', 'A positive_sense_RNA_viral_sequence is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus that can be immediately translated by the host." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001202', 'ambisense_ssRNA_viral_sequence', 'A ambisense_RNA_virus is a ss_RNA_viral_sequence that is the sequence of a single stranded RNA virus with both messenger and anti messenger polarity." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001203', 'RNA_polymerase_promoter', 'A region (DNA) to which RNA polymerase binds, to begin transcription." [xenbase:jb]');
INSERT INTO soterms (id, name, definition) VALUES ('001204', 'Phage_RNA_Polymerase_Promoter', 'A region (DNA) to which Bacteriophage RNA polymerase binds, to begin transcription." [xenbase:jb]');
INSERT INTO soterms (id, name, definition) VALUES ('001205', 'SP6_RNA_Polymerase_Promoter', 'A region (DNA) to which the SP6 RNA polymerase binds, to begin transcription." [xenbase:jb]');
INSERT INTO soterms (id, name, definition) VALUES ('001206', 'T3_RNA_Polymerase_Promoter', 'A DNA sequence to which the T3 RNA polymerase binds, to begin transcription." [xenbase:jb]');
INSERT INTO soterms (id, name, definition) VALUES ('001207', 'T7_RNA_Polymerase_Promoter', 'A region (DNA) to which the T7 RNA polymerase binds, to begin transcription." [xenbase:jb]');
INSERT INTO soterms (id, name, definition) VALUES ('001208', 'five_prime_EST', 'An EST read from the 5 end of a transcript that usually codes for a protein. These regions tend to be conserved across species and do not change much within a gene family." [http://www.ncbi.nlm.nih.gov/About/primer/est.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001209', 'three_prime_EST', 'An EST read from the 3 end of a transcript. They are more likely to fall within non-coding, or untranslated regions(UTRs)." [http://www.ncbi.nlm.nih.gov/About/primer/est.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001210', 'translational_frameshift', 'The region of mRNA (not divisible by 3 bases) that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001211', 'plus_1_translational_frameshift', 'The region of mRNA 1 base long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001212', 'plus_2_translational_frameshift', 'The region of mRNA 2 bases long that is skipped during the process of translational frameshifting (GO:0006452), causing the reading frame to be different." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001213', 'group_III_intron', 'Group III introns are introns found in the mRNA of the plastids of euglenoid protists. They are spliced by a two step transesterification with bulged adenosine as initiating nucleophile." [PMID:11377794]');
INSERT INTO soterms (id, name, definition) VALUES ('001214', 'noncoding_region_of_exon', 'The maximal intersection of exon and UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001215', 'coding_region_of_exon', 'The region of an exon that encodes for protein sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001216', 'endonuclease_spliced_intron', 'An intron that spliced via endonucleolytic cleavage and ligation rather than transesterification." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001217', 'protein_coding_gene', 'An intron that spliced via endonucleolytic cleavage and ligation rather than transesterification." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001218', 'transgenic_insertion', 'An insertion that derives from another organism, via the use of recombinant DNA technology." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001219', 'retrogene', 'An insertion that derives from another organism, via the use of recombinant DNA technology." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001220', 'silenced_by_RNA_interference', 'An attribute describing an epigenetic process where a gene is inactivated by RNA interference." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001221', 'silenced_by_histone_modification', 'An attribute describing an epigenetic process where a gene is inactivated by histone modification." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001222', 'silenced_by_histone_methylation', 'An attribute describing an epigenetic process where a gene is inactivated by histone methylation." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001223', 'silenced_by_histone_deacetylation', 'An attribute describing an epigenetic process where a gene is inactivated by histone deacetylation." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001224', 'gene_silenced_by_RNA_interference', 'A gene that is silenced by RNA interference." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001225', 'gene_silenced_by_histone_modification', 'A gene that is silenced by histone modification." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001226', 'gene_silenced_by_histone_methylation', 'A gene that is silenced by histone methylation." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001227', 'gene_silenced_by_histone_deacetylation', 'A gene that is silenced by histone deacetylation." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001228', 'dihydrouridine', 'A modified RNA base in which the 5,6-dihydrouracil is bound to the ribose ring." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001229', 'pseudouridine', 'A modified RNA base in which the 5- position of the uracil is bound to the ribose ring instead of the 4- position." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001230', 'inosine', 'A modified RNA base in which hypoxanthine is bound to the ribose ring." [http://library.med.utah.edu/RNAmods/, RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001231', 'seven_methylguanine', 'A modified RNA base in which guanine is methylated at the 7- position." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001232', 'ribothymidine', 'A modified RNA base in which thymine is bound to the ribose ring." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001233', 'methylinosine', 'A modified RNA base in which methylhypoxanthine is bound to the ribose ring." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001234', 'mobile', 'An attribute describing a feature that has either intra-genome or intracellular mobility." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001235', 'replicon', 'A region containing at least one unique origin of replication and a unique termination site." [ISBN:0716719207]');
INSERT INTO soterms (id, name, definition) VALUES ('001236', 'base', 'A base is a sequence feature that corresponds to a single unit of a nucleotide polymer." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001237', 'amino_acid', 'A sequence feature that corresponds to a single amino acid residue in a polypeptide." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001238', 'major_TSS', 'A sequence feature that corresponds to a single amino acid residue in a polypeptide." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001239', 'minor_TSS', 'A sequence feature that corresponds to a single amino acid residue in a polypeptide." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001240', 'TSS_region', 'The region of a gene from the 5 most TSS to the 3 TSS." [BBOP:nw]');
INSERT INTO soterms (id, name, definition) VALUES ('001241', 'encodes_alternate_transcription_start_sites', 'The region of a gene from the 5 most TSS to the 3 TSS." [BBOP:nw]');
INSERT INTO soterms (id, name, definition) VALUES ('001243', 'miRNA_primary_transcript_region', 'A part of an miRNA primary_transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001244', 'pre_miRNA', 'The 60-70 nucleotide region remain after Drosha processing of the primary transcript, that folds back upon itself to form a hairpin structure." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001245', 'miRNA_stem', 'The stem of the hairpin loop formed by folding of the pre-miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001246', 'miRNA_loop', 'The loop of the hairpin loop formed by folding of the pre-miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001247', 'synthetic_oligo', 'An oligo composed of synthetic nucleotides." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001248', 'assembly', 'A region of the genome of known length that is composed by ordering and aligning two or more different regions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001249', 'fragment_assembly', 'A fragment assembly is a genome assembly that orders overlapping fragments of the genome based on landmark sequences. The base pair distance between the landmarks is known allowing additivity of lengths." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001250', 'fingerprint_map', 'A fingerprint_map is a physical map composed of restriction fragments." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001251', 'STS_map', 'An STS map is a physical map organized by the unique STS landmarks." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001252', 'RH_map', 'A radiation hybrid map is a physical map." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001253', 'sonicate_fragment', 'A DNA fragment generated by sonication. Sonication is a technique used to sheer DNA into smaller fragments." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001254', 'polyploid', 'A kind of chromosome variation where the chromosome complement is an exact multiple of the haploid number and is greater than the diploid number." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001255', 'autopolyploid', 'A polyploid where the multiple chromosome set was derived from the same organism." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001256', 'allopolyploid', 'A polyploid where the multiple chromosome set was derived from a different organism." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001257', 'homing_endonuclease_binding_site', 'The binding site (recognition site) of a homing endonuclease. The binding site is typically large." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001258', 'octamer_motif', 'A sequence element characteristic of some RNA polymerase II promoters with sequence ATTGCAT that binds Pou-domain transcription factors." [GOC:dh, PMID:3095662]');
INSERT INTO soterms (id, name, definition) VALUES ('001259', 'apicoplast_chromosome', 'A chromosome originating in an apicoplast." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001260', 'sequence_collection', 'A collection of discontinuous sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001261', 'overlapping_feature_set', 'A continuous region of sequence composed of the overlapping of multiple sequence_features, which ultimately provides evidence for another sequence_feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001262', 'overlapping_EST_set', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001263', 'ncRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001264', 'gRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001265', 'miRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001266', 'scRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001267', 'snoRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001268', 'snRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001269', 'SRP_RNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001271', 'tmRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001272', 'tRNA_gene', 'A continous experimental result region extending the length of multiple overlapping ESTs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001273', 'modified_adenosine', 'A modified adenine is an adenine base feature that has been altered." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001274', 'modified_inosine', 'A modified inosine is an inosine base feature that has been altered." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001275', 'modified_cytidine', 'A modified cytidine is a cytidine base feature which has been altered." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001276', 'modified_guanosine', 'A modified cytidine is a cytidine base feature which has been altered." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001277', 'modified_uridine', 'A modified cytidine is a cytidine base feature which has been altered." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001278', 'one_methylinosine', '1-methylinosine is a modified inosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001279', 'one_two_prime_O_dimethylinosine', '1,2-O-dimethylinosine is a modified inosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001280', 'two_prime_O_methylinosine', '2-O-methylinosine is a modified inosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001281', 'three_methylcytidine', '3-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001282', 'five_methylcytidine', '5-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001283', 'two_prime_O_methylcytidine', '2-O-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001284', 'two_thiocytidine', '2-thiocytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001285', 'N4_acetylcytidine', 'N4-acetylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001286', 'five_formylcytidine', '5-formylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001287', 'five_two_prime_O_dimethylcytidine', '5,2-O-dimethylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001288', 'N4_acetyl_2_prime_O_methylcytidine', 'N4-acetyl-2-O-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001289', 'lysidine', 'Lysidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001290', 'N4_methylcytidine', 'N4-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001291', 'N4_2_prime_O_dimethylcytidine', 'N4,2-O-dimethylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001292', 'five_hydroxymethylcytidine', '5-hydroxymethylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001293', 'five_formyl_two_prime_O_methylcytidine', '5-formyl-2-O-methylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001294', 'N4_N4_2_prime_O_trimethylcytidine', 'N4_N4_2_prime_O_trimethylcytidine is a modified cytidine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001295', 'one_methyladenosine', '1_methyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001296', 'two_methyladenosine', '2_methyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001297', 'N6_methyladenosine', 'N6_methyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001298', 'two_prime_O_methyladenosine', '2prime_O_methyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001299', 'two_methylthio_N6_methyladenosine', '2_methylthio_N6_methyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001300', 'N6_isopentenyladenosine', 'N6_isopentenyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001301', 'two_methylthio_N6_isopentenyladenosine', '2_methylthio_N6_isopentenyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001302', 'N6_cis_hydroxyisopentenyl_adenosine', 'N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001303', 'two_methylthio_N6_cis_hydroxyisopentenyl_adenosine', '2_methylthio_N6_cis_hydroxyisopentenyl_adenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001304', 'N6_glycinylcarbamoyladenosine', 'N6_glycinylcarbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001305', 'N6_threonylcarbamoyladenosine', 'N6_threonylcarbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001306', 'two_methylthio_N6_threonyl_carbamoyladenosine', '2_methylthio_N6_threonyl_carbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001307', 'N6_methyl_N6_threonylcarbamoyladenosine', 'N6_methyl_N6_threonylcarbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001308', 'N6_hydroxynorvalylcarbamoyladenosine', 'N6_hydroxynorvalylcarbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001309', 'two_methylthio_N6_hydroxynorvalyl_carbamoyladenosine', '2_methylthio_N6_hydroxynorvalyl_carbamoyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001310', 'two_prime_O_ribosyladenosine_phosphate', '2prime_O_ribosyladenosine_phosphate is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001311', 'N6_N6_dimethyladenosine', 'N6_N6_dimethyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001312', 'N6_2_prime_O_dimethyladenosine', 'N6_2prime_O_dimethyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001313', 'N6_N6_2_prime_O_trimethyladenosine', 'N6_N6_2prime_O_trimethyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001314', 'one_two_prime_O_dimethyladenosine', '1,2-O-dimethyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001315', 'N6_acetyladenosine', 'N6_acetyladenosine is a modified adenosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001316', 'seven_deazaguanosine', '7-deazaguanosine is a modified guanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001317', 'queuosine', 'Queuosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001318', 'epoxyqueuosine', 'Epoxyqueuosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001319', 'galactosyl_queuosine', 'Galactosyl_queuosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001320', 'mannosyl_queuosine', 'Mannosyl_queuosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001321', 'seven_cyano_seven_deazaguanosine', '7_cyano_7_deazaguanosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001322', 'seven_aminomethyl_seven_deazaguanosine', '7_aminomethyl_7_deazaguanosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001323', 'archaeosine', 'Archaeosine is a modified 7-deazoguanosine." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001324', 'one_methylguanosine', '1_methylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001325', 'N2_methylguanosine', 'N2_methylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001326', 'seven_methylguanosine', '7_methylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001327', 'two_prime_O_methylguanosine', '2prime_O_methylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001328', 'N2_N2_dimethylguanosine', 'N2_N2_dimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001329', 'N2_2_prime_O_dimethylguanosine', 'N2_2prime_O_dimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001330', 'N2_N2_2_prime_O_trimethylguanosine', 'N2_N2_2prime_O_trimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001331', 'two_prime_O_ribosylguanosine_phosphate', '2prime_O_ribosylguanosine_phosphate is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001332', 'wybutosine', 'Wybutosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001333', 'peroxywybutosine', 'Peroxywybutosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001334', 'hydroxywybutosine', 'Hydroxywybutosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001335', 'undermodified_hydroxywybutosine', 'Undermodified_hydroxywybutosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001336', 'wyosine', 'Wyosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001337', 'methylwyosine', 'Methylwyosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001338', 'N2_7_dimethylguanosine', 'N2_7_dimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001339', 'N2_N2_7_trimethylguanosine', 'N2_N2_7_trimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001340', 'one_two_prime_O_dimethylguanosine', '1_2prime_O_dimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001341', 'four_demethylwyosine', '4_demethylwyosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001342', 'isowyosine', 'Isowyosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001343', 'N2_7_2prirme_O_trimethylguanosine', 'N2_7_2prirme_O_trimethylguanosine is a modified guanosine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001344', 'five_methyluridine', '5_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001345', 'two_prime_O_methyluridine', '2prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001346', 'five_two_prime_O_dimethyluridine', '5_2_prime_O_dimethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001347', 'one_methylpseudouridine', '1_methylpseudouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001348', 'two_prime_O_methylpseudouridine', '2prime_O_methylpseudouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001349', 'two_thiouridine', '2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001350', 'four_thiouridine', '4_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001351', 'five_methyl_2_thiouridine', '5_methyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001352', 'two_thio_two_prime_O_methyluridine', '2_thio_2prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001353', 'three_three_amino_three_carboxypropyl_uridine', '3_3_amino_3_carboxypropyl_uridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001354', 'five_hydroxyuridine', '5_hydroxyuridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001355', 'five_methoxyuridine', '5_methoxyuridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001356', 'uridine_five_oxyacetic_acid', 'Uridine_5_oxyacetic_acid is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001357', 'uridine_five_oxyacetic_acid_methyl_ester', 'Uridine_5_oxyacetic_acid_methyl_ester is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001358', 'five_carboxyhydroxymethyl_uridine', '5_carboxyhydroxymethyl_uridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001359', 'five_carboxyhydroxymethyl_uridine_methyl_ester', '5_carboxyhydroxymethyl_uridine_methyl_ester is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001360', 'five_methoxycarbonylmethyluridine', 'Five_methoxycarbonylmethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001361', 'five_methoxycarbonylmethyl_two_prime_O_methyluridine', 'Five_methoxycarbonylmethyl_2_prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001362', 'five_methoxycarbonylmethyl_two_thiouridine', '5_methoxycarbonylmethyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001363', 'five_aminomethyl_two_thiouridine', '5_aminomethyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001364', 'five_methylaminomethyluridine', '5_methylaminomethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001365', 'five_methylaminomethyl_two_thiouridine', '5_methylaminomethyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001366', 'five_methylaminomethyl_two_selenouridine', '5_methylaminomethyl_2_selenouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001367', 'five_carbamoylmethyluridine', '5_carbamoylmethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001368', 'five_carbamoylmethyl_two_prime_O_methyluridine', '5_carbamoylmethyl_2_prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001369', 'five_carboxymethylaminomethyluridine', '5_carboxymethylaminomethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001370', 'five_carboxymethylaminomethyl_two_prime_O_methyluridine', '5_carboxymethylaminomethyl_2_prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001371', 'five_carboxymethylaminomethyl_two_thiouridine', '5_carboxymethylaminomethyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001372', 'three_methyluridine', '3_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001373', 'one_methyl_three_three_amino_three_carboxypropyl_pseudouridine', '1_methyl_3_3_amino_3_carboxypropyl_pseudouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001374', 'five_carboxymethyluridine', '5_carboxymethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001375', 'three_two_prime_O_dimethyluridine', '3_2prime_O_dimethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001376', 'five_methyldihydrouridine', '5_methyldihydrouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001377', 'three_methylpseudouridine', '3_methylpseudouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001378', 'five_taurinomethyluridine', '5_taurinomethyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001379', 'five_taurinomethyl_two_thiouridine', '5_taurinomethyl_2_thiouridineis a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001380', 'five_isopentenylaminomethyl_uridine', '5_isopentenylaminomethyl_uridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001381', 'five_isopentenylaminomethyl_two_thiouridine', '5_isopentenylaminomethyl_2_thiouridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001382', 'five_isopentenylaminomethyl_two_prime_O_methyluridine', '5_isopentenylaminomethyl_2prime_O_methyluridine is a modified uridine base feature." [http://library.med.utah.edu/RNAmods/]');
INSERT INTO soterms (id, name, definition) VALUES ('001383', 'histone_binding_site', 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues of a histone." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001384', 'CDS_fragment', 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues of a histone." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001385', 'modified_amino_acid_feature', 'A post translationally modified amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001386', 'modified_glycine', 'A post translationally modified glycine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001387', 'modified_L_alanine', 'A post translationally modified alanine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001388', 'modified_L_asparagine', 'A post translationally modified asparagine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001389', 'modified_L_aspartic_acid', 'A post translationally modified aspartic acid amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001390', 'modified_L_cysteine', 'A post translationally modified cysteine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001391', 'modified_L_glutamic_acid', 'A post translationally modified cysteine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001392', 'modified_L_threonine', 'A post translationally modified threonine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001393', 'modified_L_tryptophan', 'A post translationally modified tryptophan amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001394', 'modified_L_glutamine', 'A post translationally modified glutamine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001395', 'modified_L_methionine', 'A post translationally modified methionine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001396', 'modified_L_isoleucine', 'A post translationally modified isoleucine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001397', 'modified_L_phenylalanine', 'A post translationally modified phenylalanine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001398', 'modified_L_histidine', 'A post translationally modified histidine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001399', 'modified_L_serine', 'A post translationally modified serine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001400', 'modified_L_lysine', 'A post translationally modified lysine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001401', 'modified_L_leucine', 'A post translationally modified leucine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001402', 'modified_L_selenocysteine', 'A post translationally modified selenocysteine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001403', 'modified_L_valine', 'A post translationally modified valine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001404', 'modified_L_proline', 'A post translationally modified proline amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001405', 'modified_L_tyrosine', 'A post translationally modified tyrosine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001406', 'modified_L_arginine', 'A post translationally modified arginine amino acid feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001407', 'peptidyl', 'An attribute describing the nature of a proteinaceous polymer, where by the amino acid units are joined by peptide bonds." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001408', 'cleaved_for_gpi_anchor_region', 'The C-terminal residues of a polypeptide which are exchanged for a GPI-anchor." [EBI:rh]');
INSERT INTO soterms (id, name, definition) VALUES ('001409', 'biomaterial_region', 'A region which is intended for use in an experiment." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001410', 'experimental_feature', 'A region which is the result of some arbitrary experimental procedure. The procedure may be carried out with biological material or inside a computer." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001411', 'biological_region', 'A region defined by its disposition to be involved in a biological process." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001412', 'topologically_defined_region', 'A region that is defined according to its relations with other regions within the same sequence." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001413', 'translocation_breakpoint', 'The point within a chromosome where a translocation begins or ends." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001414', 'insertion_breakpoint', 'The point within a chromosome where a insertion begins or ends." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001415', 'deletion_breakpoint', 'The point within a chromosome where a deletion begins or ends." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001416', 'five_prime_flanking_region', 'A flanking region located five prime of a specific region." [SO:chado]');
INSERT INTO soterms (id, name, definition) VALUES ('001417', 'three_prime_flanking_region', 'A flanking region located three prime of a specific region." [SO:chado]');
INSERT INTO soterms (id, name, definition) VALUES ('001418', 'transcribed_fragment', 'An experimental region, defined by a tiling array experiment to be transcribed at some level." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001419', 'cis_splice_site', 'Intronic 2 bp region bordering exon. A splice_site that adjacent_to exon and overlaps intron." [SO:cjm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001420', 'trans_splice_site', 'Primary transcript region bordering trans-splice junction." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001421', 'splice_junction', 'The boundary between an intron and an exon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001422', 'conformational_switch', 'A region of a polypeptide, involved in the transition from one conformational state to another." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001423', 'dye_terminator_read', 'A read produced by the dye terminator method of sequencing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001424', 'pyrosequenced_read', 'A read produced by pyrosequencing technology." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001425', 'ligation_based_read', 'A read produced by ligation based sequencing technologies." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001426', 'polymerase_synthesis_read', 'A read produced by the polymerase based sequence by synthesis method." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001427', 'cis_regulatory_frameshift_element', 'A structural region in an RNA molecule which promotes ribosomal frameshifting of cis coding sequence." [RFAM:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('001428', 'expressed_sequence_assembly', 'A sequence assembly derived from expressed sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001429', 'DNA_binding_site', 'A binding site that, in the molecule, interacts selectively and non-covalently with DNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001431', 'cryptic_gene', 'A gene that is not transcribed under normal conditions and is not critical to normal cellular functioning." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001432', 'sequence_variant_affecting_polyadenylation', 'A gene that is not transcribed under normal conditions and is not critical to normal cellular functioning." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001433', 'three_prime_RACE_clone', 'A three prime RACE (Rapid Amplification of cDNA Ends) clone is a cDNA clone copied from the 3 end of an mRNA (using a poly-dT primer to capture the polyA tail and a gene-specific or randomly primed 5 primer), and spliced into a vector for propagation in a suitable host." [modENCODE:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001434', 'cassette_pseudogene', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001435', 'alanine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001436', 'valine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001437', 'leucine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001438', 'isoleucine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001439', 'proline', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001440', 'tryptophan', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001441', 'phenylalanine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001442', 'methionine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001443', 'glycine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001444', 'serine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001445', 'threonine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001446', 'tyrosine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001447', 'cysteine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001448', 'glutamine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001449', 'asparagine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001450', 'lysine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001451', 'arginine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001452', 'histidine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001453', 'aspartic_acid', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001454', 'glutamic_acid', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001455', 'selenocysteine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001456', 'pyrrolysine', 'A cassette pseudogene is a kind of gene in an inactive form which may recombine at a telomeric locus to form a functional copy." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001457', 'transcribed_cluster', 'A region defined by a set of transcribed sequences from the same gene or expressed pseudogene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001458', 'unigene_cluster', 'A kind of transcribed_cluster defined by a set of transcribed sequences from the a unique gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001459', 'CRISPR', 'Clustered Palindromic Repeats interspersed with bacteriophage derived spacer sequences." [RFAM:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('001460', 'insulator_binding_site', 'A binding site that, in an insulator region of a nucleotide molecule, interacts selectively and non-covalently with polypeptide residues." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001461', 'enhancer_binding_site', 'A binding site that, in the enhancer region of a nucleotide molecule, interacts selectively and non-covalently with polypeptide residues." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001462', 'contig_collection', 'A collection of contigs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001463', 'lincRNA', 'Long intergenic non-coding RNA. A transcript that does not overlap within the start or end genomic coordinates of a coding gene or pseudogene on either strand." [PMID:19182780, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001464', 'UST', 'An EST spanning part or all of the untranslated regions of a protein-coding transcript." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001465', 'three_prime_UST', 'A UST located in the 3UTR of a protein-coding transcript." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001466', 'five_prime_UST', 'An UST located in the 5UTR of a protein-coding transcript." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001467', 'RST', 'A tag produced from a single sequencing read from a RACE product; typically a few hundred base pairs long." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001468', 'three_prime_RST', 'A tag produced from a single sequencing read from a 3-RACE product; typically a few hundred base pairs long." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001469', 'five_prime_RST', 'A tag produced from a single sequencing read from a 5-RACE product; typically a few hundred base pairs long." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001470', 'UST_match', 'A match against an UST sequence." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001471', 'RST_match', 'A match against an RST sequence." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001472', 'primer_match', 'A nucleotide match to a primer sequence." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001473', 'miRNA_antiguide', 'A region of the pri miRNA that base pairs with the guide to form the hairpin." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001474', 'trans_splice_junction', 'The boundary between the spliced leader and the first exon of the mRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001475', 'outron', 'A region of a primary transcript, that is removed via trans splicing." [PMID:16401417, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001476', 'natural_plasmid', 'A plasmid that occurs naturally." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001477', 'gene_trap_construct', 'A gene trap construct is a type of engineered plasmid which is designed to integrate into a genome and produce a fusion transcript between exons of the gene into which it inserts and a reporter element in the construct. Gene traps contain a splice acceptor, do not contain promoter elements for the reporter, and are mutagenic. Gene traps may be bicistronic with the second cassette containing a promoter driving an a selectable marker." [ZFIN:dh]');
INSERT INTO soterms (id, name, definition) VALUES ('001478', 'promoter_trap_construct', 'A promoter trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when inserted in close proximity to a promoter element. Promoter traps typically do not contain promoter elements and are mutagenic." [ZFIN:dh]');
INSERT INTO soterms (id, name, definition) VALUES ('001479', 'enhancer_trap_construct', 'An enhancer trap construct is a type of engineered plasmid which is designed to integrate into a genome and express a reporter when the expression from a basic minimal promoter is enhanced by genomic enhancer elements. Enhancer traps contain promoter elements and are not usually mutagenic." [ZFIN:dh]');
INSERT INTO soterms (id, name, definition) VALUES ('001480', 'PAC_end', 'A region of sequence from the end of a PAC clone that may provide a highly specific marker." [ZFIN:mh]');
INSERT INTO soterms (id, name, definition) VALUES ('001481', 'RAPD', 'RAPD is a PCR product where a sequence variant is identified through the use of PCR with random primers." [ZFIN:mh]');
INSERT INTO soterms (id, name, definition) VALUES ('001482', 'shadow_enhancer', 'RAPD is a PCR product where a sequence variant is identified through the use of PCR with random primers." [ZFIN:mh]');
INSERT INTO soterms (id, name, definition) VALUES ('001483', 'SNV', 'SNVs are single nucleotide positions in genomic DNA at which different sequence alternatives exist." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001484', 'X_element_combinatorial_repeat', 'An X element combinatorial repeat is a repeat region located between the X element and the telomere or adjacent Y element." [http://www.yeastgenome.org/help/glossary.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001485', 'Y_prime_element', 'A Y element is a repeat region (SO:0000657) located adjacent to telomeric repeats or X element combinatorial repeats, either as a single copy or tandem repeat of two to four copies." [http:http\://www.yeastgenome.org/help/glossary.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001486', 'standard_draft', 'The status of a whole genome sequence, where the data is minimally filtered or un-filtered, from any number of sequencing platforms, and is assembled into contigs. Genome sequence of this quality may harbour regions of poor quality and can be relatively incomplete." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001487', 'high_quality_draft', 'The status of a whole genome sequence, where overall coverage represents at least 90 percent of the genome." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001488', 'improved_high_quality_draft', 'The status of a whole genome sequence, where additional work has been performed, using either manual or automated methods, such as gap resolution." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001489', 'annotation_directed_improved_draft', 'The status of a whole genome sequence,where annotation, and verification of coding regions has occurred." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001490', 'noncontiguous_finished', 'The status of a whole genome sequence, where the assembly is high quality, closure approaches have been successful for most gaps, misassemblies and low quality regions." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001491', 'finished_genome', 'The status of a whole genome sequence, with less than 1 error per 100,000 base pairs." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001492', 'intronic_regulatory_region', 'A regulatory region that is part of an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001493', 'centromere_DNA_Element_I', 'A centromere DNA Element I (CDEI) is a conserved region, part of the centromere, consisting of a consensus region composed of 8-11bp which enables binding by the centromere binding factor 1(Cbf1p)." [PMID:11222754]');
INSERT INTO soterms (id, name, definition) VALUES ('001494', 'centromere_DNA_Element_II', 'A centromere DNA Element II (CDEII) is part a conserved region of the centromere, consisting of a consensus region that is AT-rich and ~ 75-100 bp in length." [PMID:11222754]');
INSERT INTO soterms (id, name, definition) VALUES ('001495', 'centromere_DNA_Element_III', 'A centromere DNA Element I (CDEI) is a conserved region, part of the centromere, consisting of a consensus region that consists of a 25-bp which enables binding by the centromere DNA binding factor 3 (CBF3) complex." [PMID:11222754]');
INSERT INTO soterms (id, name, definition) VALUES ('001496', 'telomeric_repeat', 'The telomeric repeat is a repeat region, part of the chromosome, which in yeast, is a G-rich terminal sequence of the form (TG(1-3))n or more precisely ((TG)(1-6)TG(2-3))n." [PMID:8720065]');
INSERT INTO soterms (id, name, definition) VALUES ('001497', 'X_element', 'The X element is a conserved region, of the telomere, of ~475 bp that contains an ARS sequence and in most cases an Abf1p binding site." [http://www.yeastgenome.org/help/glossary.html#xelemcoresequence, PMID:7785338, PMID:8005434]');
INSERT INTO soterms (id, name, definition) VALUES ('001498', 'YAC_end', 'A region of sequence from the end of a YAC clone that may provide a highly specific marker." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001499', 'whole_genome_sequence_status', 'The status of whole genome sequence." [DOI:10.1126]');
INSERT INTO soterms (id, name, definition) VALUES ('001500', 'heritable_phenotypic_marker', 'A biological_region characterized as a single heritable trait in a phenotype screen. The heritable phenotype may be mapped to a chromosome but generally has not been characterized to a specific gene locus." [JAX:hdene]');
INSERT INTO soterms (id, name, definition) VALUES ('001501', 'peptide_collection', 'A collection of peptide sequences." [BBOP:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001502', 'high_identity_region', 'An experimental feature with high sequence identity to another sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001503', 'processed_transcript', 'A transcript for which no open reading frame has been identified and for which no other function has been determined." [MGI:hdeen]');
INSERT INTO soterms (id, name, definition) VALUES ('001504', 'assortment_derived_variation', 'A chromosome variation derived from an event during meiosis." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001505', 'reference_genome', 'A collection of sequences (often chromosomes) taken as the standard for a given organism and genome assembly." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001506', 'variant_genome', 'A collection of sequences (often chromosomes) of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001507', 'variant_collection', 'A collection of one or more sequences of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001508', 'alteration_attribute', 'A collection of one or more sequences of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001509', 'chromosomal_variation_attribute', 'A collection of one or more sequences of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001510', 'intrachromosomal', 'A collection of one or more sequences of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001511', 'interchromosomal', 'A collection of one or more sequences of an individual." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001512', 'insertion_attribute', 'A quality of a chromosomal insertion,." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001513', 'tandem', 'A quality of a chromosomal insertion,." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001514', 'direct', 'A quality of an insertion where the insert is not in a cytologically inverted orientation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001515', 'inverted', 'A quality of an insertion where the insert is in a cytologically inverted orientation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001516', 'free', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001517', 'inversion_attribute', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001518', 'pericentric', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001519', 'paracentric', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001520', 'translocaton_attribute', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001521', 'reciprocal', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001522', 'insertional', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001523', 'duplication_attribute', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001524', 'chromosomally_aberrant_genome', 'The quality of a duplication where the new region exists independently of the original." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001525', 'assembly_error_correction', 'A region of sequence where the final nucleotide assignment differs from the original assembly due to an improvement that replaces a mistake." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001526', 'base_call_error_correction', 'A region of sequence where the final nucleotide assignment is different from that given by the base caller due to an improvement that replaces a mistake." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001527', 'peptide_localization_signal', 'A region of peptide sequence used to target the polypeptide molecule to a specific organelle." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001528', 'nuclear_localization_signal', 'A polypeptide region that targets a polypeptide to the nucleus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001529', 'endosomal_localization_signal', 'A polypeptide region that targets a polypeptide to the endosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001530', 'lysosomal_localization_signal', 'A polypeptide region that targets a polypeptide to the lysosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001531', 'nuclear_export_signal', 'A polypeptide region that targets a polypeptide to he cytoplasm." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001532', 'recombination_signal_sequence', 'A region recognized by a recombinase." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001533', 'cryptic_splice_site', 'A splice site that is in part of the transcript not normally spliced. They occur via mutation or transcriptional error." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001534', 'nuclear_rim_localization_signal', 'A polypeptide region that targets a polypeptide to the nuclear rim." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001535', 'p_element', 'A P_element is a DNA transposon responsible for hybrid dysgenesis." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001536', 'functional_variant', 'A sequence variant in which the function of a gene product is altered with respect to a reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001537', 'structural_variant', 'A sequence variant that changes one or more sequence features." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001538', 'transcript_function_variant', 'A sequence variant which alters the functioning of a transcript with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001539', 'translational_product_function_variant', 'A sequence variant that affects the functioning of a translational product with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001540', 'level_of_transcript_variant', 'A sequence variant which alters the level of a transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001541', 'decreased_transcript_level_variant', 'A sequence variant that increases the level of mature, spliced and processed RNA with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001542', 'increased_transcript_level_variant', 'A sequence variant that increases the level of mature, spliced and processed RNA with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001543', 'transcript_processing_variant', 'A sequence variant that affects the post transcriptional processing of a transcript with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001544', 'editing_variant', 'A transcript processing variant whereby the process of editing is disrupted with respect to the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001545', 'polyadenylation_variant', 'A sequence variant that changes polyadenylation with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001546', 'transcript_stability_variant', 'A variant that changes the stability of a transcript with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001547', 'decreased_transcript_stability_variant', 'A sequence variant that decreases transcript stability with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001548', 'increased_transcript_stability_variant', 'A sequence variant that increases transcript stability with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001549', 'transcription_variant', 'A variant that changes alters the transcription of a transcript with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001550', 'rate_of_transcription_variant', 'A sequence variant that changes the rate of transcription with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001551', 'increased_transcription_rate_variant', 'A sequence variant that increases the rate of transcription with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001552', 'decreased_transcription_rate_variant', 'A sequence variant that decreases the rate of transcription with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001553', 'translational_product_level_variant', 'A functional variant that changes the translational product level with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001554', 'polypeptide_function_variant', 'A sequence variant which changes polypeptide functioning with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001555', 'decreased_translational_product_level', 'A sequence variant which decreases the translational product level with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001556', 'increased_translational_product_level', 'A sequence variant which increases the translational product level with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001557', 'polypeptide_gain_of_function_variant', 'A sequence variant which causes gain of polypeptide function with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001558', 'polypeptide_localization_variant', 'A sequence variant which changes the localization of a polypeptide with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001559', 'polypeptide_loss_of_function_variant', 'A sequence variant that causes the loss of a polypeptide function with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001560', 'inactive_ligand_binding_site', 'A sequence variant that causes the inactivation of a ligand binding site with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001561', 'polypeptide_partial_loss_of_function', 'A sequence variant that causes some but not all loss of polypeptide function with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001562', 'polypeptide_post_translational_processing_variant', 'A sequence variant that causes a change in post translational processing of the peptide with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001563', 'copy_number_change', 'A sequence variant where copies of a feature (CNV) are either increased or decreased." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001564', 'gene_variant', 'A sequence variant where the structure of the gene is changed." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001565', 'gene_fusion', 'A sequence variant whereby a two genes have become joined." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001566', 'regulatory_region_variant', 'A sequence variant located within a regulatory region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001567', 'stop_retained_variant', 'A sequence variant where at least one base in the terminator codon is changed, but the terminator remains." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001568', 'splicing_variant', 'A sequence variant that changes the process of splicing." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001569', 'cryptic_splice_site_variant', 'A sequence variant causing a new (functional) splice site." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001570', 'cryptic_splice_acceptor', 'A sequence variant whereby a new splice site is created due to the activation of a new acceptor." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001571', 'cryptic_splice_donor', 'A sequence variant whereby a new splice site is created due to the activation of a new donor." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001572', 'exon_loss_variant', 'A sequence variant whereby an exon is lost from the transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001573', 'intron_gain_variant', 'A sequence variant whereby an intron is gained by the processed transcript; usually a result of an alteration of the donor or acceptor." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001574', 'splice_acceptor_variant', 'A splice variant that changes the 2 base region at the 3 end of an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001575', 'splice_donor_variant', 'A splice variant that changes the 2 base pair region at the 5 end of an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001576', 'transcript_variant', 'A sequence variant that changes the structure of the transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001577', 'complex_transcript_variant', 'A transcript variant with a complex INDEL- Insertion or deletion that spans an exon/intron border or a coding sequence/UTR border." [http://ensembl.org/info/docs/variation/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001578', 'stop_lost', 'A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001579', 'transcript_sequence_variant', 'A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001580', 'coding_sequence_variant', 'A sequence variant that changes the coding sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001582', 'initiator_codon_variant', 'A codon variant that changes at least one base of the first codon of a transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001583', 'missense_variant', 'A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved." [EBI:fc, EBI:gr, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001585', 'conservative_missense_variant', 'A sequence variant whereby at least one base of a codon is changed resulting in a codon that encodes for a different but similar amino acid. These variants may or may not be deleterious." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001586', 'non_conservative_missense_variant', 'A sequence variant whereby at least one base of a codon is changed resulting in a codon that encodes for an amino acid with different biochemical properties." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001587', 'stop_gained', 'A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001589', 'frameshift_variant', 'A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001590', 'terminator_codon_variant', 'A sequence variant whereby at least one of the bases in the terminator codon is changed." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001591', 'frame_restoring_variant', 'A sequence variant that reverts the sequence of a previous frameshift mutation back to the initial frame." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001592', 'minus_1_frameshift_variant', 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base ahead." [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]');
INSERT INTO soterms (id, name, definition) VALUES ('001593', 'minus_2_frameshift_variant', 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base ahead." [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]');
INSERT INTO soterms (id, name, definition) VALUES ('001594', 'plus_1_frameshift_variant', 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base backward." [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]');
INSERT INTO soterms (id, name, definition) VALUES ('001595', 'plus_2_frameshift_variant', 'A sequence variant which causes a disruption of the translational reading frame, by shifting one base backward." [http://arjournals.annualreviews.org/doi/pdf/10.1146/annurev.ge.08.120174.001535]');
INSERT INTO soterms (id, name, definition) VALUES ('001596', 'transcript_secondary_structure_variant', 'A sequence variant within a transcript that changes the secondary structure of the RNA product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001597', 'compensatory_transcript_secondary_structure_variant', 'A secondary structure variant that compensate for the change made by a previous variant." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001598', 'translational_product_structure_variant', 'A sequence variant within the transcript that changes the structure of the translational product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001599', '3D_polypeptide_structure_variant', 'A sequence variant that changes the resulting polypeptide structure." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001600', 'complex_3D_structural_variant', 'A sequence variant that changes the resulting polypeptide structure." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001601', 'conformational_change_variant', 'A sequence variant in the CDS region that causes a conformational change in the resulting polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001602', 'complex_change_of_translational_product_variant', 'A sequence variant in the CDS region that causes a conformational change in the resulting polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001603', 'polypeptide_sequence_variant', 'A sequence variant with in the CDS that causes a change in the resulting polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001604', 'amino_acid_deletion', 'A sequence variant within a CDS resulting in the loss of an amino acid from the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001605', 'amino_acid_insertion', 'A sequence variant within a CDS resulting in the gain of an amino acid to the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001606', 'amino_acid_substitution', 'A sequence variant of a codon resulting in the substitution of one amino acid for another in the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001607', 'conservative_amino_acid_substitution', 'A sequence variant of a codon causing the substitution of a similar amino acid for another in the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001608', 'non_conservative_amino_acid_substitution', 'A sequence variant of a codon causing the substitution of a non conservative amino acid for another in the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001609', 'elongated_polypeptide', 'A sequence variant with in the CDS that causes elongation of the resulting polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001610', 'elongated_polypeptide_C_terminal', 'A sequence variant with in the CDS that causes elongation of the resulting polypeptide sequence at the C terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001611', 'elongated_polypeptide_N_terminal', 'A sequence variant with in the CDS that causes elongation of the resulting polypeptide sequence at the N terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001612', 'elongated_in_frame_polypeptide_C_terminal', 'A sequence variant with in the CDS that causes in frame elongation of the resulting polypeptide sequence at the C terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001613', 'elongated_out_of_frame_polypeptide_C_terminal', 'A sequence variant with in the CDS that causes out of frame elongation of the resulting polypeptide sequence at the C terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001614', 'elongated_in_frame_polypeptide_N_terminal_elongation', 'A sequence variant with in the CDS that causes in frame elongation of the resulting polypeptide sequence at the N terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001615', 'elongated_out_of_frame_polypeptide_N_terminal', 'A sequence variant with in the CDS that causes out of frame elongation of the resulting polypeptide sequence at the N terminus." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001616', 'polypeptide_fusion', 'A sequence variant that causes a fusion of two polypeptide sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001617', 'polypeptide_truncation', 'A sequence variant of the CD that causes a truncation of the resulting polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001618', 'inactive_catalytic_site', 'A sequence variant that causes the inactivation of a catalytic site with respect to a reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001619', 'non_coding_transcript_variant', 'A transcript variant of a non coding RNA gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001620', 'mature_miRNA_variant', 'A transcript variant located with the sequence of the mature miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001621', 'NMD_transcript_variant', 'A variant in a transcript that is the target of NMD." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001622', 'UTR_variant', 'A transcript variant that is located within the UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001623', '5_prime_UTR_variant', 'A UTR variant of the 5 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001624', '3_prime_UTR_variant', 'A UTR variant of the 3 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001626', 'incomplete_terminal_codon_variant', 'A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001627', 'intron_variant', 'A transcript variant occurring within an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001628', 'intergenic_variant', 'A sequence variant located in the intergenic region, between genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001629', 'splice_site_variant', 'A sequence variant that changes the first two or last two bases of an intron, or the 5th base from the start of the intron in the orientation of the transcript." [http://ensembl.org/info/docs/variation/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001630', 'splice_region_variant', 'A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron." [http://ensembl.org/info/docs/variation/index.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001631', 'upstream_gene_variant', 'A sequence variant located 5 of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001632', 'downstream_gene_variant', 'A sequence variant located 3 of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001633', '5KB_downstream_variant', 'A sequence variant located within 5 KB of the end of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001634', '500B_downstream_variant', 'A sequence variant located within a half KB of the end of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001635', '5KB_upstream_variant', 'A sequence variant located within 5KB 5 of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001636', '2KB_upstream_variant', 'A sequence variant located within 2KB 5 of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001637', 'rRNA_gene', 'A gene that encodes for ribosomal RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001638', 'piRNA_gene', 'A gene that encodes for an piwi associated RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001639', 'RNase_P_RNA_gene', 'A gene that encodes an RNase P RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001640', 'RNase_MRP_RNA_gene', 'A gene that encodes a RNase_MRP_RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001641', 'lincRNA_gene', 'A gene that encodes large intervening non-coding RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001642', 'mathematically_defined_repeat', 'A mathematically defined repeat (MDR) is a experimental feature that is determined by querying overlapping oligomers of length k against a database of shotgun sequence data and identifying regions in the query sequence that exceed a statistically determined threshold of repetitiveness." [SO:jestill]');
INSERT INTO soterms (id, name, definition) VALUES ('001643', 'telomerase_RNA_gene', 'A telomerase RNA gene is a non coding RNA gene the RNA product of which is a component of telomerase." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001644', 'targeting_vector', 'An engineered vector that is able to take part in homologous recombination in a host with the intent of introducing site specific genomic modifications." [MGD:tm, PMID:10354467]');
INSERT INTO soterms (id, name, definition) VALUES ('001645', 'genetic_marker', 'A measurable sequence feature that varies within a population." [SO:db]');
INSERT INTO soterms (id, name, definition) VALUES ('001646', 'DArT_marker', 'A genetic marker, discovered using Diversity Arrays Technology (DArT) technology." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001647', 'kozak_sequence', 'A kind of ribosome entry site, specific to Eukaryotic organisms that overlaps part of both 5 UTR and CDS sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001648', 'nested_transposon', 'A transposon that is disrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001649', 'nested_repeat', 'A repeat that is disrupted by the insertion of another element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001650', 'inframe_variant', 'A sequence variant which does not cause a disruption of the translational reading frame." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001653', 'retinoic_acid_responsive_element', 'A transcription factor binding site of variable direct repeats of the sequence PuGGTCA spaced by five nucleotides (DR5) found in the promoters of retinoic acid-responsive genes, to which retinoic acid receptors bind." [PMID:11327309, PMID:19917671]');
INSERT INTO soterms (id, name, definition) VALUES ('001654', 'nucleotide_to_protein_binding_site', 'A binding site that, in the nucleotide molecule, interacts selectively and non-covalently with polypeptide residues." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001655', 'nucleotide_binding_site', 'A binding site that, in the molecule, interacts selectively and non-covalently with nucleotide residues." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001656', 'metal_binding_site', 'A binding site that, in the molecule, interacts selectively and non-covalently with metal ions." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001657', 'ligand_binding_site', 'A binding site that, in the molecule, interacts selectively and non-covalently with a small molecule such as a drug, or hormone." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001658', 'nested_tandem_repeat', 'An NTR is a nested repeat of two distinct tandem motifs interspersed with each other." [SO:AF]');
INSERT INTO soterms (id, name, definition) VALUES ('001659', 'promoter_element', 'An NTR is a nested repeat of two distinct tandem motifs interspersed with each other." [SO:AF]');
INSERT INTO soterms (id, name, definition) VALUES ('001660', 'core_promoter_element', 'An NTR is a nested repeat of two distinct tandem motifs interspersed with each other." [SO:AF]');
INSERT INTO soterms (id, name, definition) VALUES ('001661', 'RNA_polymerase_II_TATA_box', 'A TATA box core promoter of a gene transcribed by RNA polymerase II." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001662', 'RNA_polymerase_III_TATA_box', 'A TATA box core promoter of a gene transcribed by RNA polymerase III." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001663', 'BREd_motif', 'A core TRNA polymerase II promoter element with consensus (G/A)T(T/G/A)(T/A)(G/T)(T/G)(T/G)." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001664', 'DCE', 'A discontinuous core element of RNA polymerase II transcribed genes, situated downstream of the TSS. It is composed of three sub elements: SI, SII and SIII." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001665', 'DCE_SI', 'A sub element of the DCE core promoter element, with consensus sequence CTTC." [PMID:16858867, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001666', 'DCE_SII', 'A sub element of the DCE core promoter element with consensus sequence CTGT." [PMID:16858867, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001667', 'DCE_SIII', 'A sub element of the DCE core promoter element with consensus sequence AGC." [PMID:16858867, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001668', 'proximal_promoter_element', 'DNA segment that ranges from about -250 to -40 relative to +1 of RNA transcription start site, where sequence specific DNA-binding transcription factors binds, such as Sp1, CTF (CCAAT-binding transcription factor), and CBF (CCAAT-box binding factor)." [PMID:12515390, PMID:9679020, SO:ml]');
INSERT INTO soterms (id, name, definition) VALUES ('001669', 'RNApol_II_core_promoter', 'The minimal portion of the promoter required to properly initiate transcription in RNA polymerase II transcribed genes." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001670', 'distal_promoter_element', 'The minimal portion of the promoter required to properly initiate transcription in RNA polymerase II transcribed genes." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001671', 'bacterial_RNApol_promoter_sigma_70', 'The minimal portion of the promoter required to properly initiate transcription in RNA polymerase II transcribed genes." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001672', 'bacterial_RNApol_promoter_sigma54', 'The minimal portion of the promoter required to properly initiate transcription in RNA polymerase II transcribed genes." [PMID:16858867]');
INSERT INTO soterms (id, name, definition) VALUES ('001673', 'minus_12_signal', 'A conserved region about 12-bp upstream of the start point of bacterial transcription units, involved with sigma factor 54." [PMID:18331472]');
INSERT INTO soterms (id, name, definition) VALUES ('001674', 'minus_24_signal', 'A conserved region about 12-bp upstream of the start point of bacterial transcription units, involved with sigma factor 54." [PMID:18331472]');
INSERT INTO soterms (id, name, definition) VALUES ('001675', 'A_box_type_1', 'An A box within an RNA polymerase III type 1 promoter." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001676', 'A_box_type_2', 'An A box within an RNA polymerase III type 2 promoter." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001677', 'intermediate_element', 'A core promoter region of RNA polymerase III type 1 promoters." [PMID:12381659]');
INSERT INTO soterms (id, name, definition) VALUES ('001678', 'regulatory_promoter_element', 'A promoter element that is not part of the core promoter, but provides the promoter with a specific regulatory region." [PMID:12381659]');
INSERT INTO soterms (id, name, definition) VALUES ('001679', 'transcription_regulatory_region', 'A regulatory region that is involved in the control of the process of transcription." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001680', 'translation_regulatory_region', 'A regulatory region that is involved in the control of the process of translation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001681', 'recombination_regulatory_region', 'A regulatory region that is involved in the control of the process of recombination." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001682', 'replication_regulatory_region', 'A regulatory region that is involved in the control of the process of nucleotide replication." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001683', 'sequence_motif', 'A sequence motif is a nucleotide or amino-acid sequence pattern that may have biological significance." [http://en.wikipedia.org/wiki/Sequence_motif]');
INSERT INTO soterms (id, name, definition) VALUES ('001684', 'experimental_feature_attribute', 'An attribute of an experimentally derived feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001685', 'score', 'The score of an experimentally derived feature such as a p-value." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001686', 'quality_value', 'An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001687', 'restriction_enzyme_recognition_site', 'The nucleotide region (usually a palindrome) that is recognized by a restriction enzyme. This may or may not be equal to the restriction enzyme binding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001688', 'restriction_enzyme_cleavage_junction', 'The boundary at which a restriction enzyme breaks the nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001689', 'five_prime_restriction_enzyme_junction', 'The restriction enzyme cleavage junction on the 5 strand of the nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001690', 'three_prime_restriction_enzyme_junction', 'The restriction enzyme cleavage junction on the 5 strand of the nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001691', 'blunt_end_restriction_enzyme_cleavage_site', 'A restriction enzyme recognition site that, when cleaved, results in no overhangs." [SBOL:jgquinn, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001692', 'sticky_end_restriction_enzyme_cleavage_site', 'A restriction enzyme recognition site that, when cleaved, results in no overhangs." [SBOL:jgquinn, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001693', 'blunt_end_restriction_enzyme_cleavage_junction', 'A restriction enzyme cleavage site where both strands are cut at the same position." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001694', 'single_strand_restriction_enzyme_cleavage_site', 'A restriction enzyme cleavage site whereby only one strand is cut." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001695', 'restriction_enzyme_single_strand_overhang', 'A terminal region of DNA sequence where the end of the region is not blunt ended." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001696', 'experimentally_defined_binding_region', 'A region that has been implicated in binding although the exact coordinates of binding may be unknown." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001697', 'ChIP_seq_region', 'A region of sequence identified by CHiP seq technology to contain a protein binding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001698', 'ASPE_primer', '\"A primer containing an SNV at the 3 end for accurate genotyping." [http://www.ncbi.nlm.nih.gov/pubmed/11252801]');
INSERT INTO soterms (id, name, definition) VALUES ('001699', 'dCAPS_primer', 'A primer with one or more mismatches to the DNA template corresponding to a position within a restriction enzyme recognition site." [http://www.ncbi.nlm.nih.gov/pubmed/9628033]');
INSERT INTO soterms (id, name, definition) VALUES ('001700', 'histone_modification', 'Histone modification is a post translationally modified region whereby residues of the histone protein are modified by methylation, acetylation, phosphorylation, ubiquitination, sumoylation, citrullination, or ADP-ribosylation." [http:en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001701', 'histone_methylation_site', 'A histone modification site where the modification is the methylation of the residue." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001702', 'histone_acetylation_site', 'A histone modification where the modification is the acylation of the residue." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001703', 'H3K9_acetylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is acetylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001704', 'H3K14_acetylation_site', 'A kind of histone modification site, whereby the 14th residue (a lysine), from the start of the H3 histone protein is acetylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001705', 'H3K4_monomethylation_site', 'A kind of histone modification, whereby the 4th residue (a lysine), from the start of the H3 protein is mono-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001706', 'H3K4_trimethylation', 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 protein is tri-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001707', 'H3K9_trimethylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is tri-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001708', 'H3K27_monomethylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is mono-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001709', 'H3K27_trimethylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is tri-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001710', 'H3K79_monomethylation_site', 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is mono- methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001711', 'H3K79_dimethylation_site', 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is di-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001712', 'H3K79_trimethylation_site', 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is tri-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001713', 'H4K20_monomethylation_site', 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H4histone protein is mono-methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001714', 'H2BK5_monomethylation_site', 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2B protein is methylated." [http://en.wikipedia.org/wiki/Histone]');
INSERT INTO soterms (id, name, definition) VALUES ('001715', 'ISRE', 'An ISRE is a transcriptional cis regulatory region, containing the consensus region: YAGTTTC(A/T)YTTTYCC, responsible for increased transcription via interferon binding." [http://genesdev.cshlp.org/content/2/4/383.abstrac]');
INSERT INTO soterms (id, name, definition) VALUES ('001716', 'histone_ubiqitination_site', 'A histone modification site where ubiquitin may be added." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001717', 'H2B_ubiquitination_site', 'A histone modification site on H2B where ubiquitin may be added." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001718', 'H3K18_acetylation_site', 'A kind of histone modification site, whereby the 18th residue (a lysine), from the start of the H3 histone protein is acetylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001719', 'H3K23_acetylation_site', 'A kind of histone modification, whereby the 23rd residue (a lysine), from the start of the H3 histone protein is acetylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001720', 'epigenetically_modified_region', 'A biological region implicated in inherited changes caused by mechanisms other than changes in the underlying DNA sequence." [http://en.wikipedia.org/wiki/Epigenetics, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001721', 'H3K27_acylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is acylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001722', 'H3K36_monomethylation_site', 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is mono-methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001723', 'H3K36_dimethylation_site', 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is dimethylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001724', 'H3K36_trimethylation_site', 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is tri-methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001725', 'H3K4_dimethylation_site', 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 histone protein is di-methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001726', 'H3K27_dimethylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is di-methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001727', 'H3K9_monomethylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is mono-methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001728', 'H3K9_dimethylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein may be dimethylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001729', 'H4K16_acetylation_site', 'A kind of histone modification site, whereby the 16th residue (a lysine), from the start of the H4 histone protein is acetylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001730', 'H4K5_acetylation_site', 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H4 histone protein is acetylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001731', 'H4K8_acetylation_site', 'A kind of histone modification site, whereby the 8th residue (a lysine), from the start of the H4 histone protein is acetylated." [SO:KE]');
INSERT INTO soterms (id, name, definition) VALUES ('001732', 'H3K27_methylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001733', 'H3K36_methylation_site', 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001734', 'H3K4_methylation_site', 'A kind of histone modification, whereby the 4th residue (a lysine), from the start of the H3 protein is methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001735', 'H3K79_methylation_site', 'A kind of histone modification site, whereby the 79th residue (a lysine), from the start of the H3 histone protein is methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001736', 'H3K9_methylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H3 histone protein is methylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001737', 'histone_acylation_region', 'A histone modification, whereby the histone protein is acylated at multiple sites in a region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001738', 'H4K_acylation_region', 'A region of the H4 histone whereby multiple lysines are acylated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001739', 'gene_with_non_canonical_start_codon', 'A gene with a start codon other than AUG." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001740', 'gene_with_start_codon_CUG', 'A gene with a translational start codon of CUG." [SO:mc]');
INSERT INTO soterms (id, name, definition) VALUES ('001741', 'pseudogenic_gene_segment', 'A gene segment which when incorporated by somatic recombination in the final gene transcript results in a nonfunctional product." [SO:hd]');
INSERT INTO soterms (id, name, definition) VALUES ('001742', 'copy_number_gain', 'A sequence alteration whereby the copy number of a given regions is greater than the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001743', 'copy_number_loss', 'A sequence alteration whereby the copy number of a given region is less than the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001744', 'UPD', 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from one parent and no copies of the same chromosome or region from the other parent." [SO:BM]');
INSERT INTO soterms (id, name, definition) VALUES ('001745', 'maternal_uniparental_disomy', 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from the mother and no copies of the same chromosome or region from the father." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001746', 'paternal_uniparental_disomy', 'Uniparental disomy is a sequence_alteration where a diploid individual receives two copies for all or part of a chromosome from the father and no copies of the same chromosome or region from the mother." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001747', 'open_chromatin_region', 'A DNA sequence that in the normal state of the chromosome corresponds to an unfolded, un-complexed stretch of double-stranded DNA." [SO:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('001748', 'SL3_acceptor_site', 'A SL2_acceptor_site which appends the SL3 RNA leader sequence to the 5 end of an mRNA. SL3 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001749', 'SL4_acceptor_site', 'A SL2_acceptor_site which appends the SL4 RNA leader sequence to the 5 end of an mRNA. SL4 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001750', 'SL5_acceptor_site', 'A SL2_acceptor_site which appends the SL5 RNA leader sequence to the 5 end of an mRNA. SL5 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001751', 'SL6_acceptor_site', 'A SL2_acceptor_site which appends the SL6 RNA leader sequence to the 5 end of an mRNA. SL6 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001752', 'SL7_acceptor_site', 'A SL2_acceptor_site which appends the SL7 RNA leader sequence to the 5 end of an mRNA. SL7 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001753', 'SL8_acceptor_site', 'A SL2_acceptor_site which appends the SL8 RNA leader sequence to the 5 end of an mRNA. SL8 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001754', 'SL9_acceptor_site', 'A SL2_acceptor_site which appends the SL9 RNA leader sequence to the 5 end of an mRNA. SL9 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001755', 'SL10_acceptor_site', 'A SL2_acceptor_site which appends the SL10 RNA leader sequence to the 5 end of an mRNA. SL10 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001756', 'SL11_acceptor_site', 'A SL2_acceptor_site which appends the SL11 RNA leader sequence to the 5 end of an mRNA. SL11 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001757', 'SL12_acceptor_site', 'A SL2_acceptor_site which appends the SL12 RNA leader sequence to the 5 end of an mRNA. SL12 acceptor sites occur in genes in internal segments of polycistronic transcripts." [SO:nlw]');
INSERT INTO soterms (id, name, definition) VALUES ('001758', 'duplicated_pseudogene', 'A pseudogene that arose via gene duplication. Generally duplicated pseudogenes have the same structure as the original gene, including intron-exon structure and some regulatory sequence." [http://en.wikipedia.org/wiki/Pseudogene]');
INSERT INTO soterms (id, name, definition) VALUES ('001759', 'unitary_pseudogene', 'A pseudogene, deactivated from original state by mutation, fixed in a population,where the ortholog in a reference species such as mouse remains functional." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, http://en.wikipedia.org/wiki/Pseudogene, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001760', 'non_processed_pseudogene', 'A pseudogene that arose from a means other than retrotransposition. A pseudogene created via genomic duplication of a functional protein-coding parent gene followed by accumulation of deleterious mutations." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001761', 'variant_quality', 'A dependent entity that inheres in a bearer, a sequence variant." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001762', 'variant_origin', 'A quality inhering in a variant by virtue of its origin." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001763', 'variant_frequency', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001764', 'unique_variant', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001765', 'rare_variant', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001766', 'polymorphic_variant', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001767', 'common_variant', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001768', 'fixed_variant', 'A physical quality which inheres to the variant by virtue of the number instances of the variant within a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001769', 'variant_phenotype', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001770', 'benign_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001771', 'disease_associated_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001772', 'disease_causing_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001773', 'lethal_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001774', 'quantitative_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001775', 'maternal_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001776', 'paternal_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001777', 'somatic_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001778', 'germline_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001779', 'pedigree_specific_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001780', 'population_specific_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001781', 'de_novo_variant', 'A quality inhering in a variant by virtue of its phenotype." [PMID:17597783, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001782', 'TF_binding_site_variant', 'A sequence variant located within a transcription factor binding site." [EBI:fc]');
INSERT INTO soterms (id, name, definition) VALUES ('001784', 'complex_structural_alteration', 'A structural sequence alteration or rearrangement encompassing one or more genome fragments, with 4 or more breakpoints." [FB:reference_manual, NCBI:th, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001785', 'structural_alteration', 'A structural sequence alteration or rearrangement encompassing one or more genome fragments, with 4 or more breakpoints." [FB:reference_manual, NCBI:th, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001786', 'loss_of_heterozygosity', 'A functional variant whereby the sequence alteration causes a loss of function of one allele of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001787', 'splice_donor_5th_base_variant', 'A sequence variant that causes a change at the 5th base pair after the start of the intron in the orientation of the transcript." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001788', 'U_box', 'An U-box is a conserved T-rich region upstream of a retroviral polypurine tract that is involved in PPT primer creation during reverse transcription." [PMID:10556309, PMID:11577982, PMID:9649446]');
INSERT INTO soterms (id, name, definition) VALUES ('001789', 'mating_type_region', 'A specialized region in the genomes of some yeast and fungi, the genes of which regulate mating type." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001790', 'paired_end_fragment', 'An assembly region that has been sequenced from both ends resulting in a read_pair (mate_pair)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001791', 'exon_variant', 'A sequence variant that changes exon sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001792', 'non_coding_transcript_exon_variant', 'A sequence variant that changes non-coding exon sequence in a non-coding transcript." [EBI:fc, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001793', 'clone_end', 'A read from an end of the clone sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001794', 'point_centromere', 'A point centromere is a relatively small centromere (about 125 bp DNA) in discrete sequence, found in some yeast including S. cerevisiae." [PMID:7502067, SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001795', 'regional_centromere', 'A regional centromere is a large modular centromere found in fission yeast and higher eukaryotes. It consist of a central core region flanked by inverted inner and outer repeat regions." [PMID:7502067, SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001796', 'regional_centromere_central_core', 'A conserved region within the central region of a modular centromere, where the kinetochore is formed." [SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001797', 'centromeric_repeat', 'A repeat region found within the modular centromere." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001798', 'regional_centromere_inner_repeat_region', 'The inner inverted repeat region of a modular centromere and part of the central core surrounding a non-conserved central region. This region is adjacent to the central core, on each chromosome arm." [SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001799', 'regional_centromere_outer_repeat_region', 'The heterochromatic outer repeat region of a modular centromere. These repeats exist in tandem arrays on both chromosome arms." [SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001800', 'tasiRNA', 'The sequence of a 21 nucleotide double stranded, polyadenylated non coding RNA, transcribed from the TAS gene." [PMID:16145017]');
INSERT INTO soterms (id, name, definition) VALUES ('001801', 'tasiRNA_primary_transcript', 'A primary transcript encoding a tasiRNA." [PMID:16145017]');
INSERT INTO soterms (id, name, definition) VALUES ('001802', 'increased_polyadenylation_variant', 'A transcript processing variant whereby polyadenylation of the encoded transcript is increased with respect to the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001803', 'decreased_polyadenylation_variant', 'A transcript processing variant whereby polyadenylation of the encoded transcript is decreased with respect to the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001804', 'DDB_box', 'A conserved polypeptide motif that mediates protein-protein interaction and defines adaptor proteins for DDB1/cullin 4 ubiquitin ligases." [PMID:18794354, PMID:19818632]');
INSERT INTO soterms (id, name, definition) VALUES ('001805', 'destruction_box', 'A conserved polypeptide motif that can be recognized by both Fizzy/Cdc20- and FZR/Cdh1-activated anaphase-promoting complex/cyclosome (APC/C) and targets a protein for ubiquitination and subsequent degradation by the APC/C. The consensus sequence is RXXLXXXXN." [PMID:12208841, PMID:1842691]');
INSERT INTO soterms (id, name, definition) VALUES ('001806', 'ER_retention_signal', 'A C-terminal tetrapeptide motif that mediates retention of a protein in (or retrieval to) the endoplasmic reticulum. In mammals the sequence is KDEL, and in fungi HDEL or DDEL." [doi:10.1093/jxb/50.331.157, PMID:2077689]');
INSERT INTO soterms (id, name, definition) VALUES ('001807', 'KEN_box', 'A conserved polypeptide motif that can be recognized by FZR/Cdh1-activated anaphase-promoting complex/cyclosome (APC/C) and targets a protein for ubiquitination and subsequent degradation by the APC/C. The consensus sequence is KENXXXN." [PMID:10733526, PMID:1220884, PMID:18426916]');
INSERT INTO soterms (id, name, definition) VALUES ('001808', 'mitochondrial_targeting_signal', 'A polypeptide region that targets a polypeptide to the mitochondrion." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001809', 'signal_anchor', 'A signal sequence that is not cleaved from the polypeptide. Anchors a Type II membrane protein to the membrane." [http://www.cbs.dtu.dk/services/SignalP/background/biobackground.php]');
INSERT INTO soterms (id, name, definition) VALUES ('001810', 'PIP_box', 'A polypeptide region that mediates binding to PCNA. The consensus sequence is QXX(hh)XX(aa), where (h) denotes residues with moderately hydrophobic side chains and (a) denotes residues with highly hydrophobic aromatic side chains." [PMID:9631646]');
INSERT INTO soterms (id, name, definition) VALUES ('001811', 'phosphorylation_site', 'A post-translationally modified region in which residues of the protein are modified by phosphorylation." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001812', 'transmembrane_helix', 'A region that traverses the lipid bilayer and adopts a helical secondary structure." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001813', 'vacuolar_sorting_signal', 'A polypeptide region that targets a polypeptide to the vacuole." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001814', 'coding_variant_quality', 'A polypeptide region that targets a polypeptide to the vacuole." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001815', 'synonymous', 'A polypeptide region that targets a polypeptide to the vacuole." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001816', 'non_synonymous', 'A polypeptide region that targets a polypeptide to the vacuole." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001817', 'inframe', 'An attribute describing a sequence that contains a mutation involving the deletion or insertion of one or more bases, where this number is divisible by 3." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001818', 'protein_altering_variant', 'A sequence_variant which is predicted to change the protein encoded in the coding sequence." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001819', 'synonymous_variant', 'A sequence variant where there is no resulting change to the encoded amino acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001820', 'inframe_indel', 'A coding sequence variant where the change does not alter the frame of the transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001821', 'inframe_insertion', 'An inframe non synonymous variant that inserts bases into in the coding sequence." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001822', 'inframe_deletion', 'An inframe non synonymous variant that deletes bases from the coding sequence." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001823', 'conservative_inframe_insertion', 'An inframe increase in cds length that inserts one or more codons into the coding sequence between existing codons." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001824', 'disruptive_inframe_insertion', 'An inframe increase in cds length that inserts one or more codons into the coding sequence within an existing codon." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001825', 'conservative_inframe_deletion', 'An inframe decrease in cds length that deletes one or more entire codons from the coding sequence but does not change any remaining codons." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001826', 'disruptive_inframe_deletion', 'An inframe decrease in cds length that deletes bases from the coding sequence starting within an existing codon." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001827', 'mRNA_read', 'A sequencer read of an mRNA substrate." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001828', 'genomic_DNA_read', 'A sequencer read of a genomic DNA substrate." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001829', 'mRNA_contig', 'A contig composed of mRNA_reads." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001830', 'AFLP_fragment', 'A PCR product obtained by applying the AFLP technique, based on a restriction enzyme digestion of genomic DNA and an amplification of the resulting fragments." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001831', 'protein_hmm_match', 'A match to a protein HMM such as pfam." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001832', 'immunoglobulin_region', 'A region of immunoglobulin sequence, either constant or variable." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001833', 'V_region', 'The variable region of an immunoglobulin polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001834', 'C_region', 'The constant region of an immunoglobulin polypeptide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001835', 'N_region', 'Extra nucleotides inserted between rearranged immunoglobulin segments." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001836', 'S_region', 'The switch region of immunoglobulin heavy chains; it is involved in the rearrangement of heavy chain DNA leading to the expression of a different immunoglobulin classes from the same B-cell." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001837', 'mobile_element_insertion', 'A kind of insertion where the inserted sequence is a mobile element." [EBI:dvga]');
INSERT INTO soterms (id, name, definition) VALUES ('001838', 'novel_sequence_insertion', 'An insertion the sequence of which cannot be mapped to the reference genome." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('001839', 'CSL_response_element', 'A promoter element with consensus sequence GTGRGAA, bound by CSL (CBF1/RBP-JK/Suppressor of Hairless/LAG-1) transcription factors." [PMID:19101542]');
INSERT INTO soterms (id, name, definition) VALUES ('001840', 'GATA_box', 'A GATA transcription factor element containing the consensus sequence WGATAR (in which W indicates A/T and R indicates A/G)." [PMID:8321208]');
INSERT INTO soterms (id, name, definition) VALUES ('001841', 'polymorphic_pseudogene', 'A pseudogene in the reference genome, though known to be intact in the genomes of other individuals of the same species. The annotation process has confirmed that the pseudogenisation event is not a genomic sequencing error." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, JAX:hd]');
INSERT INTO soterms (id, name, definition) VALUES ('001842', 'AP_1_binding_site', 'A promoter element with consensus sequence TGACTCA, bound by AP-1 and related transcription factors." [PMID:1899230, PMID:3034432, PMID:3125983]');
INSERT INTO soterms (id, name, definition) VALUES ('001843', 'CRE', 'A promoter element with consensus sequence TGACGTCA; bound by the ATF/CREB family of transcription factors." [PMID:11483355, PMID:11483993]');
INSERT INTO soterms (id, name, definition) VALUES ('001844', 'CuRE', 'A promoter element bound by copper ion-sensing transcription factors such as S. cerevisiae Mac1p or S. pombe Cuf1; the consensus sequence is HTHNNGCTGD (more specifically TTTGCKCR in budding yeast)." [PMID:10593913, PMID:9188496, PMID:9211922]');
INSERT INTO soterms (id, name, definition) VALUES ('001845', 'DRE', 'A promoter element with consensus sequence CGWGGWNGMM, bound by transcription factors related to RecA and found in promoters of genes expressed following several types of DNA damage or inhibition of DNA synthesis." [PMID:11073995, PMID:8668127]');
INSERT INTO soterms (id, name, definition) VALUES ('001846', 'FLEX_element', 'A promoter element that has consensus sequence GTAAACAAACAAAM and contains a heptameric core GTAAACA, bound by transcription factors with a forkhead DNA-binding domain." [PMID:10747048, PMID:14871934]');
INSERT INTO soterms (id, name, definition) VALUES ('001847', 'forkhead_motif', 'A promoter element with consensus sequence TTTRTTTACA, bound by transcription factors with a forkhead DNA-binding domain." [PMID:15195092]');
INSERT INTO soterms (id, name, definition) VALUES ('001848', 'homol_D_box', 'A core promoter element that has the consensus sequence CAGTCACA (or its inverted form TGTGACTG), and plays the role of a TATA box in promoters that do not contain a canonical TATA sequence." [PMID:21673110, PMID:7501449, PMID:8458332]');
INSERT INTO soterms (id, name, definition) VALUES ('001849', 'homol_E_box', 'A core promoter element that has the consensus sequence ACCCTACCCT (or its inverted form AGGGTAGGGT), and is found near the homol D box in some promoters that use a homol D box instead of a canonical TATA sequence." [PMID:7501449]');
INSERT INTO soterms (id, name, definition) VALUES ('001850', 'HSE', 'A promoter element that consists of at least three copies of the pentanucleotide NGAAN, bound by the heat shock transcription factor HSF." [PMID:17347150, PMID:8689565]');
INSERT INTO soterms (id, name, definition) VALUES ('001851', 'iron_repressed_GATA_element', 'A GATA promoter element with consensus sequence WGATAA, found in promoters of genes repressed in the presence of iron." [PMID:11956219, PMID:17211681]');
INSERT INTO soterms (id, name, definition) VALUES ('001852', 'mating_type_M_box', 'A promoter element with consensus sequence ACAAT, found in promoters of mating type M-specific genes in fission yeast and bound by the transcription factor Mat1-Mc." [PMID:9233811]');
INSERT INTO soterms (id, name, definition) VALUES ('001853', 'androgen_response_element', 'A non-palindromic sequence found in the promoters of genes whose expression is regulated in response to androgen." [PMID:21796522]');
INSERT INTO soterms (id, name, definition) VALUES ('001854', 'smFISH_probe', 'A smFISH is a probe that binds RNA in a single molecule in situ hybridization experiment." [PMID:18806792]');
INSERT INTO soterms (id, name, definition) VALUES ('001855', 'MCB', 'A promoter element with consensus sequence ACGCGT, bound by the transcription factor complex MBF (MCB-binding factor) and found in promoters of genes expressed during the G1/S transition of the cell cycle." [PMID:16285853]');
INSERT INTO soterms (id, name, definition) VALUES ('001856', 'CCAAT_motif', 'A promoter element with consensus sequence CCAAT, bound by a protein complex that represses transcription in response to low iron levels." [PMID:16963626]');
INSERT INTO soterms (id, name, definition) VALUES ('001857', 'Ace2_UAS', 'A promoter element with consensus sequence CCAGCC, bound by the fungal transcription factor Ace2." [PMID:16678171]');
INSERT INTO soterms (id, name, definition) VALUES ('001858', 'TR_box', 'A promoter element with consensus sequence TTCTTTGTTY, bound an HMG-box transcription factor such as S. pombe Ste11, and found in promoters of genes up-regulated early in meiosis." [PMID:1657709]');
INSERT INTO soterms (id, name, definition) VALUES ('001859', 'STREP_motif', 'A promoter element with consensus sequence CCCCTC, bound by the PKA-responsive zinc finger transcription factor Rst2." [PMID:11739717]');
INSERT INTO soterms (id, name, definition) VALUES ('001860', 'rDNA_intergenic_spacer_element', 'A DNA motif that contains a core consensus sequence AGGTAAGGGTAATGCAC, is found in the intergenic regions of rDNA repeats, and is bound by an RNA polymerase I transcription termination factor (e.g. S. pombe Reb1)." [PMID:9016645]');
INSERT INTO soterms (id, name, definition) VALUES ('001861', 'sterol_regulatory_element', 'A 10-bp promoter element bound by sterol regulatory element binding proteins (SREBPs), found in promoters of genes involved in sterol metabolism. Many variants of the sequence ATCACCCCAC function as SREs." [GO:mah, PMID:11111080, PMID:16537923]');
INSERT INTO soterms (id, name, definition) VALUES ('001862', 'GT_dinucleotide_repeat', 'A dinucleotide repeat region composed of GT repeating elements." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001863', 'GTT_trinucleotide_repeat', 'A trinucleotide repeat region composed of GTT repeating elements." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001864', 'Sap1_recognition_motif', 'A DNA motif to which the S. pombe Sap1 protein binds. The consensus sequence is 5-TARGCAGNTNYAACGMG-3; it is found at the mating type locus, where it is important for mating type switching, and at replication fork barriers in rDNA repeats." [PMID:16166653, PMID:7651412]');
INSERT INTO soterms (id, name, definition) VALUES ('001865', 'CDRE_motif', 'An RNA polymerase II promoter element found in the promoters of genes regulated by calcineurin. The consensus sequence is GNGGCKCA." [PMID:16928959]');
INSERT INTO soterms (id, name, definition) VALUES ('001866', 'BAC_read_contig', 'A contig of BAC reads." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001867', 'candidate_gene', 'A gene suspected of being involved in the expression of a trait." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001868', 'positional_candidate_gene', 'A candidate gene whose association with a trait is based on the genes location on a chromosome." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001869', 'functional_candidate_gene', 'A candidate gene whose function has something in common biologically with the trait under investigation." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001870', 'enhancerRNA', 'A short ncRNA that is transcribed from an enhancer. May have a regulatory function." [doi:10.1038/465173a, SO:cjm]');
INSERT INTO soterms (id, name, definition) VALUES ('001871', 'PCB', 'A promoter element with consensus sequence GNAACR, bound by the transcription factor complex PBF (PCB-binding factor) and found in promoters of genes expressed during the M/G1 transition of the cell cycle." [GO:mah, PMID:12411492]');
INSERT INTO soterms (id, name, definition) VALUES ('001872', 'rearrangement_region', 'A region of a chromosome, where the chromosome has undergone a large structural rearrangement that altered the genome organization. There is no longer synteny to the reference genome." [NCBI:th, PMID:18564416]');
INSERT INTO soterms (id, name, definition) VALUES ('001873', 'interchromosomal_breakpoint', 'A rearrangement breakpoint between two different chromosomes." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('001874', 'intrachromosomal_breakpoint', 'A rearrangement breakpoint within the same chromosome." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('001875', 'unassigned_supercontig', 'A supercontig that is not been assigned to any ultracontig during a genome assembly project." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001876', 'partial_genomic_sequence_assembly', 'A partial DNA sequence assembly of a chromosome or full genome, which contains gaps that are filled with Ns." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001877', 'lnc_RNA', 'A non-coding RNA over 200nucleotides in length." [HGNC:mw]');
INSERT INTO soterms (id, name, definition) VALUES ('001878', 'feature_variant', 'A sequence variant that falls entirely or partially within a genomic feature." [EBI:fc, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001879', 'feature_ablation', 'A sequence variant, caused by an alteration of the genomic sequence, where the deletion, is greater than the extent of the underlying genomic features." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001880', 'feature_amplification', 'A sequence variant, caused by an alteration of the genomic sequence, where the structural change, an amplification of sequence, is greater than the extent of the underlying genomic features." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001881', 'feature_translocation', 'A sequence variant, caused by an alteration of the genomic sequence, where the structural change, a translocation, is greater than the extent of the underlying genomic features." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001882', 'feature_fusion', 'A sequence variant, caused by an alteration of the genomic sequence, where a deletion fuses genomic features." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001883', 'transcript_translocation', 'A feature translocation where the region contains a transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001884', 'regulatory_region_translocation', 'A feature translocation where the region contains a regulatory region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001885', 'TFBS_translocation', 'A feature translocation where the region contains a transcription factor binding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001886', 'transcript_fusion', 'A feature fusion where the deletion brings together transcript regions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001887', 'regulatory_region_fusion', 'A feature fusion where the deletion brings together regulatory regions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001888', 'TFBS_fusion', 'A fusion where the deletion brings together transcription factor binding sites." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001889', 'transcript_amplification', 'A feature amplification of a region containing a transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001890', 'transcript_regulatory_region_fusion', 'A feature fusion where the deletion brings together a regulatory region and a transcript region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001891', 'regulatory_region_amplification', 'A feature amplification of a region containing a regulatory region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001892', 'TFBS_amplification', 'A feature amplification of a region containing a transcription factor binding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001893', 'transcript_ablation', 'A feature ablation whereby the deleted region includes a transcript feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001894', 'regulatory_region_ablation', 'A feature ablation whereby the deleted region includes a regulatory region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001895', 'TFBS_ablation', 'A feature ablation whereby the deleted region includes a transcription factor binding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001896', 'transposable_element_CDS', 'A CDS that is part of a transposable element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001897', 'transposable_element_pseudogene', 'A pseudogene contained within a transposable element." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001898', 'dg_repeat', 'A repeat region which is part of the regional centromere outer repeat region." [PMID:16407326, SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001899', 'dh_repeat', 'A repeat region which is part of the regional centromere outer repeat region." [PMID:16407326, SO:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('001900', 'M26_binding_site', 'A promoter element that contains a core sequence TGACGT, bound by a protein complex that regulates transcription of genes encoding PKA pathway components." [PMID:15448137]');
INSERT INTO soterms (id, name, definition) VALUES ('001901', 'AACCCT_box', 'A conserved 17-bp sequence (5-ATCA(C/A)AACCCTAACCCT-3) commonly present upstream of the start site of histone transcription units functioning as a transcription factor binding site." [PMID:17452352, PMID:4092687]');
INSERT INTO soterms (id, name, definition) VALUES ('001902', 'splice_region', 'A region surrounding a cis_splice site, either within 1-3 bases of the exon or 3-8 bases of the intron." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001903', 'intronic_lncRNA', 'A lnc_RNA totally contained within an intron." [PMID:19071207, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001904', 'antisense_lncRNA', 'Non-coding RNA transcribed from the opposite DNA strand compared with other transcripts and overlap in part with sense RNA." [PMID:19638999]');
INSERT INTO soterms (id, name, definition) VALUES ('001905', 'regional_centromere_outer_repeat_transcript', 'A transcript that is transcribed from the outer repeat region of a regional centromere." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001906', 'feature_truncation', 'A sequence variant that causes the reduction of a genomic feature, with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001907', 'feature_elongation', 'A sequence variant that causes the extension of a genomic feature, with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001908', 'internal_feature_elongation', 'A sequence variant that causes the extension of a genomic feature from within the feature rather than from the terminus of the feature, with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001909', 'frameshift_elongation', 'A frameshift variant that causes the translational reading frame to be extended relative to the reference feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001910', 'frameshift_truncation', 'A frameshift variant that causes the translational reading frame to be shortened relative to the reference feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001911', 'copy_number_increase', 'A sequence variant where copies of a feature are increased relative to the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001912', 'copy_number_decrease', 'A sequence variant where copies of a feature are decreased relative to the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001913', 'bacterial_RNApol_promoter_sigma_ecf', 'A bacterial promoter with sigma ecf factor binding dependency. This is a type of bacterial promoters that requires a sigma ECF factor to bind to identified -10 and -35 sequence regions in order to mediate binding of the RNA polymerase to the promoter region as part of transcription initiation." [Invitrogen:kc]');
INSERT INTO soterms (id, name, definition) VALUES ('001914', 'rDNA_replication_fork_barrier', 'A DNA motif that is found in eukaryotic rDNA repeats, and is a site of replication fork pausing." [PMID:14645529]');
INSERT INTO soterms (id, name, definition) VALUES ('001915', 'transcription_start_cluster', 'A region defined by a cluster of experimentally determined transcription starting sites." [PMID:19624849, PMID:21372179, SO:andrewgibson]');
INSERT INTO soterms (id, name, definition) VALUES ('001916', 'CAGE_tag', 'A CAGE tag is a sequence tag hat corresponds to 5 ends of mRNA at cap sites, produced by cap analysis gene expression and used to identify transcriptional start sites." [SO:andrewgibson]');
INSERT INTO soterms (id, name, definition) VALUES ('001917', 'CAGE_cluster', 'A kind of transcription_initiation_cluster defined by the clustering of CAGE tags on a sequence region." [PMID:16645617, SO:andrewgibson]');
INSERT INTO soterms (id, name, definition) VALUES ('001918', '5_methylcytosine', 'A cytosine methylated at the 5 carbon." [SO:rtapella]');
INSERT INTO soterms (id, name, definition) VALUES ('001919', '4_methylcytosine', 'A cytosine methylated at the 4 nitrogen." [SO:rtapella]');
INSERT INTO soterms (id, name, definition) VALUES ('001920', 'N6_methyladenine', 'An adenine methylated at the 6 nitrogen." [SO:rtapella]');
INSERT INTO soterms (id, name, definition) VALUES ('001921', 'mitochondrial_contig', 'A contig of mitochondria derived sequences." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001922', 'mitochondrial_supercontig', 'A scaffold composed of mitochondrial contigs." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001923', 'TERRA', 'A non-coding RNA transcript, derived from the transcription of the telomere. These  transcripts contain G rich telomeric RNA repeats and RNA tracts corresponding to adjacent subtelomeric sequences. They are 100-9000 bases long." [PMID:22139915]');
INSERT INTO soterms (id, name, definition) VALUES ('001924', 'ARRET', 'A non coding RNA transcript, complementary to subtelomeric tract of TERRA transcript but devoid of the repeats." [PMID:2139915]');
INSERT INTO soterms (id, name, definition) VALUES ('001925', 'ARIA', 'A non-coding RNA transcript, derived from the transcription of the telomere. These  transcripts consist of C rich repeats." [PMID:22139915]');
INSERT INTO soterms (id, name, definition) VALUES ('001926', 'anti_ARRET', 'A non-coding RNA transcript, derived from the transcription of the telomere. These  transcripts are antisense of ARRET transcripts." [PMID:22139915]');
INSERT INTO soterms (id, name, definition) VALUES ('001927', 'telomeric_transcript', 'A non-coding transcript derived from the transcript of the telomere." [PMID:22139915]');
INSERT INTO soterms (id, name, definition) VALUES ('001928', 'distal_duplication', 'A duplication of the distal region of a chromosome." [SO:bm]');
INSERT INTO soterms (id, name, definition) VALUES ('001929', 'mitochondrial_DNA_read', 'A sequencer read of a mitochondrial DNA sample." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001930', 'chloroplast_DNA_read', 'A sequencer read of a chloroplast DNA sample." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001931', 'consensus_gDNA', 'Genomic DNA sequence produced from some base calling or alignment algorithm which uses aligned or assembled multiple gDNA sequences as input." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001932', 'restriction_enzyme_five_prime_single_strand_overhang', 'A terminal region of DNA sequence where the end of the region is not blunt ended and the exposed single strand terminates at the 5 end." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001933', 'restriction_enzyme_three_prime_single_strand_overhang', 'A terminal region of DNA sequence where the end of the region is not blunt ended and the exposed single strand terminates at the 3 end." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001934', 'monomeric_repeat', 'A repeat_region containing repeat_units of 1 bp that is repeated multiple times in tandem." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001935', 'H3K20_trimethylation_site', 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H3 protein is tri-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001936', 'H3K36_acetylation_site', 'A kind of histone modification site, whereby the 36th residue (a lysine), from the start of the H3 histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001937', 'H2BK12_acetylation_site', 'A kind of histone modification site, whereby the 12th residue (a lysine), from the start of the H2B protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001938', 'H2AK5_acetylation_site', 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2A histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001939', 'H4K12_acetylation_site', 'A kind of histone modification site, whereby the 12th residue (a lysine), from the start of the H4 histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001940', 'H2BK120_acetylation_site', 'A kind of histone modification site, whereby the 120th residue (a lysine), from the start of the H2B histone protein is acetylated." [EBI:nj, http://dx.doi.org/10.4161/epi.6.5.15623]');
INSERT INTO soterms (id, name, definition) VALUES ('001941', 'H4K91_acetylation_site', 'A kind of histone modification site, whereby the 91st residue (a lysine), from the start of the H4 histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001942', 'H2BK20_acetylation_site', 'A kind of histone modification site, whereby the 20th residue (a lysine), from the start of the H2B histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001943', 'H3K4_acetylation_site', 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H3 histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001944', 'H2AK9_acetylation_site', 'A kind of histone modification site, whereby the 9th residue (a lysine), from the start of the H2A histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001945', 'H3K56_acetylation_site', 'A kind of histone modification site, whereby the 56th residue (a lysine), from the start of the H3 histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001946', 'H2BK15_acetylation_site', 'A kind of histone modification site, whereby the 15th residue (a lysine), from the start of the H2B histone protein is acetylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001947', 'H3R2_monomethylation_site', 'A kind of histone modification site, whereby the 2nd residue (an arginine), from the start of the H3 protein is mono-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001948', 'H3R2_dimethylation_site', 'A kind of histone modification site, whereby the 2nd residue (an arginine), from the start of the H3 protein is di-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001949', 'H4R3_dimethylation_site', 'A kind of histone modification site, whereby the 3nd residue (an arginine), from the start of the H4 protein is di-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001950', 'H4K4_trimethylation_site', 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H4 protein is tri-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001951', 'H3K23_dimethylation_site', 'A kind of histone modification site, whereby the 23rd residue (a lysine), from the start of the H3 protein is di-methylated." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001952', 'promoter_flanking_region', 'A region immediately adjacent to a promoter which may or may not contain transcription factor binding sites." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001953', 'restriction_enzyme_assembly_scar', 'A region of DNA sequence formed from the ligation of two sticky ends where the palindrome is broken and no longer comprises the recognition site and thus cannot be re-cut by the restriction enzymes used to create the sticky ends." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001954', 'restriction_enzyme_region', 'A region related to restriction enzyme function." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001955', 'protein_stability_element', 'A polypeptide region that proves structure in a protein that affects the stability of the protein." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001956', 'protease_site', 'A polypeptide_region that  codes for a protease cleavage site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001957', 'RNA_stability_element', 'RNA secondary structure that affects the stability of an RNA molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001958', 'lariat_intron', 'A kind of intron whereby the excision is driven by lariat formation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001959', 'TCT_motif', 'A cis-regulatory element, conserved sequence YYC+1TTTYY, and spans -2 to +6 relative to +1 TSS. It is present in most ribosomal protein genes in Drosophila and mammals but not in the yeast Saccharomyces cerevisiae. Resembles the initiator (TCAKTY in Drosophila) but functionally distinct from initiator." [PMID:20801935, SO:myl]');
INSERT INTO soterms (id, name, definition) VALUES ('001960', '5_hydroxymethylcytosine', 'A modified DNA cytosine base feature, modified by a hydroxymethyl group at the 5 carbon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001961', '5_formylcytosine', 'A modified DNA cytosine base feature, modified by a formyl group at the 5 carbon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001962', 'modified_adenine', 'A modified adenine DNA base feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001963', 'modified_cytosine', 'A modified cytosine DNA base feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001964', 'modified_guanine', 'A modified guanine DNA base feature." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001965', '8_oxoguanine', 'A modified DNA guanine base,at the 8 carbon, often the product of DNA damage." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001966', '5_carboxylcytosine', 'A modified DNA cytosine base feature, modified by a carboxy group at the 5 carbon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001967', '8_oxoadenine', 'A modified DNA adenine base,at the 8 carbon, often the product of DNA damage." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001968', 'coding_transcript_variant', 'A transcript variant of a protein coding gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001969', 'coding_transcript_intron_variant', 'A transcript variant occurring within an intron of a coding transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001970', 'non_coding_transcript_intron_variant', 'A transcript variant occurring within an intron of a non coding transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001971', 'zinc_finger_binding_site', 'A transcript variant occurring within an intron of a non coding transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001972', 'histone_4_acetylation_site', 'A histone 4 modification where the modification is the acetylation of the residue." [EBI:nj, ISBN:0815341059, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001973', 'histone_3_acetylation_site', 'A histone 3 modification where the modification is the acetylation of the residue." [EBI:nj, ISBN:0815341059, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001974', 'CTCF_binding_site', 'A transcription factor binding site with consensus sequence CCGCGNGGNGGCAG, bound by CCCTF-binding factor." [EBI:nj]');
INSERT INTO soterms (id, name, definition) VALUES ('001975', 'five_prime_sticky_end_restriction_enzyme_cleavage_site', 'A restriction enzyme recognition site that, when cleaved, results in 5 prime overhangs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001976', 'three_prime_sticky_end_restriction_enzyme_cleavage_site', 'A restriction enzyme recognition site that, when cleaved, results in 3 prime overhangs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001977', 'ribonuclease_site', 'A region of a transcript encoding the cleavage site for a ribonuclease enzyme." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001978', 'signature', 'A region of  sequence where developer information is encoded." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001979', 'RNA_stability_element', 'A motif that affects the stability of RNA." [PMID:22495308, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001980', 'G_box', 'A regulatory promoter element identified in mutation experiments, with consensus sequence: CACGTG. Present in promoters, intergenic regions, coding regions, and introns. They are involved in gene expression responses to light and interact with G-box binding factor and I-box binding factor 1a." [PMID:19249238, PMID:8571452, SO:ml]');
INSERT INTO soterms (id, name, definition) VALUES ('001981', 'L_box', 'An orientation dependent  regulatory promoter element, with consensus sequence of TTGCACAN4TTGCACA, found in plants." [PMID:17381552, PMID:2902624, SO:ml]');
INSERT INTO soterms (id, name, definition) VALUES ('001982', 'I-box', 'A plant regulatory promoter motif,  composed of a highly conserved hexamer GATAAG (I-box core)." [PMID:2347304, PMID:2902624, SO:ml]');
INSERT INTO soterms (id, name, definition) VALUES ('001983', '5_prime_UTR_premature_start_codon_variant', 'A 5 UTR variant where a premature start codon is introduced, moved or lost." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001984', 'silent_mating_type_cassette_array', 'A gene cassette array that corresponds to a silenced version of a mating type region." [PomBase:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('001985', 'Okazaki_fragment', 'Any of the DNA segments produced by discontinuous synthesis of the lagging strand during DNA replication." [ISBN:0805350152]');
INSERT INTO soterms (id, name, definition) VALUES ('001986', 'upstream_transcript_variant', 'A feature variant, where the alteration occurs upstream of the transcript TSS." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001987', 'downstream_transcript_variant', 'A feature variant, where the alteration occurs upstream of the transcript TSS." [EBI:gr]');
INSERT INTO soterms (id, name, definition) VALUES ('001988', '5_prime_UTR_premature_start_codon_gain_variant', 'A 5 UTR variant where a premature start codon is gained." [Sanger:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001989', '5_prime_UTR_premature_start_codon_loss_variant', 'A 5 UTR variant where a premature start codon is lost." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001990', 'five_prime_UTR_premature_start_codon_location_variant', 'A 5 UTR variant where a premature start codon is moved." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001991', 'consensus_AFLP_fragment', 'A consensus AFLP fragment is an AFLP sequence produced from any alignment algorithm which uses assembled multiple AFLP sequences as input." [GMOD:ea]');
INSERT INTO soterms (id, name, definition) VALUES ('001992', 'nonsynonymous_variant', 'A non-synonymous variant is an inframe, protein altering variant, resulting in a codon change." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001993', 'extended_cis_splice_site', 'Intronic positions associated with cis-splicing. Contains the first and second positions immediately before the exon and the first, second and fifth positions immediately after." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001994', 'intron_base_5', 'Fifth intronic position after the intron exon boundary, close to the 5 edge of the intron." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001995', 'extended_intronic_splice_region_variant', 'A sequence variant occurring in the intron, within 10 bases of exon." [sanger:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001996', 'extended_intronic_splice_region', 'Region of intronic sequence within 10 bases of an exon." [SANGER:am]');
INSERT INTO soterms (id, name, definition) VALUES ('001997', 'subtelomere', 'A heterochromatic region of the chromosome,  adjacent to the telomere (on the centromeric side) that contains repetitive DNA and sometimes genes and it is transcribed." [POMBE:al]');
INSERT INTO soterms (id, name, definition) VALUES ('001998', 'sgRNA', 'A small RNA oligo, typically about 20 bases, that guides the cas nuclease to a target DNA sequence in the CRISPR/cas mutagenesis method." [PMID:23934893]');
INSERT INTO soterms (id, name, definition) VALUES ('001999', 'mating_type_region_motif', 'DNA motif that is a component of a mating type region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002001', 'Y_region', 'A segment of non-homology between a and alpha mating alleles, found at all three mating loci (HML, MAT, and HMR), has two forms (Ya and Yalpha)." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002002', 'Z1_region', 'A mating type region motif, one of two segments of homology found at all three mating loci (HML, MAT, and HMR)." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002003', 'Z2_region', 'A mating type region motif, the rightmost segment of homology in the HML and MAT mating loci (not present in HMR)." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002004', 'ARS_consensus_sequence', 'The ACS is an 11-bp sequence of the form 5-WTTTAYRTTTW-3 which is at the core of every yeast ARS, and is necessary but not sufficient for recognition and binding by the origin recognition complex (ORC). Functional ARSs require an ACS, as well as other cis elements in the 5 (C domain) and 3 (B domain) flanking sequences of the ACS." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002005', 'DSR_motif', 'The determinant of selective removal (DSR) motif consists of repeats of U(U/C)AAAC. The motif targets meiotic transcripts for removal during mitosis via the exosome." [PMID:22645662]');
INSERT INTO soterms (id, name, definition) VALUES ('002006', 'zinc_repressed_element', 'A promoter element that has the consensus sequence GNMGATC, and is found in promoters of genes repressed in the presence of zinc." [PMID:24003116, POMBE:mh]');
INSERT INTO soterms (id, name, definition) VALUES ('002007', 'MNV', 'An MNV is a multiple nucleotide variant (substitution) in which the inserted sequence is the same length as the replaced sequence." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002008', 'rare_amino_acid_variant', 'A sequence variant whereby at least one base of a codon encoding a rare amino acid is changed, resulting in a different encoded amino acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002009', 'selenocysteine_loss', 'A sequence variant whereby at least one base of a codon encoding selenocysteine  is changed, resulting in a different encoded amino acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002010', 'pyrrolysine_loss', 'A sequence variant whereby at least one base of a codon encoding pyrrolysine is changed, resulting in a different encoded amino acid." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002011', 'intragenic_variant', 'A variant that occurs within a gene but falls outside of all transcript features. This occurs when alternate transcripts of a gene do not share overlapping  sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002012', 'start_lost', 'A codon variant that changes at least one base of the canonical start codon." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002013', '5_prime_UTR_truncation', 'A sequence variant that causes the reduction of a the 5UTR  with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002014', '5_prime_UTR_elongation', 'A sequence variant that causes the extension of 5 UTR, with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002015', '3_prime_UTR_truncation', 'A sequence variant that causes the reduction of a the 3 UTR  with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002016', '3_prime_UTR_elongation', 'A sequence variant that causes the extension of 3 UTR, with regard to the reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002017', 'conserved_intergenic_variant', 'A sequence variant located in a conserved intergenic region, between genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002018', 'conserved_intron_variant', 'A transcript variant occurring within a conserved region of an intron." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002019', 'start_retained_variant', 'A sequence variant where at least one base in the start codon is changed, but the start remains." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002020', 'boundary_element', 'Boundary elements are DNA motifs that prevent heterochromatin from spreading into neighboring euchromatic regions." [PMID:24013502]');
INSERT INTO soterms (id, name, definition) VALUES ('002021', 'mating_type_region_replication_fork_barrier', 'A DNA motif that is found in eukaryotic rDNA repeats, and is a site of replication fork pausing." [PMID:17614787]');
INSERT INTO soterms (id, name, definition) VALUES ('002022', 'priRNA', 'A small RNA molecule, 22-23 nt in size, that is the product of a longer RNA. The production of priRNAs is independent of dicer and involves binding of RNA by argonaute and trimming by triman. In fission yeast, priRNAs trigger the establishment of heterochromatin. PriRNAs are primarily generated from centromeric transcripts (dg and dh repeats), but may also be produced from degradation products of primary transcripts." [PMID:20178743, PMID:24095277, PomBase:al]');
INSERT INTO soterms (id, name, definition) VALUES ('002023', 'multiplexing_sequence_identifier', 'A nucleic tag which is used in a ligation step of library preparation process to allow pooling of samples while maintaining ability to identify individual source material and creation of a multiplexed library." [OBO:prs, PMID:22574170]');
INSERT INTO soterms (id, name, definition) VALUES ('002024', 'W_region', 'The leftmost segment of homology in the HML and MAT mating loci,  but not present in HMR." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002025', 'cis_acting_homologous_chromosome_pairing_region', 'A genome region where chromosome pairing occurs preferentially during homologous chromosome pairing during early meiotic prophase of Meiosis I." [PMID:22582262, PMID:23117617, PMID:24173580, PomBase:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('002026', 'intein_encoding_region', 'The nucleotide sequence which encodes the intein portion of the precursor gene." [PMID:8165123]');
INSERT INTO soterms (id, name, definition) VALUES ('002027', 'uORF', 'A short open reading frame that is found in the 5 untranslated region of an mRNA and plays a role in translational regulation." [PMID:12890013, PMID:16153175, POMBASE:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('002028', 'sORF', 'An open reading frame that encodes a peptide of less than 100 amino acids." [PMID:23970561, PMID:24705786, POMBASE:mah]');
INSERT INTO soterms (id, name, definition) VALUES ('002029', 'tnaORF', 'A translated ORF encoded entirely within the antisense strand of a known protein coding gene." [POMBASE:vw]');
INSERT INTO soterms (id, name, definition) VALUES ('002030', 'X_region', 'One of two segments of homology found at all three mating loci (HML, MAT and HMR)." [SGD:jd]');
INSERT INTO soterms (id, name, definition) VALUES ('002031', 'shRNA', 'A short hairpin RNA (shRNA) is an RNA transcript that makes a tight hairpin turn that can be used to silence target gene expression via RNA interference." [PMID:6699500, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002032', 'moR', 'A non-coding transcript encoded by sequences adjacent to the ends of the 5 and 3 miR-encoding sequences that abut the loop in precursor miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002033', 'loR', 'A short, non coding transcript of loop-derived sequences encoded in precursor miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002034', 'miR_encoding_snoRNA_primary_transcript', 'A snoRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002035', 'lncRNA_primary_transcript', 'A primary transcript encoding a  lncRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002036', 'miR_encoding_lncRNA_primary_transcript', 'A lncRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002037', 'miR_encoding_tRNA_primary_transcript', 'A tRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002038', 'shRNA_primary_transcript', 'A primary transcript encoding an shRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002039', 'miR_encoding_shRNA_primary_transcript', 'A shRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002040', 'vaultRNA_primary_transcript', 'A primary transcript encoding a  vaultRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002041', 'miR_encoding_vaultRNA_primary_transcript', 'A vaultRNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002042', 'Y_RNA_primary_transcript', 'A primary transcript encoding a  Y-RNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002043', 'miR_encoding_Y_RNA_primary_transcript', 'A Y-RNA primary transcript that also encodes pre-miR sequence that is processed to form functionally active miRNA." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002044', 'TCS_element', 'A TCS element is a (yeast) transcription factor binding site, bound by the TEA DNA binding domain (DBD) of transcription factors. The consensus site is CATTCC or CATTCT." [PMID:1489142, PMID:20118212, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002045', 'pheromone_response_element', 'A PRE is a (yeast) TFBS with consensus site [TGAAAC(A/G)]." [PMID:1489142, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002046', 'FRE', 'A FRE is an enhancer element necessary and sufficient to confer filamentation associated expression in S. cerevisiae." [PMID:1489142, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002047', 'transcription_pause_site', 'Transcription pause sites are regions of a gene where RNA polymerase may pause during transcription. The functional role of pausing may be to facilitate factor recruitment, RNA folding, and synchronization with translation. Consensus transcription pause site have been observed in E. coli." [PMID:24789973, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002048', 'disabled_reading_frame', 'A reading frame that could encode a full-length protein but which contains obvious mid-sequence disablements (frameshifts or premature stop codons)." [SGD:se]');
INSERT INTO soterms (id, name, definition) VALUES ('002049', 'H3K27_acetylation_site', 'A kind of histone modification site, whereby the 27th residue (a lysine), from the start of the H3 histone protein is acetylated." [SO:rs]');
INSERT INTO soterms (id, name, definition) VALUES ('002050', 'constitutive_promoter', 'A promoter that allows for continual transcription of gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002051', 'inducible_promoter', 'A promoter whereby activity is induced by the presence or absence of biotic or abiotic factors." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002052', 'dominant_negative_variant', 'A variant where the mutated gene product adversely affects the other (wild type) gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002053', 'gain_of_function_variant', 'A sequence variant whereby new or enhanced function is conferred on the gene product." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002054', 'loss_of_function_variant', 'A sequence variant whereby the gene product has diminished or abolished function." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002055', 'null_mutation', 'A variant whereby the gene product is not functional or the gene product is not produced." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002056', 'intronic_splicing_silencer', 'An intronic  splicing regulatory element that functions to recruit trans acting splicing factors suppress the transcription of the gene or genes they control." [PMID:23241926, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002057', 'intronic_splicing_enhancer', 'An intronic  splicing regulatory element that functions to recruit trans acting splicing factors suppress the transcription of the gene or genes they control." [PMID:23241926, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002058', 'exonic_splicing_silencer', 'An exonic  splicing regulatory element that functions to recruit trans acting splicing factors suppress the transcription of the gene or genes they control." [PMID:23241926, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002059', 'recombination_enhancer', 'A regulatory_region that promotes or induces the process of recombination." [PMID:8861911, SGD:se]');
INSERT INTO soterms (id, name, definition) VALUES ('002060', 'interchromosomal_translocation', 'A translocation where the regions involved are from different chromosomes." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002061', 'intrachromosomal_translocation', 'A translocation where the regions involved are from the same chromosome." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002062', 'complex_chromosomal_rearrangement', 'A contiguous cluster of translocations, usually the result of a single catastrophic event such as chromothripsis or chromoanasynthesis." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002063', 'Alu_insertion', 'An insertion of sequence from the Alu family of mobile elements." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002064', 'LINE1_insertion', 'An insertion from the Line1 family of mobile elements." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002065', 'SVA_insertion', 'An insertion of sequence from the SVA family of mobile elements." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002066', 'mobile_element_deletion', 'A deletion of a mobile element when comparing a reference sequence (has mobile element) to a individual sequence (does not have mobile element)." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002067', 'HERV_deletion', 'A deletion of the HERV mobile element with respect to  a reference." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002068', 'SVA_deletion', 'A deletion of an SVA mobile element." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002069', 'LINE1_deletion', 'A deletion of a LINE1 mobile element with respect to a reference." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002070', 'Alu_deletion', 'A deletion of an Alu mobile element with respect to a reference." [NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('002071', 'CDS_supported_by_peptide_spectrum_match', 'A CDS that is supported by proteomics data." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002072', 'sequence_comparison', 'A CDS that is supported by proteomics data." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002073', 'no_sequence_alteration', 'A position or feature within a sequence that is identical to the comparable position or feature of a specified reference sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002074', 'intergenic_1kb_variant', 'A variant that falls in an intergenic region that is 1 kb or less between 2 genes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002075', 'incomplete_transcript_variant', 'A sequence variant that intersects an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002076', 'incomplete_transcript_3UTR_variant', 'A sequence variant that intersects the 3 UTR of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002077', 'incomplete_transcript_5UTR_variant', 'A sequence variant that intersects the 5 UTR of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002078', 'incomplete_transcript_intronic_variant', 'A sequence variant that intersects the intron of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002079', 'incomplete_transcript_splice_region_variant', 'A sequence variant that intersects the splice region of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002080', 'incomplete_transcript_exonic_variant', 'A sequence variant that intersects the exon of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002081', 'incomplete_transcript_CDS', 'A sequence variant that intersects the coding regions of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002082', 'incomplete_transcript_coding_splice_variant', 'A sequence variant that intersects the coding sequence near a splice region of an incompletely annotated transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002083', '2KB_downstream_variant', 'A sequence variant located within 2KB 3 of a gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002084', 'exonic_splice_region_variant', 'A sequence variant in which a change has occurred within the exonic region of the splice site, 1-2 bases from boundary." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002085', 'unidirectional_gene_fusion', 'A sequence variant whereby two genes, on the same strand have become joined." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002086', 'bidirectional_gene_fusion', 'A sequence variant whereby two genes, on alternate strands have become joined." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002087', 'pseudogenic_CDS', 'A non functional descendant of the coding portion of a coding transcript, part of a pseudogene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002088', 'non_coding_transcript_splice_region_variant', 'A transcript variant occurring within the splice region (1-3 bases of the exon or 3-8 bases of the intron) of a non coding transcript." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002089', '3_prime_UTR_exon_variant', 'A UTR variant of exonic sequence of the 3 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002090', '3_prime_UTR_intron_variant', 'A UTR variant of intronic sequence of the 3 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002091', '5_prime_UTR_intron_variant', 'A UTR variant of intronic sequence of the 5 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002092', '5_prime_UTR_exon_variant', 'A UTR variant of exonic sequence of the 5 UTR." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002093', 'structural_interaction_variant', 'A variant that impacts the internal interactions of the resulting polypeptide structure." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002094', 'non_allelic_homologous_recombination_region', 'A genomic region at a non-allelic position where exchange of genetic material happens as a result of homologous recombination." []');
INSERT INTO soterms (id, name, definition) VALUES ('002095', 'scaRNA', 'A ncRNA, specific to the Cajal body, that has been demonstrated to function as a guide RNA in the site-specific synthesis of 2-O-ribose-methylated nucleotides and pseudouridines in the RNA polymerase II-transcribed U1, U2, U4 and U5 spliceosomal small nuclear RNAs (snRNAs)." [PMC:126017, SO:nrs]');
INSERT INTO soterms (id, name, definition) VALUES ('002096', 'short_tandem_repeat_variation', 'A variation that expands or contracts a tandem repeat with regard to a reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002097', 'vertebrate_immune_system_pseudogene', 'A pseudogene derived from a vertebrate immune system gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002098', 'immunoglobulin_pseudogene', 'A pseudogene derived from an immunoglobulin gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002099', 'T_cell_receptor_pseudogene', 'A pseudogene derived from a T-cell receptor gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002100', 'IG_C_pseudogene', 'A pseudogenic constant region of an immunoglobulin gene which closely resembles a known functional Imunoglobulin constant gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002101', 'IG_J_pseudogene', 'A pseudogenic joining region which closely resembles a known functional imunoglobulin joining gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002102', 'IG_V_pseudogene', 'A pseudogenic variable region which closely resembles a known functional imunoglobulin variable gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the variable region of  an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002103', 'TR_V_pseudogene', 'A pseudogenic variable region which closely resembles a known functional T receptor variable gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the variable region of  an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002104', 'TR_J_pseudogene', 'A pseudogenic joining region which closely resembles a known functional T receptor (TR) joining gene but in which the coding region has stop codons, frameshift mutations or a mutation that effects the initiation codon that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002105', 'translated_processed_pseudogene', 'A processed pseudogene where there is evidence, (mass spec data) suggesting that it is also translated." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002106', 'translated_unprocessed_pseudogene', 'A non-processed pseudogene where there is evidence, (mass spec data) suggesting that it is also translated." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002107', 'transcribed_unprocessed_pseudogene', 'A unprocessed pseudogene supported by locus-specific evidence of transcription." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002108', 'transcribed_unitary_pseudogene', 'A species specific unprocessed pseudogene without a parent gene, as it has an active orthologue in another species." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002109', 'transcribed_processed_pseudogene', 'A processed_pseudogene overlapped by locus-specific evidence of transcription." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002110', 'polymorphic_pseudogene_with_retained_intron', 'A polymorphic  pseudogene in the reference genome, containing a retained intron,  known to be intact in the genomes of other individuals of the same species. The annotation process has confirmed that the pseudogenisation event is not a genomic sequencing error." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002111', 'pseudogene_processed_transcript', 'A processed_transcript supported by EST and/or mRNA evidence that aligns unambiguously to a pseudogene locus (i.e. alignment to the pseudogene locus clearly better than alignment to parent locus)." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002112', 'coding_transcript_with_retained_intron', 'A protein coding transcript containing a retained intron." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002113', 'lncRNA_with_retained_intron', 'A lncRNA transcript containing a retained intron." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002114', 'NMD_transcript', 'A protein coding transcript that contains a CDS but has one or more splice junctions >50bp downstream of stop codon, making it susceptible to nonsense mediated decay." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002115', 'pseudogenic_transcript_with_retained_intron', 'A transcript supported by EST and/or mRNA evidence that aligns unambiguously to the pseudogene locus; has retained intronic sequence compared to a reference transcript sequence." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002116', 'polymorphic_pseudogene_processed_transcript', 'A processed transcript that does not contain a CDS that fullfills annotation criteria and not necessarily functionally non-coding." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002117', '<new term>', 'A processed transcript that does not contain a CDS that fullfills annotation criteria and not necessarily functionally non-coding." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002118', 'NMD_polymorphic_pseudogene_transcript', 'A polymorphic pseudogene transcript that contains a CDS but has one or more splice junctions >50bp downstream of stop codon. Premature stop codon is not introduced, directly or indirectly, as a result of the variation i.e. must be present in both protein_coding and pseudogenic alleles." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002119', 'allelic_frequency', 'A physical quality which inheres to the allele by virtue of the number instances of the allele within a population. This is the relative frequency of the allele at a given locus in a population." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002120', 'three_prime_overlapping_ncrna', 'Transcript where ditag (digital gene expression profiling)and/or published experimental data strongly supports the existence of short non-coding transcripts transcribed from the 3UTR." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002121', 'vertebrate_immune_system_gene', 'The configuration of the  IG and TR variable (V), diversity (D) and joining (J) germline genes before DNA rearrangements  (with or without constant (C) genes in undefined configuration. (germline, non rearranged regions of the IG DNA loci)." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002122', 'immunoglobulin_gene', 'A germline immunoglobulin  gene." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002123', 'IG_C_gene', 'A constant (C) gene, a gene that codes the constant region of an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002124', 'IG_D_gene', 'A gene that rearranges at the DNA level and codes the diversity region of the variable domain of an immunoglobuin (IG) gene." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002125', 'IG_J_gene', 'A joining gene that rearranges at the DNA level and codes the joining region of the variable domain of an immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002126', 'IG_V_gene', 'A variable gene that rearranges at the DNA level and codes the variable region of the variable domain of an Immunoglobulin chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002127', 'lncRNA_gene', 'A gene that encodes a long non-coding RNA." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002128', 'mt_rRNA', 'Mitochondrial ribosomal RNA." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002129', 'mt_tRNA', 'Mitochondrial transfer RNA." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002130', 'NSD_transcript', 'A transcript that contains a CDS but has no stop codon before the polyA site is reached." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002131', 'sense_intronic_ncRNA', 'A non-coding transcript found within an intron of a coding or non-coding gene, with no overlap of exonic sequence." [GENECODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002132', 'sense_overlap_ncRNA', 'A non-coding transcript that contains a protein coding gene within its intronic sequence on the same strand, with no overlap of exonic sequence." [GENECODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002133', 'T_cell_receptor_gene', 'A T-cell receptor germline gene." []');
INSERT INTO soterms (id, name, definition) VALUES ('002134', 'TR_C_Gene', 'A constant (C) gene, a gene that codes the constant region of a T-cell receptor chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002135', 'TR_D_Gene', 'A gene that rearranges at the DNA level and codes the diversity region of the variable domain of aT-cell receptor gene." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002136', 'TR_J_Gene', 'A joining gene that rearranges at the DNA level and codes the joining region of the variable domain of aT-cell receptor chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002137', 'TR_V_Gene', 'A variable gene that rearranges at the DNA level and codes the variable region of the variable domain of aT-cell receptor chain." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html, IGMT:http\://www.imgt.org/IMGTScientificChart/SequenceDescription/Keywords.php]');
INSERT INTO soterms (id, name, definition) VALUES ('002138', 'predicted_transcript', 'A transcript feature that has been predicted but is not yet validated." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002139', 'unconfirmed_transcript', 'This is used for non-spliced EST clusters that have polyA features. This category has been specifically created for the ENCODE project to highlight regions that could indicate the presence of protein coding genes that require experimental validation, either by 5 RACE or RT-PCR to extend the transcripts, or by confirming expression of the putatively-encoded peptide with specific antibodies." [GENCODE:http\://www.gencodegenes.org/gencode_biotypes.html]');
INSERT INTO soterms (id, name, definition) VALUES ('002140', 'early_origin_of_replication', 'An origin of replication that initiates early in S phase." [PMID:23348837, PMID:9115207]');
INSERT INTO soterms (id, name, definition) VALUES ('002141', 'late_origin_of_replication', 'An origin of replication that initiates late in S phase." [PMID:23348837, PMID:9115207]');
INSERT INTO soterms (id, name, definition) VALUES ('002142', 'histone_2A_acetylation_site', 'A histone 2A modification where the modification is the acetylation of the residue." [ISBN:0815341059]');
INSERT INTO soterms (id, name, definition) VALUES ('002143', 'histone_2B_acetylation_site', 'A histone 2B modification where the modification is the acetylation of the residue." [ISBN:0815341059]');
INSERT INTO soterms (id, name, definition) VALUES ('002144', 'histone_2AZ_acetylation_site', 'A histone 2AZ modification where the modification is the acetylation of the residue." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002145', 'H2AZK4_acetylation_site', 'A kind of histone modification site, whereby the 4th residue (a lysine), from the start of the H2AZ histone protein is acetylated." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002146', 'H2AZK7_acetylation_site', 'A kind of histone modification site, whereby the 7th residue (a lysine), from the start of the H2AZ histone protein is acetylated." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002147', 'H2AZK11_acetylation_site', 'A kind of histone modification site, whereby the 11th residue (a lysine), from the start of the H2AZ histone protein is acetylated." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002148', 'H2AZK13_acetylation_site', 'A kind of histone modification site, whereby the 13th residue (a lysine), from the start of the H2AZ histone protein is acetylated." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002149', 'H2AZK15_acetylation_site', 'A kind of histone modification site, whereby the 15th residue (a lysine), from the start of the H2AZ histone protein is acetylated." [PMID:19385636, PMID:24316985, PMID:27087541]');
INSERT INTO soterms (id, name, definition) VALUES ('002150', 'AUG_initiated_uORF', 'A uORF beginning with the canonical start codon AUG." [PMID:26684391, PMID:27313038]');
INSERT INTO soterms (id, name, definition) VALUES ('002151', 'non_AUG_initiated_uORF', 'A uORF beginning with a codon other than AUG." [PMID:26684391, PMID:27313038]');
INSERT INTO soterms (id, name, definition) VALUES ('002152', 'genic_downstream_transcript_variant', 'A variant that falls downstream of a transcript, but within the genic region of the gene due to alternately transcribed isoforms." [NCBI:dm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002153', 'genic_upstream_transcript_variant', 'A variant that falls upstream of a transcript, but within the genic region of the gene due to alternately transcribed isoforms." [NCBI:dm, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002154', 'mitotic_recombination_region', 'A genomic region where there is an exchange of genetic material with another genomic region, occurring in somatic cells." [NCBI:cf, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002155', 'meiotic_recombination_region', 'A genomic region in which there is an exchange of genetic material as a result of the repair of meiosis-specific double strand breaks that occur during meiotic prophase." [NCBI:cf, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('002156', 'CArG_box', 'A promoter element bound by the MADS family of transcription factors with consensus 5-(C/T)TA(T/A)4TA(G/A)-3." [PMID:1748287, PMID:7623803]');
INSERT INTO soterms (id, name, definition) VALUES ('002157', 'Mat2P', 'A gene cassette array containing H+ mating type specific information." [PMID:18354497]');
INSERT INTO soterms (id, name, definition) VALUES ('002158', 'Mat3M', 'A gene cassette array containing H- mating type specific information." [PMID:18354497]');
INSERT INTO soterms (id, name, definition) VALUES ('002159', 'SHP_box', 'A conserved Cdc48/p97 interaction motif with strict consensus sequence F[PI]GKG[TK][RK]LG[GT] and relaxed consensus sequence FXGKGX[RK]LG." [PMID:17083136, PMID:27655872]');
INSERT INTO soterms (id, name, definition) VALUES ('002160', 'sequence_length_variant', 'A sequence variant that changes the length of one or more sequence features." []');
INSERT INTO soterms (id, name, definition) VALUES ('002161', 'short_tandem_repeat_change', 'A sequence variant where the copies of a short tandem repeat (STR) feature are either contracted or expanded." []');
INSERT INTO soterms (id, name, definition) VALUES ('002162', 'short_tandem_repeat_expansion', 'A short tandem repeat variant containing more repeat units than the reference sequence." []');
INSERT INTO soterms (id, name, definition) VALUES ('002163', 'short_tandem_repeat_contraction', 'A short tandem repeat variant containing fewer repeat units than the reference sequence." []');
INSERT INTO soterms (id, name, definition) VALUES ('002164', 'H2BK5_acetylation_site', 'A kind of histone modification site, whereby the 5th residue (a lysine), from the start of the H2B histone protein is acetylated." [http://www.actrec.gov.in/histome/ptm_sp.php?ptm_sp=H2BK5ac, PMID:18552846]');
INSERT INTO soterms (id, name, definition) VALUES ('002165', 'trinucleotide_repeat_expansion', 'A short tandem repeat expansion with an increase in a sequence of three nucleotide units repeated in tandem compared to a reference sequence." []');
INSERT INTO soterms (id, name, definition) VALUES ('002166', 'ref_miRNA', 'A ref_miRNA (RefSeq-miRNA) sequence is assigned at the creation of a new mature miRNA entry in a database. The ref_miRNA sequence designation remains unchanged even if a different isomiR is later shown to be expressed at a higher level. A ref_miRNA can be produced by one or multiple pre-miRNA." [PMID:26453491]');
INSERT INTO soterms (id, name, definition) VALUES ('002167', 'isomiR', 'IsomiRs are all the bona fide variants of a mature product. IsomiRs should be connected to the ref_miRNA it is most likely to be the variant of. Some isomiRs can be variations of one or multiple ref_miRNA." [PMID:26453491]');
INSERT INTO soterms (id, name, definition) VALUES ('002168', 'RNA_thermometer', 'An RNA_thermometer is a cis element in the 5 end of an mRNA that can change its secondary structure in response to temperature and coordinate temperature-dependent gene expression." [PMID:22421878]');
INSERT INTO soterms (id, name, definition) VALUES ('002169', 'splice_polypyrimidine_tract_variant', 'A sequence variant that falls in the polypyrimidine tract at 3 end of intron between 17 and 3 bases from the end (acceptor -3 to acceptor -17)." []');
INSERT INTO soterms (id, name, definition) VALUES ('002170', 'splice_donor_region_variant', 'A sequence variant that falls in the region between the 3rd and 6th base after splice junction (5 end of intron)." []');
INSERT INTO soterms (id, name, definition) VALUES ('002171', 'telomeric_D_loop', 'A telomeric D-loop is a three-stranded DNA displacement loop that forms at the site where the telomeric 3 single-stranded DNA overhang (formed of the repeat sequence TTAGGG in mammals) is tucked back inside the double-stranded component of telomeric DNA molecule, thus forming a t-loop or telomeric-loop and protecting the chromosome terminus." [PMID:10338204, PMID:15071557, PMID:24012755]');
INSERT INTO soterms (id, name, definition) VALUES ('005836', 'regulatory_region', 'A region of sequence that is involved in the control of a biological process." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('005837', 'U14_snoRNA_primary_transcript', 'The primary transcript of an evolutionarily conserved eukaryotic low molecular weight RNA capable of intermolecular hybridization with both homologous and heterologous 18S rRNA." [PMID:2251119]');
INSERT INTO soterms (id, name, definition) VALUES ('005841', 'methylation_guide_snoRNA', 'A snoRNA that specifies the site of 2-O-ribose methylation in an RNA molecule by base pairing with a short sequence around the target residue." [GOC:mah, PMID:12457565]');
INSERT INTO soterms (id, name, definition) VALUES ('005843', 'rRNA_cleavage_RNA', 'An ncRNA that is part of a ribonucleoprotein that cleaves the primary pre-rRNA transcript in the process of producing mature rRNA molecules." [GOC:kgc]');
INSERT INTO soterms (id, name, definition) VALUES ('005845', 'exon_of_single_exon_gene', 'An exon that is the only exon in a gene." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('005847', 'cassette_array_member', 'An exon that is the only exon in a gene." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('005848', 'gene_cassette_member', 'An exon that is the only exon in a gene." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('005849', 'gene_subarray_member', 'An exon that is the only exon in a gene." [RSC:cb]');
INSERT INTO soterms (id, name, definition) VALUES ('005850', 'primer_binding_site', 'Non-covalent primer binding site for initiation of replication, transcription, or reverse transcription." [http://www.ebi.ac.uk/embl/Documentation/FT_definitions/feature_table.html]');
INSERT INTO soterms (id, name, definition) VALUES ('005851', 'gene_array', 'An array includes two or more genes, or two or more gene subarrays, contiguously arranged where the individual genes, or subarrays, are either identical in sequence, or essentially so." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('005852', 'gene_subarray', 'A subarray is, by defintition, a member of a gene array (SO:0005851); the members of a subarray may differ substantially in sequence, but are closely related in function." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('005853', 'gene_cassette', 'A gene that can be substituted for a related gene at a different site in the genome." [SGD:se]');
INSERT INTO soterms (id, name, definition) VALUES ('005854', 'gene_cassette_array', 'An array of non-functional genes whose members, when captured by recombination form functional genes." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('005855', 'gene_group', 'A collection of related genes." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('005856', 'selenocysteine_tRNA_primary_transcript', 'A primary transcript encoding seryl tRNA (SO:000269)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('005857', 'selenocysteinyl_tRNA', 'A tRNA sequence that has a selenocysteine anticodon, and a 3 selenocysteine binding region." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('005858', 'syntenic_region', 'A region in which two or more pairs of homologous markers occur on the same chromosome in two or more species." [http://www.informatics.jax.org/silverbook/glossary.shtml]');
INSERT INTO soterms (id, name, definition) VALUES ('100001', 'biochemical_region_of_peptide', 'A region of a peptide that is involved in a biochemical function." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100002', 'molecular_contact_region', 'A region that is involved a contact with another molecule." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100003', 'intrinsically_unstructured_polypeptide_region', 'A region of polypeptide chain with high conformational flexibility." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100004', 'catmat_left_handed_three', 'A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('100005', 'catmat_left_handed_four', 'A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170.  The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('100006', 'catmat_right_handed_three', 'A motif of 3 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -75 bounds -100 to -50, res i+1: psi 140 bounds 110 to 170. An extra restriction of the length of the O to O distance would be useful, that it be less than 5 Angstrom. More precisely these two oxygens are the main chain carbonyl oxygen atoms of residues i-1 and i+1." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('100007', 'catmat_right_handed_four', 'A motif of 4 consecutive residues with dihedral angles as follows: res i: phi -90 bounds -120 to -60, res i: psi -10 bounds -50 to 30, res i+1: phi -90 bounds -120 to -60, res i+1: psi -10 bounds -50 to 30, res i+2: phi -75 bounds -100 to -50, res i+2: psi 140 bounds 110 to 170. The extra restriction of the length of the O to O distance is similar, that it be less than 5 Angstrom. In this case these two Oxygen atoms are the main chain carbonyl oxygen atoms of residues i-1 and i+2." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('100008', 'alpha_beta_motif', 'A motif of five consecutive residues and two H-bonds in which: H-bond between CO of residue(i) and NH of residue(i+4), H-bond between CO of residue(i) and NH of residue(i+3),Phi angles of residues(i+1), (i+2) and (i+3) are negative." [EBIBS:GAR, http://www.ebi.ac.uk/msd-srv/msdmotif/]');
INSERT INTO soterms (id, name, definition) VALUES ('100009', 'lipoprotein_signal_peptide', 'A peptide that acts as a signal for both membrane translocation and lipid attachment in prokaryotes." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100010', 'no_output', 'An experimental region wherean analysis has been run and not produced any annotation." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100011', 'cleaved_peptide_region', 'The cleaved_peptide_region is the region of a peptide sequence that is cleaved during maturation." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100012', 'peptide_coil', 'Irregular, unstructured regions of a proteins backbone, as distinct from the regular region (namely alpha helix and beta strand - characterised by specific patterns of main-chain hydrogen bonds)." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100013', 'hydrophobic_region_of_peptide', 'Hydrophobic regions are regions with a low affinity for water." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100014', 'n_terminal_region', 'The amino-terminal positively-charged region of a signal peptide (approx 1-5 aa)." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100015', 'c_terminal_region', 'The more polar, carboxy-terminal region of the signal peptide (approx 3-7 aa)." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100016', 'central_hydrophobic_region_of_signal_peptide', 'The central, hydrophobic region of the signal peptide (approx 7-15 aa)." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100017', 'polypeptide_conserved_motif', 'A conserved motif is a short (up to 20 amino acids) region of biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100018', 'polypeptide_binding_motif', 'A polypeptide binding motif is a short (up to 20 amino acids) polypeptide region of biological interest that contains one or more amino acids experimentally shown to bind to a ligand." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100019', 'polypeptide_catalytic_motif', 'A polypeptide catalytic motif is a short (up to 20 amino acids) polypeptide region that contains one or more active site residues." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('100020', 'polypeptide_DNA_contact', 'A binding site that, in the polypeptide molecule, interacts selectively and non-covalently with DNA." [EBIBS:GAR, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('100021', 'polypeptide_conserved_region', 'A subsection of sequence with biological interest that is conserved in different proteins. They may or may not have functional or structural significance within the proteins in which they are found." [EBIBS:GAR]');
INSERT INTO soterms (id, name, definition) VALUES ('000002', 'substitution', 'A sequence alteration where the length of the change in the variant is the same as that of the reference." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000005', 'complex_substitution', 'When no simple or well defined DNA mutation event describes the observed DNA change, the keyword \"complex\" should be used. Usually there are multiple equally plausible explanations for the change." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000008', 'point_mutation', 'A single nucleotide change which has occurred at the same position of a corresponding nucleotide in a reference sequence." [SO:immuno_workshop]');
INSERT INTO soterms (id, name, definition) VALUES ('000009', 'transition', 'Change of a pyrimidine nucleotide, C or T, into an other pyrimidine nucleotide, or change of a purine nucleotide, A or G, into an other purine nucleotide." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000010', 'pyrimidine_transition', 'A substitution of a pyrimidine, C or T, for another pyrimidine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000011', 'C_to_T_transition', 'A transition of a cytidine to a thymine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000012', 'C_to_T_transition_at_pCpG_site', 'The transition of cytidine to thymine occurring at a pCpG site as a consequence of the spontaneous deamination of 5-methylcytidine." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000013', 'T_to_C_transition', 'The transition of cytidine to thymine occurring at a pCpG site as a consequence of the spontaneous deamination of 5-methylcytidine." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000014', 'purine_transition', 'A substitution of a purine, A or G, for another purine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000015', 'A_to_G_transition', 'A transition of an adenine to a guanine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000016', 'G_to_A_transition', 'A transition of a guanine to an adenine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000017', 'transversion', 'Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G, or vice versa." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000018', 'pyrimidine_to_purine_transversion', 'Change of a pyrimidine nucleotide, C or T, into a purine nucleotide, A or G." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000019', 'C_to_A_transversion', 'A transversion from cytidine to adenine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000020', 'C_to_G_transversion', 'A transversion from cytidine to adenine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000021', 'T_to_A_transversion', 'A transversion from T to A." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000022', 'T_to_G_transversion', 'A transversion from T to G." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000023', 'purine_to_pyrimidine_transversion', 'Change of a purine nucleotide, A or G , into a pyrimidine nucleotide C or T." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000024', 'A_to_C_transversion', 'A transversion from adenine to cytidine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000025', 'A_to_T_transversion', 'A transversion from adenine to thymine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000026', 'G_to_C_transversion', 'A transversion from guanine to cytidine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000027', 'G_to_T_transversion', 'A transversion from guanine to thymine." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000028', 'intrachromosomal_mutation', 'A chromosomal structure variation within a single chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000029', 'chromosomal_deletion', 'An incomplete chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000030', 'chromosomal_inversion', 'An interchromosomal mutation where a region of the chromosome is inverted with respect to wild type." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000031', 'interchromosomal_mutation', 'A chromosomal structure variation whereby more than one chromosome is involved." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000032', 'indel', 'A sequence alteration which included an insertion and a deletion, affecting 2 or more bases." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html, http:http\://www.hgvs.org/mutnomen/recs-DNA.html#indel]');
INSERT INTO soterms (id, name, definition) VALUES ('000035', 'duplication', 'An insertion which derives from, or is identical in sequence to, nucleotides present at a known location in the genome." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html, NCBI:th]');
INSERT INTO soterms (id, name, definition) VALUES ('000036', 'inversion', 'A continuous nucleotide sequence is inverted in the same position." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000037', 'chromosomal_duplication', 'An extra chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000038', 'intrachromosomal_duplication', 'A duplication that occurred within a chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000039', 'direct_tandem_duplication', 'A tandem duplication where the individual regions are in the same orientation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000040', 'inverted_tandem_duplication', 'A tandem duplication where the individual regions are not in the same orientation." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000041', 'intrachromosomal_transposition', 'A chromosome structure variation whereby a transposition occurred within a chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000042', 'compound_chromosome', 'A chromosome structure variant where a monocentric element is caused by the fusion of two chromosome arms." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000043', 'Robertsonian_fusion', 'A non reciprocal translocation whereby the participating chromosomes break at their centromeres and the long arms fuse to form a single chromosome with a single centromere." [http://en.wikipedia.org/wiki/Robertsonian_translocation]');
INSERT INTO soterms (id, name, definition) VALUES ('000044', 'chromosomal_translocation', 'A chromosomal mutation. Rearrangements that alter the pairing of telomeres are classified as translocations." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000045', 'ring_chromosome', 'A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome." [http://en.wikipedia.org/wiki/Ring_chromosome]');
INSERT INTO soterms (id, name, definition) VALUES ('000046', 'pericentric_inversion', 'A chromosomal inversion that includes the centromere." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000047', 'paracentric_inversion', 'A chromosomal inversion that does not include the centromere." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000048', 'reciprocal_chromosomal_translocation', 'A chromosomal translocation with two breaks; two chromosome segments have simply been exchanged." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000049', 'sequence_variation_affecting_transcript', 'Any change in mature, spliced and processed, RNA that results from a change in the corresponding DNA sequence." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000050', 'sequence_variant_causing_no_change_in_transcript', 'No effect on the state of the RNA." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000054', 'sequence_variation_affecting_coding_sequence', 'Any of the amino acid coding triplets of a gene are affected by the DNA mutation." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000055', 'sequence_variant_causing_initiator_codon_change_in_transcript', 'The DNA mutation changes, usually destroys, the first coding triplet of a gene. Usually prevents translation although another initiator codon may be used." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000056', 'sequence_variant_causing_amino_acid_coding_codon_change_in_transcript', 'The DNA mutation affects the amino acid coding sequence of a gene; this region includes both the initiator and terminator codons." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000057', 'sequence_variant_causing_synonymous_codon_change_in_transcript', 'The changed codon has the same translation product as the original codon." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000058', 'sequence_variant_causing_non_synonymous_codon_change_in_transcript', 'A DNA point mutation that causes a substitution of an amino acid by an other." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000059', 'sequence_variant_causing_missense_codon_change_in_transcript', 'The nucleotide change in the codon leads to a new codon coding for a new amino acid." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000060', 'sequence_variant_causing_conservative_missense_codon_change_in_transcript', 'The amino acid change following from the codon change does not change the gross properties (size, charge, hydrophobicity) of the amino acid at that position." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000061', 'sequence_variant_causing_nonconservative_missense_codon_change_in_transcript', 'The amino acid change following from the codon change changes the gross properties (size, charge, hydrophobicity) of the amino acid in that position." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000062', 'sequence_variant_causing_nonsense_codon_change_in_transcript', 'The nucleotide change in the codon triplet creates a terminator codon." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000063', 'sequence_variant_causing_terminator_codon_change_in_transcript', 'The nucleotide change in the codon triplet changes the stop codon, causing an elongated transcript sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000064', 'sequence_variation_affecting_reading_frame', 'An umbrella term for terms describing an effect of a sequence variation on the frame of translation." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000065', 'frameshift_sequence_variation', 'A mutation causing a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000066', 'sequence_variant_causing_plus_1_frameshift_mutation', 'A mutation causing a disruption of the translational reading frame, due to the insertion of a nucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000067', 'sequence_variant_causing_minus_1_frameshift', 'A mutation causing a disruption of the translational reading frame, due to the deletion of a nucleotide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000068', 'sequence_variant_causing_plus_2_frameshift', 'A mutation causing a disruption of the translational reading frame, due to the insertion of two nucleotides." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000069', 'sequence_variant_causing_minus_2_frameshift', 'A mutation causing a disruption of the translational reading frame, due to the deletion of two nucleotides." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000070', 'sequence_variant_affecting_transcript_processing', 'Sequence variant affects the way in which the primary transcriptional product is processed to form the mature transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000071', 'sequence_variant_affecting_splicing', 'A sequence_variant_effect where the way in which the primary transcriptional product is processed to form the mature transcript, specifically by the removal (splicing) of intron sequences is changed." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000072', 'sequence_variant_affecting_splice_donor', 'A sequence_variant_effect that changes the splice donor sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000073', 'sequence_variant_affecting_splice_acceptor', 'A sequence_variant_effect that changes the splice acceptor sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000074', 'sequence_variant_causing_cryptic_splice_activation', 'A sequence variant causing a new (functional) splice site." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000075', 'sequence_variant_affecting_editing', 'Sequence variant affects the editing of the transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000076', 'sequence_variant_affecting_transcription', 'Mutation affects the process of transcription, its initiation, progression or termination." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000078', 'sequence_variant_decreasing_rate_of_transcription', 'A sequence variation that decreases the rate a which transcription of the sequence occurs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000079', 'sequence_variation_affecting_transcript_sequence', 'A sequence variation that decreases the rate a which transcription of the sequence occurs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000080', 'sequence_variant_increasing_rate_of_transcription', 'A sequence variation that decreases the rate a which transcription of the sequence occurs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000081', 'sequence_variant_affecting_rate_of_transcription', 'A mutation that alters the rate a which transcription of the sequence occurs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000082', 'sequence variant_affecting_transcript_stability', 'Sequence variant affects the stability of the transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000083', 'sequence_variant_increasing_transcript_stability', 'Sequence variant increases the stability (half-life) of the transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000084', 'sequence_variant_decreasing_transcript_stability', 'Sequence variant decreases the stability (half-life) of the transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000085', 'sequence_variation_affecting_level_of_transcript', 'A sequence variation that causes a change in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000086', 'sequence_variation_decreasing_level_of_transcript', 'A sequence variation that causes a decrease in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000087', 'sequence_variation_increasing_level_of_transcript', 'A sequence_variation that causes an increase in the level of mature, spliced and processed RNA, resulting from a change in the corresponding DNA sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000088', 'sequence_variant_affecting_translational_product', 'A sequence variant causing a change in primary translation product of a transcript." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000089', 'sequence_variant_causing_no_change_of_translational_product', 'The sequence variant at RNA level does not lead to any change in polypeptide." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000092', 'sequence_variant_causing_complex_change_of_translational_product', 'Any sequence variant effect that is known at nucleotide level but cannot be explained by using other key terms." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000093', 'sequence_variant_causing_amino_acid_substitution', 'The replacement of a single amino acid by another." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000094', 'sequence_variant_causing_conservative_amino_acid_substitution', 'The replacement of a single amino acid by another." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000095', 'sequence_variant_causing_nonconservative_amino_acid_substitution', 'The replacement of a single amino acid by another." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000096', 'sequence_variant_causing_amino_acid_insertion', 'The insertion of one or more amino acids from the polypeptide, without affecting the surrounding sequence." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000097', 'sequence_variant_causing_amino_acid_deletion', 'The deletion of one or more amino acids from the polypeptide, without affecting the surrounding sequence." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000098', 'sequence_variant_causing_polypeptide_truncation', 'The translational product is truncated at its C-terminus, usually a result of a nonsense codon change in transcript (SO:1000062)." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000099', 'sequence_variant_causing_polypeptide_elongation', 'The extension of the translational product at either (or both) the N-terminus and/or the C-terminus." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000100', 'mutation_causing_polypeptide_N_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000101', 'mutation_causing_polypeptide_C_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000102', 'sequence_variant_affecting_level_of_translational_product', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000103', 'sequence_variant_decreasing_level_of_translation_product', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000104', 'sequence_variant_increasing_level_of_translation_product', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000105', 'sequence_variant_affecting_polypeptide_amino_acid_sequence', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000106', 'mutation_causing_inframe_polypeptide_N_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000107', 'mutation_causing_out_of_frame_polypeptide_N_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000108', 'mutaton_causing_inframe_polypeptide_C_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000109', 'mutation_causing_out_of_frame_polypeptide_C_terminal_elongation', '." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000110', 'frame_restoring_sequence_variant', 'A mutation that reverts the sequence of a previous frameshift mutation back to the initial frame." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000111', 'sequence_variant_affecting_3D_structure_of_polypeptide', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000112', 'sequence_variant_causing_no_3D_structural_change', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000115', 'sequence_variant_causing_complex_3D_structural_change', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000116', 'sequence_variant_causing_conformational_change', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000117', 'sequence_variant_affecting_polypeptide_function', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000118', 'sequence_variant_causing_loss_of_function_of_polypeptide', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000119', 'sequence_variant_causing_inactive_ligand_binding_site', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000120', 'sequence_variant_causing_inactive_catalytic_site', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000121', 'sequence_variant_causing_polypeptide_localization_change', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000122', 'sequence_variant_causing_polypeptide_post_translational_processing_change', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000123', 'polypeptide_post_translational_processing_affected', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000124', 'sequence_variant_causing_partial_loss_of_function_of_polypeptide', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000125', 'sequence_variant_causing_gain_of_function_of_polypeptide', 'A mutation that changes the amino acid sequence of the peptide in such a way that it changes the 3D structure of the molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000126', 'sequence_variant_affecting_transcript_secondary_structure', 'A sequence variant that affects the secondary structure (folding) of the RNA transcript molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000127', 'sequence_variant_causing_compensatory_transcript_secondary_structure_mutation', 'A sequence variant that affects the secondary structure (folding) of the RNA transcript molecule." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000132', 'sequence_variant_effect', 'The effect of a change in nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000134', 'sequence_variant_causing_polypeptide_fusion', 'The effect of a change in nucleotide sequence." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000136', 'autosynaptic_chromosome', 'An autosynaptic chromosome is the aneuploid product of recombination between a pericentric inversion and a cytologically wild-type chromosome." [PMID:6804304]');
INSERT INTO soterms (id, name, definition) VALUES ('000138', 'homo_compound_chromosome', 'A compound chromosome whereby two copies of the same chromosomal arm attached to a common centromere. The chromosome is diploid for the arm involved." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000140', 'hetero_compound_chromosome', 'A compound chromosome whereby two arms from different chromosomes are connected through the centromere of one of them." [FB:reference_manual, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000141', 'chromosome_fission', 'A chromosome that occurred by the division of a larger chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000142', 'dexstrosynaptic_chromosome', 'An autosynaptic chromosome carrying the two right (D = dextro) telomeres." [FB:manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000143', 'laevosynaptic_chromosome', 'LS is an autosynaptic chromosome carrying the two left (L = levo) telomeres." [FB:manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000144', 'free_duplication', 'A chromosome structure variation whereby the duplicated sequences are carried as a free centric element." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000145', 'free_ring_duplication', 'A ring chromosome which is a copy of another chromosome." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000147', 'deficient_translocation', 'A chromosomal deletion whereby a translocation occurs in which one of the four broken ends loses a segment before re-joining." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000148', 'inversion_cum_translocation', 'A chromosomal translocation whereby the first two breaks are in the same chromosome, and the region between them is rejoined in inverted order to the other side of the first break, such that both sides of break one are present on the same chromosome. The remaining free ends are joined as a translocation with those resulting from the third break." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000149', 'bipartite_duplication', 'An interchromosomal mutation whereby the (large) region between the first two breaks listed is lost, and the two flanking segments (one of them centric) are joined as a translocation to the free ends resulting from the third break." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000150', 'cyclic_translocation', 'A chromosomal translocation whereby three breaks occurred in three different chromosomes. The centric segment resulting from the first break listed is joined to the acentric segment resulting from the second, rather than the third." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000151', 'bipartite_inversion', 'A chromosomal inversion caused by three breaks in the same chromosome; both central segments are inverted in place (i.e., they are not transposed)." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000152', 'uninverted_insertional_duplication', 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000153', 'inverted_insertional_duplication', 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000154', 'insertional_duplication', 'A chromosome duplication involving the insertion of a duplicated region (as opposed to a free duplication)." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000155', 'interchromosomal_transposition', 'A chromosome structure variation whereby a transposition occurred between chromosomes." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000156', 'inverted_interchromosomal_transposition', 'An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segment." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000157', 'uninverted_interchromosomal_transposition', 'An interchromosomal transition where the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000158', 'inverted_intrachromosomal_transposition', 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically inverted orientation with respect to its flanking segments." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000159', 'uninverted_intrachromosomal_transposition', 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the insertion is in cytologically the same orientation as its flanking segments." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000160', 'unoriented_insertional_duplication', 'An insertional duplication where a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000161', 'unoriented_interchromosomal_transposition', 'An interchromosomal transposition whereby a copy of the segment between the first two breaks listed is inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000162', 'unoriented_intrachromosomal_transposition', 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000170', 'uncharacterized_chromosomal_mutation', 'An intrachromosomal transposition whereby the segment between the first two breaks listed is removed and inserted at the third break; the orientation of the insertion with respect to its flanking segments is not recorded." [FB:reference_manual]');
INSERT INTO soterms (id, name, definition) VALUES ('000171', 'deficient_inversion', 'A chromosomal deletion whereby three breaks occur in the same chromosome; one central region is lost, and the other is inverted." [FB:reference_manual, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000173', 'tandem_duplication', 'A duplication consisting of 2 identical adjacent regions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000175', 'partially_characterized_chromosomal_mutation', 'A duplication consisting of 2 identical adjacent regions." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000180', 'sequence_variant_affecting_gene_structure', 'A sequence_variant_effect that changes the gene structure." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000181', 'sequence_variant_causing_gene_fusion', 'A sequence_variant_effect that changes the gene structure by causing a fusion to another gene." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000182', 'chromosome_number_variation', 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000183', 'chromosome_structure_variation', 'A kind of chromosome variation where the chromosome complement is not an exact multiple of the haploid number." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000184', 'sequence_variant_causes_exon_loss', 'A sequence variant affecting splicing and causes an exon loss." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000185', 'sequence_variant_causes_intron_gain', 'A sequence variant effect, causing an intron to be gained by the processed transcript; usually a result of a donor acceptor mutation (SO:1000072)." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('000186', 'sequence_variant_causing_cryptic_splice_donor_activation', 'A sequence variant effect, causing an intron to be gained by the processed transcript; usually a result of a donor acceptor mutation (SO:1000072)." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001186', 'sequence_variant_causing_cryptic_splice_acceptor_activation', 'A sequence variant effect, causing an intron to be gained by the processed transcript; usually a result of a donor acceptor mutation (SO:1000072)." [EBI:www.ebi.ac.uk/mutations/recommendations/mutevent.html]');
INSERT INTO soterms (id, name, definition) VALUES ('001187', 'alternatively_spliced_transcript', 'A transcript that is alternatively spliced." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001188', 'encodes_1_polypeptide', 'A gene that is alternately spliced, but encodes only one polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001189', 'encodes_greater_than_1_polypeptide', 'A gene that is alternately spliced, and encodes more than one polypeptide." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001190', 'encodes_different_polypeptides_different_stop', 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different stop codons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001191', 'encodes_overlapping_peptides_different_start', 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start codons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001192', 'encodes_disjoint_polypeptides', 'A gene that is alternately spliced, and encodes more than one polypeptide, that do not have overlapping peptide sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001193', 'encodes_overlapping_polypeptides_different_start_and_stop', 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start and stop codons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001194', 'alternatively_spliced_gene_encoding_greater_than_1_polypeptide_coding_regions_overlapping', 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences, but use different start and stop codons." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001195', 'encodes_overlapping_peptides', 'A gene that is alternately spliced, and encodes more than one polypeptide, that have overlapping peptide sequences." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001196', 'cryptogene', 'A maxicircle gene so extensively edited that it cannot be matched to its edited mRNA sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001197', 'dicistronic_primary_transcript', 'A primary transcript that has the quality dicistronic." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001217', 'member_of_regulon', 'A primary transcript that has the quality dicistronic." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001244', 'alternatively_spliced_transcript_encoding_greater_than_1_polypeptide_different_start_codon_different_stop_codon_coding_regions_non_overlapping', 'A primary transcript that has the quality dicistronic." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001246', 'CDS_independently_known', 'A CDS with the evidence status of being independently known." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001247', 'orphan_CDS', 'A CDS whose predicted amino acid sequence is unsupported by any experimental evidence or by any match with any other known sequence." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001249', 'CDS_supported_by_domain_match_data', 'A CDS that is supported by domain similarity." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001251', 'CDS_supported_by_sequence_similarity_data', 'A CDS that is supported by sequence similarity data." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001254', 'CDS_predicted', 'A CDS that is predicted." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001255', 'status_of_coding_sequence', 'A CDS that is predicted." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001259', 'CDS_supported_by_EST_or_cDNA_data', 'A CDS that is supported by similarity to EST or cDNA data." [SO:xp]');
INSERT INTO soterms (id, name, definition) VALUES ('001260', 'internal_Shine_Dalgarno_sequence', 'A Shine-Dalgarno sequence that stimulates recoding through interactions with the anti-Shine-Dalgarno in the RNA of small ribosomal subunits of translating ribosomes. The signal is only operative in Bacteria." [PMID:12519954, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001261', 'recoded_mRNA', 'The sequence of a mature mRNA transcript, modified before translation or during translation, usually by special cis-acting signals." [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8811194&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('001262', 'minus_1_translationally_frameshifted', 'An attribute describing a translational frameshift of -1." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001263', 'plus_1_translationally_frameshifted', 'An attribute describing a translational frameshift of +1." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001264', 'mRNA_recoded_by_translational_bypass', 'A recoded_mRNA where translation was suspended at a particular codon and resumed at a particular non-overlapping downstream codon." [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8811194&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('001265', 'mRNA_recoded_by_codon_redefinition', 'A recoded_mRNA that was modified by an alteration of codon meaning." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001266', 'stop_codon_redefinition_as_selenocysteine', 'A recoded_mRNA that was modified by an alteration of codon meaning." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001267', 'stop_codon_readthrough', 'A recoded_mRNA that was modified by an alteration of codon meaning." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001268', 'recoding_stimulatory_region', 'A site in an mRNA sequence that stimulates the recoding of a region in the same mRNA." [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12519954&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('001269', 'four_bp_start_codon', 'A non-canonical start codon with 4 base pairs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001270', 'stop_codon_redefinition_as_pyrrolysine', 'A non-canonical start codon with 4 base pairs." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001271', 'archaeal_intron', 'An intron characteristic of Archaeal tRNA and rRNA genes, where intron transcript generates a bulge-helix-bulge motif that is recognised by a splicing endoribonuclease." [PMID:9301331, SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001272', 'tRNA_intron', 'An intron found in tRNA that is spliced via endonucleolytic cleavage and ligation rather than transesterification." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001273', 'CTG_start_codon', 'A non-canonical start codon of sequence CTG." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001274', 'SECIS_element', 'The incorporation of selenocysteine into a protein sequence is directed by an in-frame UGA codon (usually a stop codon) within the coding region of the mRNA. Selenoprotein mRNAs contain a conserved secondary structure in the 3 UTR that is required for the distinction of UGA stop from UGA selenocysteine. The selenocysteine insertion sequence (SECIS) is around 60 nt in length and adopts a hairpin structure which is sufficiently well-defined and conserved to act as a computational screen for selenoprotein genes." [http://www.sanger.ac.uk/cgi-bin/Rfam/getacc?RF00031]');
INSERT INTO soterms (id, name, definition) VALUES ('001275', 'retron', 'Sequence coding for a short, single-stranded, DNA sequence via a retrotransposed RNA intermediate; characteristic of some microbial genomes." [SO:ma]');
INSERT INTO soterms (id, name, definition) VALUES ('001277', 'three_prime_recoding_site', 'The recoding stimulatory signal located downstream of the recoding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001279', 'three_prime_stem_loop_structure', 'A recoding stimulatory region, the stem-loop secondary structural element is downstream of the redefined region." [PMID:12519954, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001280', 'five_prime_recoding_site', 'The recoding stimulatory signal located upstream of the recoding site." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001281', 'flanking_three_prime_quadruplet_recoding_signal', 'Four base pair sequence immediately downstream of the redefined region. The redefined region is a frameshift site. The quadruplet is 2 overlapping codons." [PMID:12519954, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001282', 'UAG_stop_codon_signal', 'A stop codon signal for a UAG stop codon redefinition." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001283', 'UAA_stop_codon_signal', 'A stop codon signal for a UAA stop codon redefinition." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001284', 'regulon', 'A group of genes, whether linked as a cluster or not, that respond to a common regulatory signal." [ISBN:0198506732]');
INSERT INTO soterms (id, name, definition) VALUES ('001285', 'UGA_stop_codon_signal', 'A stop codon signal for a UGA stop codon redefinition." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001286', 'three_prime_repeat_recoding_signal', 'A recoding stimulatory signal, downstream sequence important for recoding that contains repetitive elements." [PMID:12519954, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('001287', 'distant_three_prime_recoding_signal', 'A recoding signal that is found many hundreds of nucleotides 3 of a redefined stop codon." [http://www.ncbi.nlm.nih.gov\:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=8709208&dopt=Abstract]');
INSERT INTO soterms (id, name, definition) VALUES ('001288', 'stop_codon_signal', 'A recoding stimulatory signal that is a stop codon and has effect on efficiency of recoding." [PMID:12519954, SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('000061', 'databank_entry', 'The sequence referred to by an entry in a databank such as GenBank or SwissProt." [SO:ke]');
INSERT INTO soterms (id, name, definition) VALUES ('ant_of', 'variant_of', 'A is a variant (mutation) of A = definition every instance of A is either an immediate mutation of some instance of A, or there is a chain of immediate mutation processes linking A to some instance of A." [SO:immuno_workshop]');
